<!doctype html>
<html>
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="shortcut icon" href="https://static.igem.wiki/teams/4294/wiki/favicon-white.png"/>
    <link rel="license" href="https://creativecommons.org/licenses/by/4.0/"/>
    
    <!-- Custom CSS -->
    <link href="{{ url_for('static', filename = 'css/collaborations_style_aspa.css') }}" rel="stylesheet">

    <!-- Loader -->
    <script type= "text/javascript" src="../../athens/static/scripts/jquery.min.js"></script>

    <!--Custom menu CSS-->
    <link href="{{url_for('static',filename='css/menu_style_aspa.css')}}" rel="stylesheet">

    <!--Custom footer CSS-->
    <link href="{{url_for('static',filename='css/footer_style_aspa.css')}}" rel="stylesheet">

    <!--Custom to top button CSS-->
    <link href="{{url_for('static',filename='css/to_top_button.css')}}" rel="stylesheet">
    <title>PERspectives | Collaborations</title>
</head>

<body>
<!-- Navigation -->
{% include 'menu_custom.html' %}

<!--To top button-->
{% include 'to_top_button.html'%}

<!--Loader-->   
{% include 'loader.html'%}

<!--Header-->

<div class="page_header">
    <img class="img_header" src="https://static.igem.wiki/teams/4294/wiki/team-page/project-reveal-1920-1080-px-4.gif" width="100%">
    <div class="heading"> Collaborations </div>    
</div>


<section class="timeline-section">
    <h1><span style="color: #3aa9c5;"> Organised by iGEM Athens </span></h1> <br><br><br>
 <div class="timeline-items">
    <div class="timeline-item">
        <div class="timeline-dot-athens"></div>
        <div class="timeline-date-athens">iGEM Athens</div>
        <div class="timeline-content">
            <div class="collapsible">
                <input type="checkbox" id="noise">
                <label for="noise">Exploring Biological Noise</label>
                <div class="collapsible-text">
                    <p>Noise, in electronics, is an unwanted disturbance in an electrical signal. On the other hand, in information theory, noise is considered to be 
                        the opposite of information. But what about noise and stochasticity in biological systems? 
                        <br><br>
                        Noise permeates biology on all levels, from the most basic molecular, sub-cellular processes to the dynamics of tissues, organs, organisms, 
                        and populations.  Due to biological noise, it is mostly impossible to determine with absolute certainty how a system will evolve from a 
                        specific initial state. The stochastic fluctuations in the level of cellular components can be caused by multiple factors. They can roughly be 
                        divided into two categories: intrinsic and extrinsic. Intrinsic noise refers to the stochastic fluctuations within the system under 
                        consideration, usually caused by the inherently probabilistic nature of the underlying biochemical reactions. The stochastic processes outside 
                        the system under consideration may serve a source of extrinsic noise.
                        <br><br>
                        At first, synthetic biologists considered stochastic gene expression an important obstacle to overcome. Nowadays, it has arguably become one 
                        of the main insights contributed by the discipline, since it reconstructs our comprehension of why, how and when specific genes are expressed. 
                        Within synthetic biology research, the use of the term noise often refers to stochastic fluctuations which can potentially have a functional status. For example, with synthetic biology pursuing the re-organisation and control of biological components to make functional devices, biological noise can be a source of inspiration and a field of observation for scientists in order to design new systems that exhibit the desired properties. Another way to approach biological stochasticity as a functional tool, is by considering it as an accelerator of the pace of evolution. 
                        <br><br>
                        However, despite the positive effect of biological noise in the field of synthetic biology, there are cases where the systems being designed 
                        must be very precise. One such example is the system our team has designed this year. The decision-making in the system we propose in our 
                        project is based on multi-prismatic stimuli which have different weight and importance for the final result. With our system based on data 
                        collection and evaluation biological noise can lead to false results and be a problem for making the final decision.
                        <br><br>
                        This is why our team during the Aegean Meeting, organised by iGEM Patras, gathered many Greek iGEMers and organised the following collaboration. 
                        Wanting to observe noise in biological systems and in which cases it can be a problem, we organised the following game. Twenty iGEMers were 
                        placed in specific positions to form a square and each of them was given three circular cards of different color, one red, one green and one 
                        blue. Each time a member of our team told the participants a specific word and then they had to pick up the card with the color they thought 
                        matched the word they had heard and place it above their head. Their decision shouldn’t have been influenced by what color the other 
                        participants had picked. We took a photo of how the square was shaped in color for each word that was heard. The technique we followed was 
                        simple. At the beginning we told the participants apparently 'simple' words that clearly express one of the three colors. For example, the 
                        word 'fire' would obviously be represented by the color red for most participants. But then we moved on to much more complex words, much more 
                        abstract, where the choice of color is purely subjective and is not influenced by the physical properties of the idea or the object. The result
                         for each word is shown below.
                        <br><br>
                        Having now received the photos in our hands we were ready to do a little statistical analysis of the results and draw our conclusions about 
                        the noise. The following chart summarizes our findings.  
                        <br><br>
                        <img src="https://static.igem.wiki/teams/4294/wiki/collaborations/noise/screenshot-2022-10-02-at-15-35-42-noise-our-collab-noise-our-collab-pdf.png" width="100%">
                        <br><br>
                        The square formed by the participants during our collaboration represents a biological system, by observing the choice of the chosen color for 
                        each word we are able to judge how an individual member would react in relation to the decision that the entire biological system finally made. 
                        As expected in the first words the participants heard the final decision was clear, the word fire brings to mind the color red, the word sky 
                        blue and the word forest green. The three words that showed 100% agreement between participants were the word fire, the word forest, and the 
                        words sky and dream. However, as we progress to much more complex concepts such as the word success or the word teamwork, the biological noise 
                        increases and now each participant can have a different opinion than what the biological system as a whole finally chooses. The most biological 
                        noise was presented in the words success and party on the island where the change of the decision of two participants would have a great 
                        influence on the final result. 
                        <br><br>
                        The conclusions we draw from this action have a big impact on how we approach our system. It is also an explanation of the reason why the 
                        biosensor we designed consists of multiple populations. So, as our system aims to recognize and analyse multifaceted stimuli and combine 
                        information, the final decision must come from many populations and be the result of the majority of the decision of the individual populations.
                         If we had designed a biosensor that consisted of only one population, it might be effective in detecting simple stimuli and recognizing small 
                         patterns, but it could not be reliable for making decisions based on complex stimuli, each of which has to have a different weight for the 
                         final decision. In conclusion, the proposed system consists of many subpopulations of bacteria and thus can overcome metabolic barriers and 
                         make better decisions since it is based on majority-based choices.
                        <br><br>
                        Finally, we would like to thank iGEM Patras because our collaboration took place during the Aegean Meeting they organised. Also, we would like 
                        to thank the teams that participated: Patras, Crete, Thessaloniki and Thessaly, Patras_Uni_Hellas for their excellent cooperation and the will 
                        to explore biological noise with us! 
                        <br><br>
                        References
                        <br><br>
                        1. <a href="https://link.springer.com/article/10.1007/s11569-020-00366-4">https://link.springer.com/article/10.1007/s11569-020-00366-4</a>
                        <br>
                        2. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4033672/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4033672/</a> 
                    </p>
                    <br><br>
                    <center>
                    <div class="container_tedx">
                        <div class="wrapper_tedx">
                            <div class="wrapper-holder-tedx">
                                <div id="tedx1"></div>
                                <div id="tedx2"></div>
                                <div id="tedx3"></div>
                                <div id="tedx4"></div>
                                <div id="tedx5"></div>
                                <div id="tedx6"></div>
                                <div id="tedx7"></div>
                                <div id="tedx8"></div>
                                <div id="tedx9"></div>
                                <div id="tedx10"></div>
                                <div id="tedx11"></div>
                                <div id="tedx12"></div>
                                <div id="tedx13"></div>
                                <div id="tedx14"></div>
                                <div id="tedx15"></div>
                            </div>
                        </div>
                        <div class="button-holder-tedx">
                            <span class="button_tedx" onclick="currentDiv(1, 'wrapper-holder-tedx');"></span>
                            <span class="button_tedx" onclick="currentDiv(2, 'wrapper-holder-tedx');"></span>
                            <span class="button_tedx" onclick="currentDiv(3, 'wrapper-holder-tedx');"></span>
                            <span class="button_tedx" onclick="currentDiv(4, 'wrapper-holder-tedx');"></span>
                            <span class="button_tedx" onclick="currentDiv(5, 'wrapper-holder-tedx');"></span>
                            <span class="button_tedx" onclick="currentDiv(6, 'wrapper-holder-tedx');"></span>
                            <span class="button_tedx" onclick="currentDiv(7, 'wrapper-holder-tedx');"></span>
                            <span class="button_tedx" onclick="currentDiv(8, 'wrapper-holder-tedx');"></span>
                            <span class="button_tedx" onclick="currentDiv(9, 'wrapper-holder-tedx');"></span>
                            <span class="button_tedx" onclick="currentDiv(10, 'wrapper-holder-tedx');"></span>
                            <span class="button_tedx" onclick="currentDiv(11, 'wrapper-holder-tedx');"></span>
                            <span class="button_tedx" onclick="currentDiv(12, 'wrapper-holder-tedx');"></span>
                            <span class="button_tedx" onclick="currentDiv(13, 'wrapper-holder-tedx');"></span>
                            <span class="button_tedx" onclick="currentDiv(14, 'wrapper-holder-tedx');"></span>
                            <span class="button_tedx" onclick="currentDiv(15, 'wrapper-holder-tedx');"></span>
                        </div>
                    </div>
                    </center>
                </div>       
            </div>
        </div>
    </div>
    <div class="timeline-item">
        <div class="timeline-dot-athens"></div>
        <div class="timeline-date-athens">iGEM Athens</div>
        <div class="timeline-content">
            <div class="collapsible">
                <input type="checkbox" id="Synbio Stories">
                <label for="Synbio Stories">Synbio Stories</label>
                <div class="collapsible-text">
                    <p>
                        An educational process that arose during the first few months of our journey in the competition was the one with the team Thessaloniki. 
                        We reached out to them to organize a virtual game on Instagram based on a very popular board game. Every week one of the teams would post a 
                        mystery story related to synthetic biology, leaving the followers with a huge question. The members of this team accepted and answered 
                        questions from the public for 24 hours. After a day, the other team would reveal the end of the mystery story, finally answering the big 
                        question that had been posed to the audience. 
                        <br><br>
                        Our collaboration aimed to increase public engagement with small questions about synthetic biology. Our followers had the opportunity to 
                        be concerned about "problems" that occurred in every iGEM team on a daily basis, as some of our mystery stories were indeed true. 
                        Troubleshooting is a big part of the iGEM competition, by creating those “SynBio Stories” we gave the participants a good picture of what 
                        paths our thinking should follow every day in the lab when problems arise or things don't go exactly as planned. We were happy to receive so 
                        many questions and possible solutions from synthetic biology fans. Some of them were very close to the real solution of the problems we were 
                        presenting each time.
                    </p>
                    <center>
                    <img class="story-1-img" src="https://static.igem.wiki/teams/4294/wiki/collaborations/story1.png">
                    <img class="story-2-img" src="https://static.igem.wiki/teams/4294/wiki/collaborations/story2.png">
                    <img class="story-3-img" src="https://static.igem.wiki/teams/4294/wiki/collaborations/story3.png">
                    </center>
                </div>
            </div>
        </div>
    </div>
    <div class="timeline-item">
        <div class="timeline-dot-athens"></div>
        <div class="timeline-date-athens">iGEM Athens</div>
        <div class="timeline-content">
            <div class="collapsible">
                <input type="checkbox" id="mistakes">
                <label for="mistakes">Beautiful Mistakes</label>
                <div class="collapsible-text">
                    <p>
                    One day in the lab we noticed that a plate was contaminated and maggots had grown on it. However, looking at them through the stereoscope, 
                    we noticed that they had grown into a beautiful shape that had a grace to it. Wanting to emphasize the spirit that mistakes are part of the 
                    process, and we can learn from them and become better, we decided to create a mini collage to emphasize that our mistakes, if they are harmless, 
                    can be very beautiful.
                    <center>
                    <img class="mistakes-img" src="https://static.igem.wiki/teams/4294/wiki/collaborations/mistakes.png">
                    </center>
                    <br><br>
                    We also thought that we are probably not the only group that has encountered these kinds of problems, and that many of our colleagues may have 
                    made "beautiful mistakes". So we decided to approach them for a collaboration. We asked them to send us photos of lab errors that had an 
                    aesthetically pleasing look. We are so happy that the teams iGEM Tec CEM, Estonia_TUIT and iGEM-RUM responded. We saw beautiful mistakes in 
                    every picture they sent us, from gels that were not running in a proper way to single dots growing on plates. We thank them for not being ashamed 
                    to show the world that mistakes are pretty and useful sometimes, and we hope they enjoy the collage as much as we do!
                    </p>
                </div>
            </div>
        </div>
    </div>
 </div>
</section>

<section class="timeline-section">
    <h1><span style="color: #61b872;"> Organised by other iGEM Teams </span></h1> <br><br><br>
    <div class="timeline-items">
        <div class="timeline-item">
            <div class="timeline-dot"></div>
            <div class="timeline-date">iGEM Maastricht</div>
            <div class="timeline-content">
                   <div class="collapsible">
                       <input type="checkbox" id="iGEM Maastricht">
                       <label for="iGEM Maastricht">MSP Vector</label>
                       <div class="collapsible-text">
                       <p>iGEM MSP-Maastricht organised one more year the iGEM Proceedings Journal 2022, an online Journal accessible to everyone 
                           interested in synthetic biology. We are honored to be a part of it.
                        <br><br>
                        Through this initiative, we had the opportunity to review the field of biocomputing by exploring realted papers and finally submitting 
                        "A Review On Extraordinary Biocomputing Applications". Through this work, we provide an overview of the most eye-catching software and 
                        hardware applications of biocomputing, from tunable oscillators to biological encryption systems. Our goal was to help researchers and welcome 
                        newcomers to the field of biocomputing.
                        <br><br>
                        The procedure consisted of three main steps:
                        </p>
                        <ul>
                            <li>1. Submission Step: First, we had to write the review an submit it to the iGEM Maastricht organising committe in order to be peer reviewed.</li>
                            <li>2. Peer-Review Step: Then, we had to peer-review the papers of three different teams. At this step, our paper was also peer-reviewed by three teams.</li>
                            <li>3. Revisions Step: Finally, after receiving our reviews we had to integrate the revisions requested and submit our final improved paper.</li>
                        </ul>
                        <br><br>
                        <p>
                        This year's alteration lies in the fact that the three best articles, based on the pper-review grading, were forwarded to researchers and journal editors working in the field of 
                        synthetic biology for review and feedback. The articles and the review from Dr. George Church, Dr, Michael Funk and Dr. Yevgeniya Nusinovich are 
                        included in iGEM MSP-Maastricht team’s Journal 2022.
                        <br><br>
                        You can find more about iGEM MSP-Maastricht team’s Journal <a href="https://mspvector.wordpress.com/journal/">here</a> 
                        and our final submission <a href="https://mspvector.files.wordpress.com/2022/10/msp-vector_2022.pdf?force_download=true">here</a>
                        </p>
                       </div>
               </div>
            </div>
        </div>
       <div class="timeline-item">
           <div class="timeline-dot"></div>
           <div class="timeline-date">iGEM Technion</div>
           <div class="timeline-content">
               <div class="collapsible">
                   <input type="checkbox" id="iGEM Technion">
                   <label for="iGEM Technion">More than a collab</label>
                   <div class="collapsible-text">
                   <p>
                   This year we had the pleasure of being in close contact with members of the Technion-Israel team. Our communication was multi-level and both groups were 
                   equally helped. In addition, it was especially pleasant to organize meetings where we could talk and see that we were not alone against the difficulties 
                   we faced.
                   <br><br>
                   <span style="font-family: 'Glacial Indifference Bold';">Consultative approach</span>
                   <br><br>
                   <img class="tech-1-img" src="https://static.igem.wiki/teams/4294/wiki/collaborations/technion-2.png">
                   During our first communication, the dry lab members understood that when modeling their systems they would need to take similar approaches. 
                   So the idea to join our forces and knowledge and create a two-way consulting relationship began to become a reality. 
                   <br><br>
                   Initially team Technion, who needed a kinetic based on enzyme equations sent a very early draft of their model to us so that we could understand the main 
                   key axes they intended to analyze. Based on knowledge we already had on enzymatic kinetics, we created a file with tips and scientific tools that could be 
                   useful to them during the development of their model. The proposals focus on areas such as:
                   <br><br>
                   On <span style="color: #3aa9c5;">enzymatic kinetics</span> we suggested using the COPASI software. COPASI is widely used in many research projects for: modeling biological, biochemical, 
                   and chemical systems, in the development of theory and computational methods,in the development of “wet” laboratory methods. This software allows the 
                   user to determine kinetic constants and production rates based on an experimental dataset. Also it gives the opportunity to predict how a system will 
                   evolve overtime based on kinetic constant the user has imported. Members of our team were already familiar with this kind of software and thought that 
                   is would be useful for team Technion to use it and reach us whether a problem came up.
                   <br><br>
                   We also suggested Matlab for running their kinetic ODEs. We provided them with useful tutorials on how to use Matlab and SimBiology.
                   <br><br>
                   <img class="tech-2-img" src="https://static.igem.wiki/teams/4294/wiki/collaborations/technion-3.png">
                   On <span style="color: #3aa9c5;">microorganisms growth</span> we suggested using the Monod model. We provided them useful equations and diagrams in order to have a better understanding on 
                   the different faces of a growth curve. We also provided equations on Monod first and second order kinetics and we summarised a way to determine the 
                   kinetic constants based on experimental data. 
                   <br><br>
                   Both teams were dealing with <span style="color: #3aa9c5;">diffusion</span>. On this subject, team Technion was already aware of the Fick’s Diffusion Law and intended to use it just like 
                   us. So we discussed on possible difficulties that would come up in the process of modeling diffusion and were open in new ideas and suggestions.
                   <br><br>
                   Finally, we suggested using the <span style="color: #3aa9c5;">COMSOL software</span> if team Technion was interested in dealing with matter transfer phenomena. This would maybe be useful 
                   if they wanted to simulate mass transfer in, outside a cell or across membranes. 
                   <br><br>
                   On a follow-up meeting between the dry lab members of each team, we were so happy to hear that they actually implemented some of our suggestions. 
                   <br><br>
                   <span style="font-family: 'Glacial Indifference Bold';">Recording a podcast episode</span>
                   <br><br>
                   <img class="tech-3-img" src="https://static.igem.wiki/teams/4294/wiki/collaborations/technion-1.png">
                   As a result of our collaboration with the team Technion, the proposal to record a podcast episode came up. We were so happy to be part of the “Angel Roots” Podcast. In the episode our team guested we talked about the scientific part of the project as well as the financial part of the iGEM Competition.  We were so lucky because team Technion had adjusted this episode’s questions according to our project and the bioinformatic aspects of it. We answered questions related to the following axes:
                   <br><br>
                   <ul style="font-size: 25px;">
                       <li>Who are we and how did we found out about iGEM?</li>
                       <li>Our project’s goal and its applications.</li>
                       <li>What financial difficulties we faced during the fundraising process this year?</li>
                       <li>Finally, we were asked to give some advice to the future iGEMers about the iGEM competition in general but also in groups that are thinking of 
                           revolving around bioinformatics. So in this case we also suggested some useful software tools that we used as well.</li>
                   </ul>
                   <br>
                   </p>
                   <p>
                   If you want to know more about the “Angel Roots” Podcast you can find all the episodes 
                   <a href="https://open.spotify.com/show/10ffNOf6d78yRF0AyQH4r0?fbclid=IwAR116hp0O2RXjdvKfLv4XlRf_QGQsLo07AKPrhbxgzdeLzlk_0zfsVHL68w">here</a> 
                   and if you want to listen to our episode click 
                   <a href="https://open.spotify.com/episode/1AbJ4teEYFo8kPydlgeYEr">here</a>.
                   <br><br>
                   <span style="font-family: 'Glacial Indifference Bold';">Mock Jamboree Judging Session</span>
                   <br><br>
                    We were so happy that iGEM Technion-Israel invited us to be a part of the Mock Jamboree that they organised. The idea is that all 
                    teams participating will have the opportunity to make a presentation of their project in an “international” environment. This action 
                    is intended to occur after the wiki freeze and after the teams have prepared their final presentation for the Judging Session. It is 
                    expected to be a three-hour event with teams from different countries and projects that fall in different tracks of the competition. 
                    <br><br>
                    Thus, a three-hour event will be organised where the teams will be invited to present their work similar to the way it will be done 
                    in the official jamboree. It will be a good opportunity to expose our project in an environment similar to that of the competition to 
                    see what questions the other teams will have. In addition, we will hear thoughts, comments and suggestions from the other groups on 
                    how we presented our results, but at the same time we will also be inspired by their work. We believe it will be a good opportunity to 
                    meet other participants with whom we have not worked so far, and we look forward to seeing their work and how they will respond to 
                    ours.
                   </p>
                   </div>
               </div>
           </div>
       </div>
       <div class="timeline-item">
           <div class="timeline-dot"></div>
           <div class="timeline-date">iGEM Cornell, iGEM McGill, iGEM Queens</div>
           <div class="timeline-content">
               <div class="collapsible">
                   <input type="checkbox" id="iGEM Queens, iGEM Cornell, iGEM McGill">
                   <label for="iGEM Queens, iGEM Cornell, iGEM McGill">Our Favorite Bacterium Collaboration</label>
                   <div class="collapsible-text">
                   <p>In June 2022 we took part in the exciting design collaboration by Queen’s University, McGill University, and Cornell University iGEM teams. 
                       Wanting to create an educational book to educate the public, these three teams invited us to send them a form filled with information about our 
                       favourite bacterial species. We chose the bacterium <i>Bdellovibrio bacteriovorus</i>, also known as The "Vampire Bacterium". <i>Bdellovibrio</i> 
                       is a genus of Gram-negative, obligate aerobic bacteria. One of the more notable characteristics of this genus is that members can prey upon other 
                       Gram-negative bacteria and feed on the biopolymers.* Our choice was based on our intention to show to the public that in nature there are several 
                       organisms with completely different ways of living, metabolizing and reproducing. As the young representatives of the scientific community, we 
                       believe it is important to make clear that every form of life is far more complicated naturally than it is in theory. 
                       <br><br>
                       <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3917220/">*Harini K, Ajila V, Hegde S. Bdellovibrio bacteriovorus : A 
                        future antimicrobial agent? J Indian Soc Periodontol. 2013 Nov;17(6):823-5. doi: 10.4103/0972-124X.124534. PMID: 24554900; 
                        PMCID: PMC3917220.</a><br>
                        <center>
                        <img class="fav-1-img" src="https://static.igem.wiki/teams/4294/wiki/collaborations/1.png">
                        <img class="fav-2-img" src="https://static.igem.wiki/teams/4294/wiki/collaborations/2.png">
                        </center>
                    </p>
                   </div>
               </div>
           </div>
       </div>
       <div class="timeline-item">
           <div class="timeline-dot"></div>
           <div class="timeline-date">iGEM Aachen,iGEM Patras_Medicine</div>
           <div class="timeline-content">
               <div class="collapsible">
                   <input type="checkbox" id="iGEM Aachen-iGEM Patras">
                   <label for="iGEM Aachen-iGEM Patras">Being a part of “iJET” video Collaboration</label>
                   <div class="collapsible-text">
                   <p>In June 2022, team iGEM Patras Med reached out to us and invited us to take part in a collaboration aiming to create a video called “iJET”. 
                       The collaboration was organized by two teams, Patras Med and Aachen. We were asked to film a short video explaining our project that was going to 
                       be a part of a bigger video that the two teams were about to create by combining all the videos they had received. We happily wrote a script and 
                       filmed the video in a park. Furthermore, we spoke about why we were taking part in the iGEM competitions and why we like it, in which track we were 
                       competing this year. Finally, we gave a brief explanation of the project, where we are aiming and what innovative solution we give to an existing 
                       project by using synthetic biology tools. Hope you enjoy the “iJET” video 
                       <a href="">here</a>.</p>
                   </div>
               </div>
           </div>
       </div>
       <div class="timeline-item">
           <div class="timeline-dot"></div>
           <div class="timeline-date">iGEM IISER</div>
           <div class="timeline-content">
               <div class="collapsible">
                   <input type="checkbox" id="iGEM IISER">
                   <label for="iGEM IISER">Biosafety video</label>
                   <div class="collapsible-text">
                   <img class="biosafety-video-img" src="https://static.igem.wiki/teams/4294/wiki/collaborations/biosafety-video.png">   
                   <p>We participated  in a series of informative videos based on Biosafety Practices organized by Team IISER Mohali from India. This series aims 
                       to inform the public about  the measures that are carried out in the laboratories and reduce the skepticism regarding Genetic Engineering and 
                       Synthetic Biology.
                      <br><br>
                       Our team demonstrated some of the Biosafety measures followed in our laboratory. It was a very fun and creative procedure! You can find our video 
                       <a href="https://youtu.be/noctwnApqTg">here</a>.</p>
                   </div>
               </div>
           </div>
       </div>
       <div class="timeline-item">
        <div class="timeline-dot"></div>
        <div class="timeline-date">iGEM Patras</div>
        <div class="timeline-content">
            <div class="collapsible">
                <input type="checkbox" id="noise2">
                <label for="noise2">Pre Aegean and Aegean Meetings</label>
                <div class="collapsible-text">
                    <p>
                    Through Pre Aegean (May 28th, 2022) and  Aegean (July 10-12, 2022) meetings, we had the opportunity to meet the other iGEM teams, present our iGEM projects, 
                    earn about interesting things regarding sequencing and PCR techniques, get training in soft skills, and most importantly have fun!
                    <br><br>
                    We would like to thank <a href="https://2022.igem.wiki/patras/">iGEM Patras</a> for the amazing opportunity to participate in this wonderful event and make memories and friends! 
                    </p>
                    <br><br>
                    <center>
                    <div class="container_tedxx">
                        <div class="wrapper_tedxx">
                            <div class="wrapper-holder-tedxx">
                                <div id="tedxx0"></div>
                                <div id="tedxx1"></div>
                                <div id="tedxx2"></div>
                                <div id="tedxx3"></div>
                                <div id="tedxx4"></div>
                            </div>
                        </div>
                        <div class="button-holder-tedxx">
                            <span class="button_tedxx" onclick="currentDiv(1, 'wrapper-holder-tedxx');"></span>
                            <span class="button_tedxx" onclick="currentDiv(2, 'wrapper-holder-tedxx');"></span>
                            <span class="button_tedxx" onclick="currentDiv(3, 'wrapper-holder-tedxx');"></span>
                            <span class="button_tedxx" onclick="currentDiv(4, 'wrapper-holder-tedxx');"></span>
                            <span class="button_tedxx" onclick="currentDiv(5, 'wrapper-holder-tedxx');"></span>
                        </div>
                    </div>
                    </center>
                </div>       
            </div>
        </div>
    </div>
       <div class="timeline-item">
           <div class="timeline-dot"></div>
           <div class="timeline-date">iGEM Patras_Medicine</div>
           <div class="timeline-content">
               <div class="collapsible">
                   <input type="checkbox" id="DNA post">
                   <label for="DNA post">Creating a post for the World DNA Day</label>
                   <div class="collapsible-text">
                    <p>
                    We are very happy for our contribution in creating an informative post for world dna day. Team Patras_Medicine invited you to take a photo 
                    holding one of the letters A, T, U, G. They then created a beautiful collage, creating a double helix structure, simulating the DNA. The post 
                    was accompanied by informative facts about the DNA. Best way to celebrate world DNA Day.                       
                    </p>
                    <center>
                    <img class="dna-collab-img" src="https://static.igem.wiki/teams/4294/wiki/collaborations/dna-collab.png">
                    </center>
                   </div>
               </div>
           </div>
       </div>
       <div class="timeline-item">
           <div class="timeline-dot"></div>
           <div class="timeline-date">iGEM Thessaly</div>
           <div class="timeline-content">
               <div class="collapsible">
                   <input type="checkbox" id="The plates">
                   <label for="The plates">Cooperation to carry out laboratory experiments: Plates</label>
                   <div class="collapsible-text">
                       <p>
                        This year we faced many difficulties with the laboratory consumables and the equipment we had available to carry out our laboratory experiments. 
                        So when we needed two 96-well plates with black clear bottom, we turned to team <a href="https://2022.igem.wiki/thessaly/">iGEM Thessaly</a>. The members of  the team were very willing to send 
                        us what we needed and we thank them for their prompt and quick response. For our part, we were able to fill their need for two 96-well plates 
                        with black bottom that they were lacking.
                        <br><br>
                        We are very happy that through our cooperation we were able to cover the gaps of each group and help each other. We thank them for the excellent 
                        cooperation and communication in every step of the way. We believe that it is the competition’s spirit to form collaborations like this, when one 
                        team is lacking something and another that actually has a surplus helps.                                                
                       </p>
                       <center>
                        <img class="dna-collab-img" src="https://static.igem.wiki/teams/4294/wiki/collaborations/thessaly-plates.jpg">
                        </center>
                   </div>
               </div>
           </div>
       </div>
       <div class="timeline-item">
        <div class="timeline-dot"></div>
        <div class="timeline-date">iGEM Crete, iGEM Thessaly, iGEM Patras</div>
        <div class="timeline-content">
            <div class="collapsible">
                <input type="checkbox" id="Safety Collage">
                <label for="Safety Collage">Safety Collage</label>
                <div class="collapsible-text">
                    <p>
                        We participated in the <a href="https://2022.igem.wiki/crete/">iGEM Crete</a> initiative to create a collage on the occasion of the World Day for Safety and Health at Work. 
                        Each team participated with the photos of their members responsible for the safety and sent a sentence about why we consider laboratory safety important in our team. 
                        In this way, we brought the teams closer together and contributed to laboratory safety vigilance!                                               
                    </p>
                    <center>
                     <img class="dna-collab-img" src="https://static.igem.wiki/teams/4294/wiki/collaborations/lab-safety-igem-crete-collab-1.png">
                     </center>
                </div>
            </div>
        </div>
    </div>
    </div>
   </section>


   <script>
    var slideIndex = 1;

    function currentDiv(n, top_level_div) {
        showDivs(slideIndex = n, top_level_div);
    }

    function showDivs(n, top_level_div) {
        var i;
        var x = document.getElementsByClassName(top_level_div)[0].children;
        
        if (n > x.length) {slideIndex = 1}
        if (n < 1) {slideIndex = x.length}

        for (i = 0; i < x.length; i++) {
            x[i].style.display = "none";  
        }
        
        x[slideIndex-1].style.display = "block";
    }
</script>


</body>


<!-- Footer -->
{% include 'footer.html' %}

</html>