<!doctype html>
<html>
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="shortcut icon" href="https://static.igem.wiki/teams/4294/wiki/favicon-white.png"/>
    <link rel="license" href="https://creativecommons.org/licenses/by/4.0/"/>
    
    <!-- Custom CSS -->
    <link href="{{ url_for('static', filename = 'css/design_style.css') }}" rel="stylesheet">

    <!-- Loader -->
    <script type= "text/javascript" src="../../athens/static/scripts/jquery.min.js"></script>

    <!--Custom menu CSS-->
    <link href="{{url_for('static',filename='css/menu_style_aspa.css')}}" rel="stylesheet">

    <!--Custom footer CSS-->
    <link href="{{url_for('static',filename='css/footer_style_aspa.css')}}" rel="stylesheet">


     <!--Custom to top button CSS-->
     <link href="{{url_for('static',filename='css/to_top_button.css')}}" rel="stylesheet">
    <title>PERspectives | Design</title>
</head>

<body>
<!-- Navigation -->
{% include 'menu_custom.html' %}

<!--To top button-->
{% include 'to_top_button.html'%}

<!--Loader-->   
{% include 'loader.html'%}  

<div class="page_header">
    <img class="img_header" src="https://static.igem.wiki/teams/4294/wiki/wet-lab/wet-bg-darker.gif" width="100%">
    <div class="heading">Design</div>    
</div>

<div class="cc" id="cc-implementaion">
  <div class="left-side">
      <div class="index-bar">
          <ul class="lateral-index">
              <li class="lateral-content">
                  <a href="#intro">Overview</a>
              </li>
              <li class="lateral-content">
                  <a href="#general_info">General Info</a>
              </li>
              <li class="lateral-content">
                  <a href="#senders">Senders</a>
              </li>
              <li class="lateral-content">
                <a href="#receivers">Receivers</a>
              </li>
              <li class="lateral-content">
                  <a href="#references">References</a>
              </li>
          </ul>
      </div>
  </div>
  
  <div class="right-side">

<!-- Overview -->
        <section class="introduction">
            <a id="intro" class="anchorOffset"></a>
            <div class="overview">
                <h1>Overview</h1>
                <p>
                The main aspect of our project is to tackle an information processing challenge at the foundational level by proposing a strategy to 
                build a biological perceptron. The goal is to imitate every part of the perceptron algorithm by a biological process. The design of the 
                project serves this by engineering two distinct bacterial populations, each one completing a separate simple task. Together these simple 
                tasks emerge into an information processing genetically engineered machine. 
                </p>

                <center>
                <img class="img1" id="img1" src="https://static.igem.wiki/teams/4294/wiki/results/results-gifs.gif">
                <figcaption>An illustration that summarizes our bacterial perceptron.
                </figcaption>
                </center>
            </div>
        </section>
        <br><br>
<!-- general_info -->
        <section class="general_info">
        <a id="general_info" class="anchorOffset"></a>
        <h1>General Design Information</h1>
        <br><br>
        <p>

        <h2>Golden Gate Assembly</h2>
        <br>

        We considered several assembly approaches to build our constructs. We finally opted for Golden Gate Assembly, as it comprises a fast and easy method to assemble different DNA parts in a single plasmid, while offering modularity and high efficiency. The fact that we found out that the 2022 iGEM DNA Distribution kit would contain Golden Gate Assembly-compatible parts further encouraged this decision of ours, since it provided a safety net in case the parts designed by us did not function as expected.
        <br><br>
        Our Golden Gate Assembly-based parts can be divided into two major categories: level 1 and level 2 constructs. 
        <br><br>
        Building level 1 plasmids deploys the BsaI recognition and restriction sites of the acceptor vector (pTU1-A-lacZ in our case) to combine separate DNA sequences (promoter, RBS, CDS, terminator) into a functional transcriptional unit. In case the level 1 acceptor vector is designated to later become a part for level 2 assembly (as in our project), the BsaI recognition sites facing “inwards” enable the insertion of a new composite sequence between them, leaving the external BsmBI cut sites unaffected and available for a level 2 assembly, where the level 1 destination vector is converted into a part-carrying backbone for the next level of the building procedure. We also used said BsmBI recognition and restriction sequences as cut sites in diagnostic digests during our efforts to validate our constructs.
        <br><br>
        Level 2 constructs deploy BsmBI recognition and restriction sites to combine discrete transcriptional units in a single plasmid. The plasmid we utilized as a level 2 destination vector in our project was pTU2-A-RFP (p15A ori) for the final senders’ constructs. Similarly to BsmBI sites present in level 1 plasmids, in level 2 vectors, where BsmBI sequences face “inwards”, the “external” BsaI sequences can be used as cut sites to be targeted in diagnostic digests for validation purposes.
        <br><br>
        Besides containing BsmBI recognition and restriction sites, which, are traditionally utilized for level 2 assemblies, pTU2-A-RFP (colE1 ori) and pTKEI-Dest were treated as level 1 acceptor vectors during the building process, since they were used to integrate entire transcriptional units as separate DNA parts.
        <br><br>
        <b>Receivers’ Circuits</b>
        <br><br>
        For the construction of the Receivers’ Circuits Golden Gate Assembly with BsmBI was performed. The overhangs and the parts are summated in the following table.
        <br><br>

        <center>
        <table class="content-table">
          <thead>
            <tr>
              <th>Overhang Code And Sequence</th>
              <th>TUs</th>
              <th>Part Code</th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td>A: ATCT</td>
              <td>A-TU_pLux_Output_(OL,PF,PFc)-C</td>
              <td>BBa_K4294806</td>
            </tr>
            <tr class="active-row">
              <td>B: TTAG</td>
              <td>A-TU_pLuxr_PhlF_(PF+R,PFc+R)-E</td>
              <td>BBa_K4294807</td>
            </tr>
            <tr>
              <td>C: GTCA</td>
              <td>C-TU_const_LuxR_(OL)-B</td>
              <td>BBa_K4294808</td>
            </tr>
            
            <tr class="active-row">
              <td>D: GCAA</td>
              <td>C-TU_pLux_LuxR_(PF,PF+R)-B</td>
              <td>BBa_K4294809</td>
            </tr>
            
            <tr>
              <td>E: CGTA</td>
              <td>C-TU_pLux_LuxR_(PFc,PFc+R)-D</td>
              <td>BBa_K4294810</td>
            </tr>
            
            <tr class="active-row">
              <td></td>
              <td>D-TU_const_LuxR_(PFc,PFc+R)-B</td>
              <td>BBa_K4294811</td>
            </tr>

              <tr>
              <td></td>
              <td>E-TU_Output_(PF+R,PFc+R)-C</td>
              <td>BBa_K4294812</td>
            </tr>
        
            
          </tbody>
        </table>
        <figcaption>
        Table 1.
        Overhangs for Level 2 Golden Gate assembly regarding the Receivers’ Circuits. TUs were ordered as gene fragments from Twist and submitted as 
        such to the iGEM registry. The part code is, therefore, included to facilitate registry search. For the parts in the following tables, please 
        refer to the <a href="https://2022.igem.wiki/athens/parts">Parts</a> page.        
        </figcaption>
        </center>

        <br><br>
        <b>Senders' Circuits</b>
        <br><br>
        Senders’ Level 1 TUs were assembled using Golden Gate Assembly with BsaI.

        <center>
        <table class="content-table">
            <thead>
              <tr>
                <th>Overhang </th>
                <th>Junction</th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td>CTAT</td>
                <td>pTU1 vector - Promoter</td>
              </tr>
              <tr class="active-row">
                <td>TACT</td>
                <td>Promoter - RBS</td>
              </tr>
              <tr>
                <td>AATG</td>
                <td>RBS - CDS</td>
              </tr>
              
              <tr class="active-row">
                <td>GCTT</td>
                <td>CDS - Terminator</td>
              </tr>
              
              <tr>
                <td>TGTT</td>
                <td>Terminator - pTU1 vector</td>
              </tr>
              
              <tr class="active-row">
                <td>CAAA</td>
                <td>36nt LuxI oligo - CDS (sfGFP)</td>
              </tr>
              
              <tr>
                <td>AATG</td>
                <td>RBS - 36nt LuxI oligo </td>
              </tr>
            </tbody>
        </table>
        <figcaption>
          Table 2. Senders’ Level 2 TUs were assembled using Golden Gate Assembly with BsmBI.           
        </figcaption>
        </center>


        <center>
          <table class="content-table">
            <thead>
              <tr>
                <th>Overhang </th>
                <th>Junction</th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td>ATCT</td>
                <td>pTU2 vector - TU from pTU1</td>
              </tr>
              <tr class="active-row">
                <td>TGCC</td>
                <td>TU from pTU1 - TetR TU</td>
              </tr>
              <tr>
                <td>TTAG</td>
                <td>TetR TU - pTU2 vector</td>
              </tr>
              
              <tr class="active-row">
                <td>TGCC</td>
                <td>TU from pTU1 - [PlacI-tet1]</td>
              </tr>
              
              <tr>
                <td>TATG</td>
                <td>[PlacI-tet1] - TetR+Terminator from plasmid</td>
              </tr>
            </tbody>
        </table>

        <figcaption>
          Table 3. Overhangs for Level 2 Golden Gate assembly regarding the Senders’ Circuits. The boxes in blue contain overhangs used in our
           senders’ redesigned circuits with a variation in TetR constitutive expression promoter and RBS (PlacI and tet1 RBS instead of J23106 and
            B0034 RBS), which are described in more detail in the <a href="https://2022.igem.wiki/athens/engineering">Engineering Sucess</a> page.
        </figcaption>
        </center>

        <h2>Plasmid Vectors</h2>
        <br>
        After we had decided that we would build our plasmids using Golden Gate Assembly, we had to select the vectors that would “host” the regulatory and coding sequences our circuits would consist of. 
        <br><br>
        Firstly, we established certain criteria the vectors we would choose had to fulfill:
        <br><br>
        <ul>
          <li>
            They would have to be Golden Gate level 1- and level 2-compatible, which means that they should have no more than two recognition and restriction sites of the desired type IIS enzyme at the right position and with the right orientation (recognition sites facing “inwards”).
          </li>
          <li>
            They would have to be no larger than 4.5kbp in size, so that our final constructs (plasmid backbone plus 2 to 2.5kbp from inserted parts) could be chemically  transformed into our cells.
          </li>
          <li>
            They would have to be for use in bacterial cells.
          </li>
          <li>
            Vectors that would be used at different levels of cloning and for different purposes should have different antibiotics resistance genes for efficient colony selection.
          </li>
          <li>
            Since we wanted to experiment with different ways of tuning our system’s protein expression dynamics, the vector that would eventually become the backbone for our senders and receivers’ main constructs should be available in versions with different copy numbers (preferably low and medium).
          </li>
          <li>
            As we eventually decided to co-transform two plasmids in our senders to tackle the cross-talk issue, the two vectors we would choose for 
            those constructs would have to be compatible regarding their origin of replication; in other words, the origin of replication of the one 
            should not occupy the machinery responsible for the replication of the other. Information about plasmid incompatibility in terms of origin of 
            replication can be found in the following table (Table 4):

            <center>
              <table class="content-table">
                <thead>
                  <tr>
                    <th>Origin of replication (ori)</th>
                    <th>Incompatibility group</th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td>pMB1</td>
                    <td>A</td>
                  </tr>
                  <tr class="active-row">
                    <td>pBR322</td>
                    <td>A</td>
                  </tr>
                  <tr>
                    <td>colE1</td>
                    <td>A</td>
                  </tr>
                  <tr class="active-row">
                    <td>R6K</td>
                    <td>C</td>
                  </tr>
                  <tr>
                    <td>p15A</td>
                    <td>B</td>
                  </tr>
                  </tr>
                  <tr>
                    <td>pSC101</td>
                    <td>C</td>
                  </tr>
                  <tr class="active-row">
                    <td>pUC</td>
                    <td>A</td>
                  </tr>
                </tbody>
            </table>
            </center>
            <br>
            For two plasmids to coexist in the same cell, they have to belong to different incompatibility groups (e.g. A and C, B and A) [1].
            <br><br>
          </li>
          <li>
            Ideally, the vectors we would choose should also have a marker (e.g. a visual index) to facilitate the selection of the right colonies after the assembly of the constructs via Golden Gate and the subsequent transformation.
          </li>
        </ul>
        <br><br>
        We began browsing Addgene’s [2] vast plasmid catalogue to find bacterial backbones that would fulfil the prerequisites mentioned above. Our research yielded four vectors that we eventually deployed in our project design; three of the four vectors are derived from the EcoFlex kit [3].
        <br><br>
        We used each vector for a different purpose as analyzed below:
        <br><br>
        <ul>

        <li>
        pTU1-A-lacZ [3]: We utilized pTU1-A-lacZ as our level 1 (as hinted by the “1” in its name)-Golden Gate acceptor vector for assembling the LuxI expressing transcriptional units with the different RBS in our senders. This vector has an ampicillin (Amp) resistance gene and is a high copy plasmid; it is also approximately 2.5kbp in size. Since it would not be present in the senders to be induced, we did not have to select a version with a particular origin of replication (taking into consideration possible interactions with other plasmids transformed in the same cell or how copy number would affect expression dynamics). pTU1-A-lacZ contains the gene coding for LacZ, which allowed us to select recombined constructs through blue-white screening on agar plates spread with x-gal and IPTG (blue colonies: transformed with the undesired parental plasmid, white colonies: transformed with a recombinant plasmid). Another trait worth mentioning of pTU1-A-lacZ is that it is optimized to be used as a part-carrying vector should the transcriptional unit it contains be utilized as a fragment in level 2 assembly, as it also includes two BsmBI cut sites “outside” its level 1 restriction positions. The “A” in the vector’s name indicates that the transcriptional unit assembled inside this particular plasmid will be placed upstream of other transcriptional units (assembled in pTU1-A-lacZ “B”, “C”, etc. counterparts) should it be included in a level 2-Golden Gate assembly. This detail was given particular attention, as many of our constructs contain repressor genes that are constitutively expressed. We designed our constructs placing the constitutively expressed genes downstream of all inducibly expressed CDS, to reduce leakiness from our constitutively translated transcriptional units.
        <br><br>
        
        <center>
        <img class="img1" src="https://static.igem.wiki/teams/4294/wiki/design-not/ptu1-a-lacz-map.png">
        <figcaption>Figure 1.
          pTU1-A-lacZ vector (created with SnapGene)
          </figcaption>
        </center>
        <br>
        </li>

        <li>
        pTU2-A-RFP (colE1 ori) [3]: We used pTU2-A-RFP (colE1 ori) as a level 2 (as indicated by the “2” in its name)-Golden Gate acceptor vector for assembling all different versions of the mNeonGreen/LuxR expression system in receivers: Open Loop (OL or OpLo), Positive Feedback (PF), Positive Feedback with a constitutive promoter (PFc), Positive Feedback with a Repressor (PF+R) and Positive Feedback with a Repressor and a constitutive promoter (PFc+R). This vector has a chloramphenicol (Cm) resistance gene and is a medium copy plasmid; it is approximately 3kbp in size. We selected a medium-copy plasmid for the expression of our system in receiver cells, as such a plasmid would allow the production of our visual output (mNeonGreen) while maintaining it at the levels receivers are actually activated by OC6 molecules; namely, without having a potentially high-copy vector counterbalance the different amount of AHL molecules receivers are stimulated by. This medium-copy backbone would not stress out cells carrying it either.  Another reason we chose this particular vector was its marker; pTU2-A-RFP (colE1 ori) contains an RFP gene, enabling screening of colonies expressing recombinant plasmids without the need to use additional reagents (magenta colonies: transformed with the undesired parental plasmid, white colonies: transformed with a recombinant plasmid). 
        <br><br>

        <center>
        <img class="img2" src="https://static.igem.wiki/teams/4294/wiki/design-not/ptu2-a-rfp-cole1-ori-map.png">
        <figcaption>
          Figure 2.
        pTU2-A-RFP (colE1 ori) vector (created with SnapGene)

        </figcaption>
        </center>
        <br>
        </li>

        <li>
          pTU2-A-RFP (p15A ori) [3]: This is the low-copy version of pTU2-A-RFP (colE1 ori) due to the substitution of colE1 ori with the p15A origin of replication. We used pTU2-A-RFP (p15A ori) as a level 2-Golden Gate acceptor vector too for building the LuxI/TetR expression system in sender cells, alongside with the LuxI-mNeonGreen/TetR (later LuxI-sfGFP/TetR as seen in our Engineering Success tab) expression system for our RBS library characterization in the same cells. We chose a low-copy number plasmid for the expression of our system in senders, because we wanted the strength of each RBS to be reflected in the expression levels of LuxI/OC6 and not be compensated by the (potentially high) replication rate of the construct carrying it. We also utilized said vector to assemble one of the receivers’ circuits too, the OL expression system, to discover how protein expression rates might depend on plasmid copy number. Since it is identical with pTU2-A-RFP (colE1 ori) in its entire sequence except for the origin of replication, this vector also has a Cm resistance gene and measures approximately 3kbp in size. Like pTU2-A-RFP (colE1 ori), pTU2-A-RFP (p15A ori) is an RFP-dropout plasmid backbone.  
          <br><br>
  
          <center>
          <img class="img3" src="https://static.igem.wiki/teams/4294/wiki/design-not/ptu2-a-rfp-p15a-ori-map.png">
          <figcaption>
            Figure 3.
        pTU2-A-RFP (p15A ori) vector (created with SnapGene)
          </figcaption>
          </center>
          <br>
          </li>


          <li>
            pTKEI-Dest [4]: We utilized pTKEI-Dest as the acceptor vector of the construct we built to tackle the cross-talk issue among discrete senders’ subpopulations (more information about the cross-talk challenge in the Cross-talk section below). It includes a kanamycin (Kan) resistance gene and measures approximately 4.5kbp in length. It is a high-copy plasmid embodying the pBR322 origin of replication. Since this vector would have to be co-transformed with the plasmid “hosting” the final construct for senders, namely pTU2-A-RFP (p15A ori), it was specifically selected because of its origin of replication; p15A is classified as an incompatibility group B ori, therefore the second plasmid senders could carry would have to belong to either incompatibility group A or C. Taking into account the criteria agreed upon above alongside with this prerequisite, our search led us to the pTKEI-Dest vector, which, based on its ori, is categorized in incompatibility group A. As in pTU1-A-lacZ, pTKEI-Dest encompasses a lacZa gene, allowing for blue-white screening in agar plates containing x-gal and IPTG.  
            <br><br>
    
            <center>
            <img class="img4" src="https://static.igem.wiki/teams/4294/wiki/design-not/ptkei-dest-map.png">
            <figcaption>
              Figure 4.
              pTKEI-Dest vector (created with SnapGene)
            </figcaption>
            </center>
            <br>
            </li>

        <h2>Bacterial Strains</h2>
        <br>

        <b>Bacterial Strains</b>
        <br><br>
        The following bacterial strains were used in our project:
        <br><br>
        <b>DH5a:</b> <br>
        They are defined by three mutations: recA1, endA1,lacZΔM15. This storage strain is appropriate for transformation and cloning because of the aforementioned mutations.
        <br><br>
        <b>BL21 (DE3):</b><br>
        BL21 is a high level expression strain that is used for induction experiments and expression of heterologous proteins. That is why we considered this strain appropriate for our project.
        <br><br>
        <b>DH5a-Z1:s</b><br>
        DH5a-Z1 strain has the TetR repressor encoded in its genome. We chose to use this strain in our engineering cycles after our experimental data indicated that the TetR expression system in our Level 2 constructs probably did not provide the right repression dynamics.
        <br><br>

        <h2>Visual output</h2>
        <br>

        We chose a visual output for our system due to its ease of detection and wide range of available options. After a lot of reflection on integrating less utilized natural visual responses of bacterial cells to quorum sensing stimuli, such as bioluminescence (which can be measured with a luminometer) or biofilm formation (which can be observed and measured by staining bacterial cultures with Congo red [5], crystal violet [6], or safranin [7] dyes), we decided to proceed with the well-characterized method of fluorescent proteins. 
        <br><br>
        GFP (eGFP or sfGFP) was the most obvious choice; however, we were urged to consider more recently-developed fluorescent proteins. Charilaos Giannitsis, a member of iGEM Greece 2017, suggested that we use mNeonGreen [8], a fluorescent protein that presents more optimal properties compared to GFP. mNeonGreen specs are presented in the following table in comparison with the corresponding properties of eGFP [9], [10]:
        <br><br>

        <center>
          <table class="content-table">
            <thead>
              <tr>
                <th></th>
                <th>eGFP</th>
                <th>mNeonGreen</th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td>Quantum yield</td>
                <td>0.6</td>
                <td>0.8</td>
              </tr>
              <tr class="active-row">
                <td>Brightness</td>
                <td>33.54</td>
                <td>92.8</td>
              </tr>
              <tr>
                 <td>Maturation (min)</td>
                <td>25</td>
                <td>10</td>
              </tr>
              <tr class="active-row">
                <td>Lifetime (ns)</td>
                <td>2.6</td>
                <td>3.1</td>
              </tr>
              <tr>
                 <td>Photostability (t1/2)</td>
                <td>50.1</td>
                <td>158.0</td>
              </tbody>
        </table>
        <figcaption>
          Table 5.
        Comparison of eGFP vs mNeonGreen properties.
        </center>
        <br><br>

        Since our experiments required a fluorescent protein that efficiently emits the light it absorbs (measured by quantum yield), is particularly bright (indicated by brightness), has a high turnover rate (shown by maturation and lifetime), as we planned to measure receivers’ fluorescence every 30 minutes to produce our curves, resists to photobleaching (hinted by photostability) and is certified as monomeric, we proceeded with integrating mNeonGreen in our constructs. An extra asset of mNeonGreen is that, although having been recognized as a better-performing fluorescent protein than eGFP, its fluorescence can still be measured with standard GFP filters (excitation peak: 488nm, emission peak: 511nm [9], [10]); this way, anyone with a standard fluorometer that measures GFP and its variants can replicate our experiments or use our receiver constructs in their own experimental setup.
        <br><br>
        For the RBS strength characterization, superfolder GFP (sfGFP) was utilized in fusion with the first 36 nucleotides of LuxI during our engineering cycle as well.
        <br><br>

        <h2>The intermediate signal in our biological perceptron - LuxI/LuxR quorum sensing system</h2>
        <br>
        In our biological Perceptron the input pattern is converted, in a weighted manner, into an intermediate signal by the senders and the sum of this intermediate dictates the final behavior of the receivers. 
        <br><br>
        Quorum sensing (QS) systems were an obvious choice, since there are plenty of QS systems well characterized and commonly used in synthetic 
        circuits. Our deployed QS system is the LuxI/LuxR system (for more information about our final choice, please refer to 
        the <a href="https://2022.igem.wiki/athens/human-practices">Integrated Human practices</a> page). LuxI synthase catalyzes the production of 3-oxohexanoyl-homoserine lactone (OC6 for short), which belongs to a family of QS molecules called N-Acetyl-Homoserin-Lactones (AHLs). Activated Senders express LuxI and the produced OC6 molecules diffuse outside of the senders and into the receivers. Inside the receivers OC6 molecules bind to LuxR, a transcriptional regulator that forms its functional dimer after binding with OC6, which activates the Plux promoter and, therefore, the  downstream gene [11].
        <br><br>   
        
        </p>

        </section>


        <!-- senders -->
        <section class="senders">
          <a id="senders" class="anchorOffset"></a>
          <h1>The Biological Input Processors - The Senders</h1>
          <br><br>
          <p>
          A perceptron algorithm includes the recognition of input patterns and the conversion of these patterns to an intermediate signal in a weighted manner. Namely, each bit in its activated (“1”) state has a different power to determine the final result.
          <br><br>
          In our bacterial perceptron the input processors are the senders, which comprise distinct subpopulations. Each one  has a specific weight, meaning that it responds to the recognition of the input by producing a different amount of the LuxI synthase, the synthase of OC6, and, consequently, a different amount of OC6 lactone per unit of time. This graded response is achieved via the manipulation of the Ribosome Binding Site (RBS), a major determinant of translation initiation rate and, therefore, protein production.
          <br><br>
  
          <h2>The Ribosome Binding Site as a plug-and-play device to tune the weights.</h2>
          <br>
          Tuning the weights in a biological perceptron corresponds to finding ways to tweak the contribution of each activated sender to the weighted sum for a given amount of time. Possible ways to achieve that is transcriptional regulation via promoter strength manipulation, like in Ximing Li’s work [12], variation of the translation rate by changing the RBS, by different mutants of the synthase possessing different enzymatic capability, synthase degradation rate deploying degradation tags, mRNA stability and more.
          <br><br>
          In our project, translational regulation via RBS manipulation was chosen as the determining factor of the weights. RBS sequences are versatile, potentially plug-and-play devices that could be placed upstream any gene of interest to provide the desired translation rate.
          <br><br>
          <h3>Synthetic RBS Library
          </h3>
          Aiming to find the right combination of weights for our pattern recognition problem, a synthetic RBS library using the RBS Calculator [13,14] was generated by our Dry Lab. After guidance of Prof. Howard Salis, the RBS Library Calculator was used to design a degenerate RBS sequence that systematically varies the translation initiation rate. Target translation rates at logarithmically varied numbers were set and a library of synthetic RBS was generated. Then, RBS variants were clustered as “Really Strong”, “Strong”, “Medium” and “Weak” according to the table:
          <br><br>
          <center>
          <table class="content-table">
            <thead>
              <tr>
                <th>Classes </th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td>Really Strong (>150.000)</td>
              </tr>
              <tr class="active-row">
                <td>Strong (70.000-90.000)</td>
              </tr>
              <tr>
                <td>Medium (25.000-40.000)</td>
              </tr>
              
              <tr class="active-row">
                <td>Weak (<15.000)</td>
              </tr>
   
            </tbody>
        </table>
        <figcaption>
         Table 1. 
        </figcaption>
        </center>
        <br><br>
        This clustering was arbitrary aiming to create RBS groups that are more probable to differ significantly in their translation initiation rate when tested. From the library, 6 synthetic RBS sequences were randomly selected to be characterized (Table 2).<br>

        <center>
        <table class="content-table">
          <thead>
            <tr>
              <th>RBS_Name</th>
              <th>RBS_sequence [Sequence of the RBS variant]</th>
              <th>CDS Start Position (nt)</th>
              <th>ORF_number [Open Reading Frame]</th>
              <th>TIR [Translation Initiation Rate (au)]</th>
              <th>Strength</th>
            </tr>
          </thead>
          <tbody>
             <tr>
              <td>synthetic 4</td>
              <td>TTGGCAACGGGTTATGGGAGGGATGACA</td>
              <td>28</td>
              <td>1</td>
              <td>85298.89117</td>
              <td>Strong</td>
            </tr>
            <tr class="active-row">
              <td>synthetic 5</td>
              <td>TTGGCAACGGCTTATGGGAGGTATGTCA</td>
              <td>28</td>
              <td>1</td>
              <td>78686.23401</td>
              <td>Strong</td>
            </tr>
            <tr>
              <td>synthetic 10</td>
              <td>TTGGCAACGGGTTATCGGAGGGATGACA</td>
              <td>28</td>
              <td>1</td>
              <td>31126.53073</td>
              <td>Medium</td>
            </tr>
            
            <tr class="active-row">
              <td>synthetic 11</td>
              <td>TTGGCAACGGGTTATCGGAGGGATGACA</td>
              <td>28</td>
              <td>1</td>
              <td>30580.84267</td>
              <td>Medium</td>
            </tr>
            
            <tr>
              <td>synthetic 12 </td>
              <td>TTGGCAACGGGTTATGGGACGGATGTCA</td>
              <td>28</td>
              <td>1</td>
              <td>11058.30802</td>
              <td>Weak</td>
            </tr>
            
            <tr class="active-row">
              <td>synthetic 13</td>
              <td>TTGGCAACGGCTTATCGGAGGTATGCCA</td>
              <td>28</td>
              <td>1</td>
              <td>10752.07857</td>
              <td>Weak</td>
            </tr>
           
          </tbody>
      </table> 
      <figcaption>
        Table 2.
Results from the RBS calculator regarding the translation initiation rate of the selected synthetic RBS sequences.

       </figcaption>
       </center>
       <br><br>
       <h3>Reliable translation elements</h3>
       <br>
       The function of a RBS sequence relies heavily on its upstream [15] and downstream [16] genetic context, i.e. the neighboring nucleotide sequence. On the grounds that the above synthetic RBS translation rates were estimates, we wanted to ensure that we had an already tested reliable alternative to express LuxI in our arsenal. Our research led us to BCDs (Bicistronic Design) by Mutalik et.[17] Inspired by overlapping genetic elements, they include short leading peptides followed downstream by the Gene of Interest (GOI). Their design “encodes a 16-amino-acid leader peptide in a first cistron that overlaps by 1 base pair with a variable downstream coding sequence, encoding both a stop and start codon via a −1 frame shift. The leader peptide is synthesized by ribosomes that bind to an upstream SD core sequence (SD1); translation of the downstream GOI is thought to result, primarily, from SD1-directed ribosomes that recognize and reinitiate translation via a second SD site (SD2) that is encoded entirely within the coding sequence of the leader peptide” -Mutalik et. al. The intrinsic helicase activity of ribosomes arriving at the stop codon of an upstream cistron might eliminate inhibitory RNA structures that would otherwise disrupt translation initiation of the downstream GOI.
       <br><br>
       Given the fact that we are aiming to demonstrate the construction of a biological perceptron on the foundational level, we decided to include the more reliable BCDs to our characterisation, so as some of our calculations to be reproducible to other systems that might include RBS variants but with a different quorum sensing synthase and, therefore, a different genetic context near the RBS. BCDs promise fewer fluctuations when applied to different systems and are therefore closer to be considered plug-and-play devices.
       <br><br>
       
       The iGEM distribution kit includes three different BCDs, BCD1 (BBa_J428034), BCD8 (BBa_K2680561) and BCD12 (BBa_K2680529). We decided to deploy these BCDs and we ordered the BCD2 (BBa_K4294102), the strongest variant of all according to Mutalik et. al and BCD14 (BBa_K4294114).
       <br><br>
       As a preliminary evaluation of their strength we took into account both the paper’s plots that compare the strength and variability of the BCDs and the RBS calculator.
       <br><br>
       <center>
        <img class="mutalik" src="https://static.igem.wiki/teams/4294/wiki/design-not/mutalik.png">
        <figcaption>Figure 1.
          BCDs relative performance scores. Derived from Mutalik et. al “Precise and reliable gene expression via standard transcription and translation initiation elements.” Supplementary Material.
           </figcaption>
        </center>
        <br>

        <center>
          <table class="content-table">
            <thead>
              <tr>
                <th>RBS_Name</th>
                <th>RBS_sequence [Sequence of the RBS variant]</th>
                <th>CDS Start Position (nt)</th>
                <th>ORF_number [Open Reading Frame]</th>
                <th>TIR [Translation Initiation Rate (au)]</th>
                <th>Strength</th>
              </tr>
            </thead>
            <tbody>
               <tr>
                <td>BCD2</td>
                <td>GGGCCCAAGTTCACTTAAAAAGGAGATCAACAATGAAAGCAATTTTCGTACTGAAACATCTTAATCATGCTAAGGAGGTTTTCTA</td>
                <td>85</td>
                <td>1</td>
                <td>287840.97</td>
                <td>Very Strong</td>
              </tr>
              <tr class="active-row">
                <td>BCD12</td>
                <td>GGGCCCAAGTTCACTTAAAAAGGAGATCAACAATGAAAGCAATTTTCGTACTGAAACATCTTAATCATGCTGCGGAGGGTTTCTA</td>
                <td>85</td>
                <td>1</td>
                <td>12782.75</td>
                <td>Weak</td>
              </tr>
              <tr>
                <td>BCD1</td>
                <td>GGGCCCAAGTTCACTTAAAAAGGAGATCAACAATGAAAGCAATTTTCGTACTGAAACATCTTAATCATGCACAGGAGACTTTCTA</td>
                <td>85</td>
                <td>1</td>
                <td>9455.22</td>
                <td>Weak</td>
              </tr>
              
              <tr class="active-row">
                <td>BCD8</td>
                <td>GGGCCCAAGTTCACTTAAAAAGGAGATCAACAATGAAAGCAATTTTCGTACTGAAACATCTTAATCATGCATCGGACCGTTTCTA</td>
                <td>85</td>
                <td>1</td>
                <td>2190.04</td>
                <td>Weak</td>
              </tr>
              
              <tr>
                <td>BCD14 </td>
                <td>GGGCCCAAGTTCACTTAAAAAGGAGATCAACAATGAAAGCAATTTTCGTACTGAAACATCTTAATCATGCGGTGGAGGGTTTCTA</td>
                <td>85</td>
                <td>1</td>
                <td>2019.63</td>
                <td>Weak</td>
              </tr>
              
            
             
            </tbody>
        </table>

          <figcaption>
            Table 3.
            Translation initiation rates predicted by the RBS calculator. The TIR of the second SD sequence was used for this analysis, since ,according to Mutalik et. al, it is the main determinant of the GOI’s translation rate. The ribosomes from SD1 reinitiate translation from SD2.
          </figcaption>
        </center>
        <br><br>
        It is obvious that the plots and the calculations contradict each other regarding some RBS sequences and therefore wet lab characterisation was necessary. 
        <br><br>

        <h2>Senders’ main circuit
        </h2>
        <br>
        <h3>Defining the input for pattern generation - The TetR repressor system.</h3>
        <br>
        After a long way to decide the right inducer as the input, we chose the well known and characterized TetR repressor system. TetR forms a dimer and binds to tet operators in PTet promoter and blocks the assembly of the transcription machinery. PTet is a medium strength promoter that is constitutively “ON” in the absence of TetR. It contains two tet operators, one inside the core promoter sequence and the other after the -10 hexamer, leading to efficient repression. The system is efficiently induced via anhydrotetracycline (aTc), a tetracycline derivative which binds Tet R with an ~35-fold higher binding constant, being an effective inducer at very low concentrations. Additionally, its antibiotic activity is ~100-fold lower and has minimal toxic effect on E.coli cells in the concentration used for induction[18].
        <br><br>
        Main characteristics needed from the senders’ induction system were the low leakiness, dynamic range. Steepness would be an extra asset, because it would make more compatible with our perceptron ideation if senders responded in an ON/OFF fashion as well. More information about the concept of steep response curves can be found in the Receivers’ circuit design, where it is more thoroughly examined.
        <br><br>
        An extensive characterization and optimization of inducible systems in E.coli was made by Meyer et. al to create the E.coli Marionette Strains [19]. This comprehensive characterization provides useful information about repressible systems’ characteristics, such as the repressor’s binding and dimerization constants, max and minimum output (i.e. the leakiness). Among the several common systems that are characterized, we thought that the TetR repressor systems suits our design goals best. It lies at the lower end of the spectrum regarding its leakiness when it is genome encoded and in the middle when it is plasmid regulated, its dynamic range lies in the middle of the spectrum, and, most importantly, it shows the steepest response curve of all. Even though these parameters were extracted from measurements of E.coli Marionette strains, these strains derive from DH10B (Marionette-Clo) and BL21 (Marionette Pro). Therefore, we believe it was rational to chose the senders’ induction system based on those parameters.
        <br><br>      
        <center>
          <table class="content-table">
            <thead>
              <tr>
                <th>Inducer</th>
                <th>Plasmid</th>
                <th>Regulator</th>
                <th>Promoter</th>
                <th>ymax (RPU)</th>
                <th>ymin (RPUx10-3)</th>
                <th>K (μM)</th>
                <th>n</th>
                <th>Dynamic range</th>
                <th></th>
              </tr>
            </thead>
            <tbody>
               <tr>
                <td>aTc</td>
                <td>pAJM.011</td>
                <td>TetR</td>
                <td>PTet*</td>
                <td>2.4</td>
                <td>4.9</td>
                <td>0.013</td>
                <td>3.8</td>
                <td>500</td>
                <td>Plasmid Encoded Regulator</td>
              </tr>
              <tr class="active-row">
               <td>aTc</td>
                <td>pAJM.717</td>
                <td>TetR</td>
                <td>PTet*</td>
                <td>3.2</td>
                <td>3.6</td>
                <td>0.012</td>
                <td>4.4</td>
                <td>890</td>
                <td>Genome Encoded Regulator</td>
              </tr>
              
            
             
            </tbody>
        </table>
        <figcaption>
          Table 4.
          Parameters from the E.coli Marionette Strains characterization. 
        </figcaption>
      </center>
        <br>
        <center>
          <img class="s-2a" src="https://static.igem.wiki/teams/4294/wiki/design-not/s-2a.png">
          <img class="s-2b" src="https://static.igem.wiki/teams/4294/wiki/design-not/s-2b.png">
          <img class="s-2c" src="https://static.igem.wiki/teams/4294/wiki/design-not/s-2c.png
          ">
          <figcaption>Figure 2.
            Senders’ Circuit Overview. TetR is expressed by a constitutive promoter (BBa_J23106 and PlacI promoters were tested, please refer to Engineering Success for more regarding these trials) and the BBa_B0034 RBS. It binds to specific operators in the PTet promoter and blocks RNA polymerase binding and, thus, transcription.  
            The circuit at its “OFF” state (“0”)
            The circuit at its “ON” state (“1”)            
             </figcaption>
          </center>
          <br>

          <center>
            <img class="s-f3" src="https://static.igem.wiki/teams/4294/wiki/design-not/senders-f3.png">
            <figcaption>Figure 3. (pTU1(S10)-luxI Map.png)
              Plasmid map of a Level 1 construct with one of the synthetic RBS controlling LuxI translation.                        
               </figcaption>
            </center>
            <br>

            <center>
              <img class="s-f4" src="https://static.igem.wiki/teams/4294/wiki/design-not/senders-f4.png">
              <figcaption>Figure 4 (pSend(S10)-luxI (p15A ori) Map)
                Plasmid map of a Level 2 construct.                                     
                 </figcaption>
              </center>
              <br><br>
          
          <h3>Final TetR regulation circuit</h3>
          As it is explained in more detail in the <a href="https://2022.igem.wiki/athens/engineering">Engineering Success</a> page, the constructed plasmid-encoded TetR regulator systems did not show a normal induction response. As a result, DH5a-z1 cells, which encode TetR in their genome, were utilized for our perceptron system. These cells were transformed with pTU1 high copy number plasmid containing the Level 1 LuxI TU constructs. This was an important compensation on the grounds that pTU1 plasmid offers resistance to a different antibiotic than pTU2, making experiments with mixed senders and receivers populations more complex.Our initial thoughts about the potential shadowing of the relative RBS strength differences due to the high copy number vector variations were disproved. The selected RBS sequences provided clearly different translation rates, as it can be found in the <a href="https://2022.igem.wiki/athens/results">Results</a> page.
          <br><br>

          <h2>Characterisation of the Ribosome Binding Sites</h2>
          <br>
          <h3>Characterisation of RBS strength utilizing a LuxI-mNeonGreen bicistronic design with a translation Termination-Reinitiation (TeRe) strategy.</h3>
          In order to characterize the strength of our senders’ RBS, we had to measure the LuxI each subpopulation produced in the scale of time. 
          <br><br>
          From a very early stage of our designing sessions, we agreed to steer clear of complex and time-consuming measurement methods, such as column-based purification procedures, or processes that required specialized equipment we did not have access to, e.g. Western blotting. Nevertheless, we added a C-terminal hexa-HisTag in our senders’ final constructs, to perform protein analysis through Western blotting, just in case our main scheme did not work as planned.
          <br><br>
          After excluding these strategies, we decided to adopt the chimeric gene approach by coupling the gene coding for LuxI with a gene producing a visible readout, namely a fluorescent protein. We ruled out placing a linker between the two CDS, as this would require additional research, and incorporated an alternative linking mechanism instead; to combine LuxI’s synthesis with the production of the fluorescent protein, we deployed a phenomenon called Termination-Reinitiation (or TeRe) observed during translation [20]. TeRe occurs when two neighboring genes have overlapping start and stop codons. More specifically, once translation of the upstream gene is terminated, the 30S ribosomal subunit lingers in the area of the stop codon; if it immediately encounters a new start codon, it re-recruits the 50S subunit and reinitiates translation, this time, of the downstream gene, as shown in the figure below. This way, the downstream gene is translated at the same rate as the upstream gene.
          <br><br>
          <center>
            <img class="s-f5" src="https://static.igem.wiki/teams/4294/wiki/design-not/tere-2.png">
            <figcaption>Figure 5
              Termination-Reinitiation translation mechanism. Figure derived from Huber et. al., 2019 (modified)                                     
               </figcaption>
            </center>         
          
          <br><br>
          Based on this mechanism, we enriched the TAA stop codon of the luxI gene with an extra TG, to convert its terminal A into the initial A of the fluorescent protein’s gene start codon, resulting in the coupling sequence 5’-TAATG-3’ for the characterization of senders’ RBS.
          <br><br>
          <center>
            <img class="s-f6" src="https://static.igem.wiki/teams/4294/wiki/design-not/overlap-2.png">
            <figcaption>Figure 6. Overlapping stop (luxI) and start codon (mNeonGreen) forming the sequence 5’-TAATG-3’                             
               </figcaption>
            </center> 
          
            <br><br>
          
          We opted for mNeonGreen as the fluorescent protein we would utilize for our TeRe-based measurement system, as we had already researched a lot about it -since we had already selected it as a visible readout for our entire perceptron-resembling device (more information about this aspect of our project below)- and believed it would also fulfil the requirements for our senders’ characterization experiments. According to the TeRe mechanism and our predictions, mNeonGreen fluorescence measurements would constitute a relatively accurate indicator of the amount of LuxI synthesized.
          <br><br>
          <center>
            <img class="s-f71" src="https://static.igem.wiki/teams/4294/wiki/design-not/4a-1.png">
            <img class="s-f72" src="https://static.igem.wiki/teams/4294/wiki/design-not/4b-1.png">
            <img class="s-f73" src="https://static.igem.wiki/teams/4294/wiki/design-not/4c-1.png">
            <figcaption>Figure 7.
              Overview of the translational coupling circuit for the characterization of the LuxI expression.
              Circuit at its off state (“0”)
              Circuit at its on state (“1”)                             
               </figcaption>
            </center> 
          
            <br><br>
            <center>
              <img class="s-f8" src="https://static.igem.wiki/teams/4294/wiki/design-not/senders-f8.png">

              <figcaption>Figure 8 (pTU1(S10)-luxI_mNeonGreen Map.png)
                Plasmid map of the Level 1 construct with the TeRe design.
                                           
                 </figcaption>
              </center> 
              <br><br>
              <center>
                <img class="s-f9" src="https://static.igem.wiki/teams/4294/wiki/design-not/senders-f9.png">
  
                <figcaption>Figure 9 (pSend(S10)-luxI_mNeonGreen (p15A ori) Map)
                  Plasmid map of the Level 2 construct with the TeRe design.
                                             
                   </figcaption>
                </center> 
                <br><br>
          
          
          
          Unfortunately, the circuit did not work as expected and it had to be replaced with a different strategy. 
                   
          <br><br>
          <h3>Characterisation of the RBS strength using the 36nt luxI - sfGFP fusion.</h3>
          <br>
          As it was stated, the function of the Ribosome Binding Site is often unpredictable, since it depends on its interaction with the upstream and downstream genetic context. Therefore, a characterisation of our RBS strength regarding the translation of LuxI would probably not be accurate, if a fluorescent reporter was simply placed after them, since the genetic context (i.e. the nucleotide sequence and so its interaction with the RBS) would differ.
          <br><br>
          To prove the reliability and context independence of the BCDs, Mutalik et. al engineered several fusions of the first 36 nucleotides of commonly used regulators and enzymes with superfolder Green Fluorescent Protein (sfGFP). These first 36 nucleotides are sufficient enough to introduce a similar genetic context of the whole sequence they originated to the RBS [21]. sfGFP is an engineered version of the green fluorescent protein that can be fused with poorly folded peptides without misfolding and losing its functionality [22]., making these 36nt fusions a simple and probably reliable way to characterize translation rates provided by the same RBS for different coding sequences. A similar strategy was recently deployed to determine the effect of the spacer sequence between a RBS and the start codon in the context dependency of the RBS [23].
          <br><br>
          Based on the above information, we built a fusion of the first 36 nucleotides of the LuxI synthase with sfGFP to characterize the relative strength of our deployed RBS in the context of the LuxI coding sequence. sfGFP maintained its fluorescent properties and allowed us to conduct the characterisation. The characterisation results can be found in the Results section.
          <br><br>
          <center>
          <img class="s-f101" src="https://static.igem.wiki/teams/4294/wiki/design-not/s-5a-1.png">
          <img class="s-f102" src="https://static.igem.wiki/teams/4294/wiki/design-not/s-5b-1.png">
          <img class="s-f103" src="https://static.igem.wiki/teams/4294/wiki/design-not/s-5c-1.png">
          <figcaption>Figure 10.
            Final RBS characterisation circuit.
            FInal RBS characterization circuit overview
            Circuit at its off state (“0”)
            Circuit at its on state (“1”)
            
                                        
              </figcaption>
          </center> 
          <br><br>
          <center>
            <img class="s-f11" src="https://static.igem.wiki/teams/4294/wiki/design-not/senders-f11.png">

            <figcaption>Figure 11 (pTU1(S10)-(36nt)luxI_sfGFP Map.png)
              Plasmid map of a Level 1 plasmid with the 36ntLuxI-sfGFP fusion CDS.         
                                         
               </figcaption>
            </center> 
            <br><br>
            <center>
              <img class="s-f12" src="https://static.igem.wiki/teams/4294/wiki/design-not/senders-f12.png">

              <figcaption>Figure 12 (pSend(S10)-(36nt)luxI_sfGFP (p15A ori) Map)
                Plasmid map of a Level 2 plasmid with the 36ntLuxI-sfGFP fusion TU. TetR is constitutively expressed by the promoter BBa_J23106.
                
                                           
                 </figcaption>
              </center> 
              <br><br>
              <center>
                <img class="s-f13" src="https://static.igem.wiki/teams/4294/wiki/design-not/senders-f13.png">
  
                <figcaption>Figure 13 (pSend(S10)-(36nt)luxI_sfGFP (pLacI_Stet1) (p15A ori) Map)
                  Plasmid map of a Level 2 plasmid with the 36ntLuxI-sfGFP fusion TU. TetR is constitutively expressed by the promoter PlacI. This is an alteration of the original circuit that was designed during troubleshooting to explore whether a change in TetR dynamics would improve the system's response to aTc.                
                                             
                   </figcaption>
                </center> 
                <br><br>
                The 36nt luxI-sfGFP fusion was proved to be a successful part in the characterization to determine the relative strength of the RBS variants in the context of the LuxI CDS. Please, refer to the Engineering Success tab and Results tab for more.  
                <br><br>     
          <h2>Cross-talk </h2> 
          <br>

          After deciding that the input patterns will be generated by a single chemical inducer, a problem that emerged was how to deal with cross-talk among senders. For this aspect of our project, cross-talk is defined as the unintended stimulation of supposedly inactivated sender cells by residual inducer molecules when they are mixed with activated senders to produce a specific pattern based on the strengths of their RBS. 
          <br><br>
          1. We came up with various plans to tackle this situation:<br><br>
          2. One idea revolved around initiating the production of a tTa transactivator upon mixing senders’ subpopulations. The expression of a molecule that would with TetR for the binding of aTc could contribute to sequestering the inducer, thus keeping uninduced senders inactivated [24].<br>
          3. Another idea pertained to designing a CRISPR-Cas9 mediated mechanism that would knock out the luxI gene in inactivated senders upon mixing them with their induced counterparts to prevent cross-talk phenomena.<br>
          4. Similarly to the previous thought, a recombinase could be induced to be produced when mixing sender cells, to misplace the luxI gene, thus deactivating it.<br>
          5. Certainly, a simple wash of the different sender cultures’ supernatants could also resolve the issue by removing the inducer altogether.<br>
          6. We even thought about harnessing the photosensitive nature of our inducer, aTc, and exposing it to blue light to photobleach it before mixing different senders’ subpopulations, so that it could not activate sender cells anymore [25].<br><br>

          All potential solutions mentioned above present a number of challenges, such as leading us to a vicious circle by perpetuating cross-talk issues or requiring specialized optogenetics equipment which we could not afford, so we eventually culminated in a completely different approach. During our investigation of potential solutions and after consulting with our advisors, we encountered a paper analyzing the tetracycline destructase family, a group of enzymes that degrade tetracycline and its derivatives, such as aTc [26]. We decided to incorporate such an enzyme, called TetX, in our cross-talk tackling strategy by co-transforming it in senders’ subpopulations we wish to keep uninduced. As already mentioned, we designed this construct with pTKEI-Dest as its backbone, because its ori (incompatibility group A) would not interfere with the replication of the senders’ final construct (incompatibility group B).
          <br><br>
          <center>
          <img class="s-f14" src="https://static.igem.wiki/teams/4294/wiki/design-not/psend-tetx-map.png">

          <figcaption> Figure 14.
            pSend-TetX cross-talk construct (created with SnapGene)                
                                        
          </figcaption>
          </center> 
         

          Having concluded the design of our cross-talk system, the parameters we had to take into account during every induction experiment with the combination of induced and uninduced can be summarized by the following three points:<br><br>
          1. All sender’ subpopulations that would have to remain inactivated to create the desired pattern should be co-transformed with pSend-TetX as described above.<br>
          2. All sender’ subpopulations that would have to be induced, but still be co-cultured with inactivated senders, should be co-transformed with a second plasmid as well, to acquire resistance to Kan too (the antibiotic of pSend-TetX); to achieve that, we selected pJUMP27-1A from the iGEM DNA Distribution kit (plate 1, position 4E), which is Kan resistant, to be our “dummy” senders’ plasmid based on its origin of replication (senders’ main construct belongs to incompatibility group B, while pJUMP27-1A belongs to incompatibility group C).<br>
          3. Similarly to induced senders, receivers being co-cultured with senders in order to extract the desired pattern should also have resistance to Kan; since receivers’ main constructs are classified as either incompatibility group B plasmids (p15A ori) or incompatibility group A plasmids (colE1 ori), pJUMP27-1A (pSC101 ori) could be utilized as the “dummy” plasmid for receivers too.<br><br>

          </p>
  
          </section>


          <!-- Receivers -->
        <section class="receivers">
          <a id="receivers" class="anchorOffset"></a>
          <h1>The Biological Activation Function - The Receivers
          </h1>
          <br><br>
          <p>

            In a common perceptron, the input patterns are converted into a weighted sum which is then processed by an activation function. This is the point of the classification of the initial pattern into a specific class. If the weighted sum surpasses a set threshold, the “neuron” gets activated and gives an output “1”. Otherwise, the output is “0”.
            <br><br>
            In order to program the receiver cells to function similarly to the activation function, a circuit that imitates it is needed. Namely, a circuit that performs like an ON/OFF switch.
            <br><br>
            Often inspired by computer science data processing modules, synthetic genetic and metabolic circuits have been engineered to function like logic-gates, oscillators and more[27]. One of the main principles of Synthetic Biology is the use of characterized genetic elements, i.e. parts, that can be combined to perform complex and desired tasks. In the relatively simple context of a system that responds to the concentration of an input by an output expression, the relation of the output signal to inducer concentration forms a sigmoid curve if plotted on a Xy axis. These plots are the “response curves” and the mathematical equation that describes this response to the input is called a “transfer function”. Such systems often need the suitable tuning to function as expected and be suitable for combination with other systems and synthetic biologists have established various possible ways to tune a systems’ response curve in terms of dynamic range, leakiness, steepness and more[28]. Before the receivers’ genetic circuits are described, it is crucial to define in a nutshell some important terms:
            <br><br>
            1. <b>Dynamic Range:</b> The difference in output between the “OFF” and the saturated “ON” state.<br>
            2. <b>Leakiness:</b> The output production in the “OFF” state. <br>
            3. <b>Steepness:</b> A steep response curve is described by a large change in output within a narrow input concentration range; small changes in input result in large changes in output. Such switch-like biochemical circuits are often referred to as ultrasensitive or bistable.<br>
            <br><br>
            The perceptron’s activation function is a switch-like function that gives a positive (“1”) output if the weighted sum is surpassed. It becomes, therefore, clear that if the activation function is to be imitated in a biological system, an ultrasensitive circuit, i.e. with a steep activation function, that responds to the quorum sensing signal summation is needed.
            <br><br>

            <center>
              <img class="s-f15" src="https://static.igem.wiki/teams/4294/wiki/design-not/design-page-gifs.gif">
    
              <figcaption> Figure 1.
                Response curves with different steepness. The typical response curve of a simple inducer-output system is a sigmoid curve and its steepness depends on the intrinsic properties of its components (e.g. the repressor’s binding cooperativity). With specific circuit architectures, steeper, switch-like response curves can be engineered. Such circuits are described as “ultrasensitive” or “bistable”.
                              
                                            
              </figcaption>
              </center> 
  
          <h2>A step-by-step approach to achieve ultrasensitivity
          </h2>
          <br>
          A number of possible ways at different levels of gene expression and protein function to build a bistable switch exist[28]. A well known example is the construction of the first toggle switch in E.coli; a circuit that responds in an ON/OFF fashion effectively[29]. Such bistable circuits are based on mutual inhibition relationships between transcriptional repressors in the circuit.  At the level of transcription, binding cooperativity of transcription factors and the number and topology of operators has been proven to correlate with a response steepness. Cooperative binding of multiple transcription factors to the same promoter or a biochemical process mimicking this effect may promote ultrasensitivity as well [28]. Moreover, circuit architectures with proven enhanced steepness are transcriptional positive feedbacks [30] and transcriptional cascades [31]. Further strategies include a sequestering molecule that binds a circuit component and prevents its effect. Sequestration has been achieved using sRNAs [32] that bind to mRNA, proteins that bind to transcription factors[33] or translation machinery components [34] and decoy DNA operators [35,36] that titrate the transcription factor away from the output promoter. The above approaches are often coupled in such a way that their respective effects generate a greater impact on the final response [34].

          <br><br>
          Irrespective of the existing literature, every circuit that is built requires testing and tuning. On the grounds of our narrow time window to perform extensive characterisation experiments, we followed a step-by-step approach regarding the receivers’ genetic circuits. Starting from a simple open loop circuit, which was necessary to obtain vital parameters of our parts (refer to <a href="https://2022.igem.wiki/athens/model">Model</a> section), two variations of a positive feedback and two variations of a coherent feed forward loop coupled with a positive feedback at its first step were built. Progressively introducing new components to our circuits enables troubleshooting and better interpretation of the results.
          <br><br>

          <h3>Open Loop Circuit (OL or OpLo)
          </h3><br>
          This is a circuit of the LuxR quorum sensing regulator at its simplest form. LuxR is constitutively expressed. When concentration of OC6 increases, it binds with LuxR, LuxR forms stable dimmers and binds to its binding site (lux box), upstream of the -35 region of the Plux promoter, activating the transcription of the downstream output.

          <center>
            <img class="r-2a" src="https://static.igem.wiki/teams/4294/wiki/design-not/r-2a.png">
            <img class="r-2b" src="https://static.igem.wiki/teams/4294/wiki/design-not/r-2b.png">
            <img class="r-2c" src="https://static.igem.wiki/teams/4294/wiki/design-not/r-2c.png">
            <img class="r-2d" src="https://static.igem.wiki/teams/4294/wiki/design-not/r-2d.png">
  
            <figcaption> Figure 2.
              Flowchart of the Open Loop circuit<br>
              OL genetic circuit overview. LuxR is constitutively expressed under the control of the constitutive BBa_J23105 promoter and BBa_B0034 RBS. <br>Output (mNeonGreen) production is controlled by the Plux inducible promoter and BBa_B0034 RBS.<br>
              OL at is “OFF” state (‘“0”)<br>
              OL at its “ON” state (“1”)   <br>                
            </figcaption>
            </center> 

          <br> <br> 
          We constructed this circuit in both a medium (ColE1 origin of replication) and a low (p15a origin of replication) copy number pTU2 plasmid vector so as to choose the copy number that provides the most suitable dynamic range, leakiness and steepness. The medium copy number plasmid was chosen and used for the rest of the constructs (see <a href="https://2022.igem.wiki/athens/results">Results</a> section for more details) and the parameter estimation for our <a href="https://2022.igem.wiki/athens/model">Model</a>.

          <center>
            <img class="r-3a" src="https://static.igem.wiki/teams/4294/wiki/design-not/receivers-f3a.png">
            <img class="r-3b" src="https://static.igem.wiki/teams/4294/wiki/design-not/receivers-f3b.png">  
            <figcaption> Figure 3.
             a. pTU2 (ColE1) vector with OL circuit. 
             b. pTU2 (p15a) vector with the OL circuit.                        
            </figcaption>
            </center> 

          <h3>Positive Feedback Circuit (PF and PFc).
          </h3><br>
          One already tested and widely utilized method to engineer a bistable circuit, are transcriptional Positive Feedbacks. In such circuits, an activator of an output induces its own production as well after induction with its respective ligand. A similar circuit was built by Li et. al [12] to achieve a steeper activation function of the CinR quorum sensing activator system. 
          <br><br>
          In our case, Plux was placed upstream of the LuxR coding sequence and the output coding sequence. Therefore, LuxR activates its own production after binding with OC6 and establishes a positive feedback.
          <br><br>
          Two possible topologies exist; LuxR being controlled only by the Plux promoter or providing an extra TU for constitutive LuxR production.
          <br><br>
          In the former circuit topology, Plux leakiness provides a low basal LuxR concentration in the cell. This concentration is sufficient for the LuxR dimer formation after induction with OC6. The activated LuxR dimers promote the expression more LuxR resulting in a positive feedback. 
          <br><br>
          The latter provides a higher basal LuxR concentration inside the cell. These circuits were both constructed and tested in order to find the most suitable regulator and circuit dynamics for our system.
          <br><br>
          Unfortunately neither topology resulted in the desired results. For more information and engineering considerations, please refer to Results and Engineering Success tabs respectively.
          <br><br>

          <center>
            <img class="r-4" src="https://static.igem.wiki/teams/4294/wiki/design-not/4a.png">
            <img class="r-4" src="https://static.igem.wiki/teams/4294/wiki/design-not/4b.png">  
            <img class="r-4" src="https://static.igem.wiki/teams/4294/wiki/design-not/4c.png">
            <img class="r-4" src="https://static.igem.wiki/teams/4294/wiki/design-not/4d.png"> 
            <img class="r-4" src="https://static.igem.wiki/teams/4294/wiki/design-not/4e.png">
            <img class="r-4" src="https://static.igem.wiki/teams/4294/wiki/design-not/4f.png"> 
            <img class="r-4" src="https://static.igem.wiki/teams/4294/wiki/design-not/4g.png">
            <figcaption> Figure 4.
              a.Flow chart of a Positive Feedback circuit<br>
              b.Flowchart of a positive feedback circuit.<br>
              c.PF (Positive Feedback) genetic circuit overview. LuxR is placed downstream of the promoter Plux inducible promoter and BBa_B0034 RBS. The d.leakiness of Plux promoter provides a basal LuxR concentration in the cell that will be activated by OC6 and promote the production of more LuxR and of the output, resulting in a positive feedback loop. Output (mNeonGreen) is placed downstream Plux promoter and BBa_B0034 RBS.<br>
              
              e.PF at is “OFF” state (‘“0”)
              PF at its “ON” state (“1”)
              Fc (Positive Feedback with constitutive LuxR production) genetic circuit overview. Two LuxR TU exist in this circuit. One expresses LuxR constitutively and provides a higher LuxR concentration in the cell. The other LuxR TU (with the Plux promoter) is responsible for the generation of the positive feedback after induction with OC6. Output (mNeonGreen) is placed downstream of the Plux promoter and BBa_B0034 RBS.<br>
              f.PFc at its “OFF” state (“0”)<br>
              g.PFc at its “ON” state (“1”)<br>
                                     
            </figcaption>
            </center> 

            <center>
              <img class="r-5" src="https://static.igem.wiki/teams/4294/wiki/design-not/receivers-f5a.png">
              <img class="r-5" src="https://static.igem.wiki/teams/4294/wiki/design-not/receivers-f5b.png">  
              <figcaption> Figure 5.

               a. PF circuit in the pTU2 (ColE1) plasmid vector. <br>
               b. PFc circuit in the pTU2 (ColE1) plasmid vector. <br>
                
                                       
              </figcaption>
              </center> 

              <h2>A coherent feed forward loop coupled with a Positive Feedback (PFR, PFcR)</h2><br>
              For transcriptionally regulated gene expression, one approach to achieve a switch-like response is an engineered promoter onto which multiple transcription regulators bind cooperatively or a biochemical mimic of this effect.
              <br><br>
              This led us to the design of the part BBa_K4294301, a hybrid promoter which includes a LuxR activator binding domain (luxbox) upstream the -35 hexamer and a PhlF operator (phlO) that spans from the core sequence and the -10 hexamer to the proximal promoter sequence.
              <br><br>
              <center>
                <img class="r-6" src="https://static.igem.wiki/teams/4294/wiki/design-not/6-2.png">
                <figcaption> Figure 6.
                  Plux-phlO hybrid promoter.                                      
                </figcaption>
                </center> 
                <br><br>
                PhlF is under the transcriptional control of a LuxR repressible promoter (Plux_rep, BBa_J107103). However, in this promoter the LuxR binding domain is not placed upstream of the -35 hexamer, but inside the core sequence between the -35 and -10 regions. Due to this localization change, the effect of the active LuxR dimers is reversed; instead of activating the production of PhlF, they repress it [37]
                <br><br>
                Even though LuxR and PhlF do not bind cooperatively to their respective binding sites, their dynamics are indirectly connected. PhlF was chosen since it is an effective repressor with a low association constant K, indicating its ability to form functional dimers in lower concentrations (according to the Ecoli Marionette strains characterization [19]). Therefore, in “intermediate” OC6 concentrations, it could possibly still reinforce repression to the output irrespective of its lower production rate (due to the effect of LuxR on the Plux_rep promoter) and the activation effect of LuxR to the output’s promoter. LuxR establishes a positive feedback loop to propagate its effect and both positive feedback topologies (with and without constitutive LuxR expression) were designed and tested. A very efficient ssrA [38] degradation tag (NDENYALAA) was added to PhlF to ensure its rapid degradation after maximum repression of the phlF gene by LuxR and to avoid PhlF concentrations that might lead to a complete circuit block in every inducer concentration. The degradation tag was empirically selected and further analysis of different degradation tags will be needed for a potential optimization of the circuit’s response.
                <br><br>
                Note: These circuit architectures are referred to as Positive Feedback + Repressor (PF+R) and Positive Feedback with constitutive LuxR expression + Repressor (PFc+R) in other pages as well.
                <br><br>
                <center>
                  <img class="r-7" src="https://static.igem.wiki/teams/4294/wiki/design-not/7a.png">
                  <img class="r-7" src="https://static.igem.wiki/teams/4294/wiki/design-not/7b.png">  
                  <img class="r-7" src="https://static.igem.wiki/teams/4294/wiki/design-not/7c.png">
                  <img class="r-7" src="https://static.igem.wiki/teams/4294/wiki/design-not/7d.png">  
                  <img class="r-7" src="https://static.igem.wiki/teams/4294/wiki/design-not/7e.png">
                  <img class="r-7" src="https://static.igem.wiki/teams/4294/wiki/design-not/7f.png">  
                  <img class="r-7" src="https://static.igem.wiki/teams/4294/wiki/design-not/7g.png">
                  <figcaption> Figure 7.
                    a. Flow chart of PFR (Positive Feedback + Repressor) and PFcR (Positive Feedback with constitutive LuxR production + Repressor) circuits. This topology resembles the one of a coherent feedforward loop with an added positive involving the first step.
                    b. PFR genetic circuit overview. The output is controlled by the hybrid promoter BBa_K4294301, which is activated by LuxR and repressed by PhlF. LuxR is placed downstream of the Plux promoter, establishing a positive feedback after induction with OC6 as previously described. LuxR also binds to a Plux repressible promoter (BBa_J107103), which controls the production of PhlF, blocking PhlF expression after induction with OC6. Every coding sequence is placed downstream a BBa_B0034 RBS. PhlF has a high efficiency NDENYALAA degradation tag.
                    c. PFR circuit at its “OFF” state (“0”)
                    d. PFR circuit at its “ON” state (“1”)
                    e. PFcR genetic circuit overview. This circuit is similar to the PFR circuit, but with an added constitutive expression of LuxR, which provides a higher basal LuxR concentration in the cell.
                    f. PFcR at is “OFF” state (‘“0”)
                    g. PFcR at its “ON” state (“1”)                
                                           
                  </figcaption>
                  </center>
                  <br><br>
                <center>
                  <img class="r-8" src="https://static.igem.wiki/teams/4294/wiki/design-not/receivers-f8a.png">
                  <img class="r-8" src="https://static.igem.wiki/teams/4294/wiki/design-not/receivers-f8b.png">  
                  <figcaption> Figure 8.
                    PFR circuit in the pTU2 (ColE1) plasmid vector. 
                    PFcR circuit in the pTU2 (ColE1) plasmid vector. 
                  </figcaption>
                  </center>  
                  After induction with OC6, PFR circuit response showed a significant difference in steepness in comparison with the original open loop circuit. For more information, please refer to the <a href="https://2022.igem.wiki/athens/model">Results</a> page and the <a href="https://2022.igem.wiki/athens/model">Model</a> page for the circuit’s model parameters.

          </p>
  
          </section>


            <!-- References --> 
                <section class="collapsible">
                    <a id="references" class="anchorOffset"></a>
                    <input type="checkbox" id="collapsible-head">
                        <label for ="collapsible-head">References</label>    
                        <div class="collapsible-content">
                          <cite>
                            <a href="https://blog.addgene.org/plasmid-101-origin-of-replication">
                              1. https://blog.addgene.org/plasmid-101-origin-of-replication [Accessed in June 2022]                    </a>
                        </cite>
        
                        <cite>
                            <a href="https://www.addgene.org/">
                                2. https://www.addgene.org/ [Accessed in June 2022]
        
                            </a>
                        </cite>
        
                        <cite>
                            <a href="https://pubs.acs.org/doi/10.1021/acssynbio.6b00031">
                                3.Moore, S.J., Lai, H.-E., Kelwick, R.J.R., Chee, S.M., Bell, D.J., Polizzi, K.M., Freemont, P.S., 2016. EcoFlex: A Multifunctional MoClo Kit for E. coli Synthetic Biology. ACS Synth. Biol. 5, 1059–1069. https://doi.org/10.1021/acssynbio.6b00031 [Accessed in June 2022]
        
                            </a>
                        </cite>
        
                        <cite>
                            <a href="">
                                4.
                            </a>
                        </cite>
        
                        <cite>
                            <a href="https://www.sciencedirect.com/science/article/abs/pii/S0167701200001226?via%3Dihub">
                                5. Stepanović, S., Vuković, D., Dakić, I., Savić, B., Švabić-Vlahović, M., 2000. A modified microtiter-plate test for quantification of staphylococcal biofilm formation. Journal of Microbiological Methods 40, 175–179. https://doi.org/10.1016/S0167-7012(00)00122-6 [Accessed in May 2022]
                            </a>
                        </cite>
        
        
        
         <cite>
          <a href="https://www.sciencedirect.com/science/article/abs/pii/S0167701217302117?via%3Dihub">
                                6.Ommen, P., Zobek, N., Meyer, R.L., 2017. Quantification of biofilm biomass by staining: Non-toxic safranin can replace the popular crystal violet. Journal of Microbiological Methods 141, 87–89. https://doi.org/10.1016/j.mimet.2017.08.003 [Accessed in May 2022]
        
                            </a>
                        </cite>
        
         <cite>
         <a href="https://www.nature.com/articles/nmeth.2413">
                                7.Shaner, N.C., Lambert, G.G., Chammas, A., Ni, Y., Cranfill, P.J., Baird, M.A., Sell, B.R., Allen, J.R., Day, R.N., Israelsson, M., Davidson, M.W., Wang, J., 2013. A bright monomeric green fluorescent protein derived from Branchiostoma lanceolatum. Nat Methods 10, 407–409. https://doi.org/10.1038/nmeth.2413 [Accessed in June 2022]
        
                            </a>
                        </cite>
        
         <cite>
        <a href="https://www.fpbase.org/protein/mneongreen/">
                                8.https://www.fpbase.org/protein/mneongreen/ [Accessed in June 2022]
                            </a>
                        </cite>
        
         <cite>
        <a href="https://www.fpbase.org/protein/egfp/">
                                9.https://www.fpbase.org/protein/egfp/ [Accessed in June 2022]
                            </a>
                        </cite>
        
         <cite>
        <a href="https://www.ptglab.com/news/blog/mneongreen-vs-gfp/">
                                10.https://www.ptglab.com/news/blog/mneongreen-vs-gfp/ [Accessed in August 2022]
                            </a>
                        </cite>
        
         <cite>
        <a href="https://pubmed.ncbi.nlm.nih.gov/8288518/">
                                11.Fuqua WC, Winans SC, Greenberg EP. Quorum sensing in bacteria: the LuxR-LuxI family of cell density-responsive transcriptional regulators. J Bacteriol. 1994 Jan;176(2):269-75. doi: 10.1128/jb.176.2.269-275.1994. PMID: 8288518; PMCID: PMC205046.
                            </a>
                        </cite>
        <cite>
        <a href="https://www.nature.com/articles/s41467-021-23336-0">
                                12.Li X, Rizik L, Kravchik V, Khoury M, Korin N, Daniel R. Synthetic neural-like computing in microbial consortia for pattern recognition. Nat Commun. 2021 May 25;12(1):3139. doi: 10.1038/s41467-021-23336-0. PMID: 34035266; PMCID: PMC8149857.
                            </a>
                        </cite>
        
        <cite>
        <a href="https://pubs.acs.org/doi/abs/10.1021/acssynbio.0c00394">
                                13.Reis AC, Salis HM. An Automated Model Test System for Systematic Development and Improvement of Gene Expression Models. ACS Synth Biol. 2020 Nov 20;9(11):3145-3156. doi: 10.1021/acssynbio.0c00394. Epub 2020 Oct 15. PMID: 33054181.
                            </a>
                        </cite>
        
        <cite>
        <a href="https://pubmed.ncbi.nlm.nih.gov/19801975/">
                                14.Salis HM, Mirsky EA, Voigt CA. Automated design of synthetic ribosome binding sites to control protein expression. Nat Biotechnol. 2009 Oct;27(10):946-50. doi: 10.1038/nbt.1568. Epub 2009 Oct 4. PMID: 19801975; PMCID: PMC2782888.
                            </a>
                        </cite>
        
        <cite>
        <a href="https://pubmed.ncbi.nlm.nih.gov/23034349/">
                                15.Lou C, Stanton B, Chen YJ, Munsky B, Voigt CA. Ribozyme-based insulator parts buffer synthetic circuits from genetic context. Nat Biotechnol. 2012 Nov;30(11):1137-42. doi: 10.1038/nbt.2401. Epub 2012 Oct 3. PMID: 23034349; PMCID: PMC3914141.
                            </a>
                        </cite>
        
        <cite>
        <a href="https://pubmed.ncbi.nlm.nih.gov/23474467//">
                                16.Mutalik VK, Guimaraes JC, Cambray G, Mai QA, Christoffersen MJ, Martin L, Yu A, Lam C, Rodriguez C, Bennett G, Keasling JD, Endy D, Arkin AP. Quantitative estimation of activity and quality for collections of functional genetic elements. Nat Methods. 2013 Apr;10(4):347-53. doi: 10.1038/nmeth.2403. Epub 2013 Mar 10. PMID: 23474467.
                            </a>
                        </cite>
        
        <cite>
        <a href="https://pubmed.ncbi.nlm.nih.gov/23474465/">
                                17.Mutalik VK, Guimaraes JC, Cambray G, Lam C, Christoffersen MJ, Mai QA, Tran AB, Paull M, Keasling JD, Arkin AP, Endy D. Precise and reliable gene expression via standard transcription and translation initiation elements. Nat Methods. 2013 Apr;10(4):354-60. doi: 10.1038/nmeth.2404. Epub 2013 Mar 10. PMID: 23474465.
                            </a>
                        </cite>
        
        <cite>
        <a href="https://pubmed.ncbi.nlm.nih.gov/9092630/">
                                18.Lutz R, Bujard H. Independent and tight regulation of transcriptional units in Escherichia coli via the LacR/O, the TetR/O and AraC/I1-I2 regulatory elements. Nucleic Acids Res. 1997 Mar 15;25(6):1203-10. doi: 10.1093/nar/25.6.1203. PMID: 9092630; PMCID: PMC146584.
                            </a>
                        </cite>
        
        <cite>
        <a href="https://pubmed.ncbi.nlm.nih.gov/30478458/">
                                19.Meyer AJ, Segall-Shapiro TH, Glassey E, Zhang J, Voigt CA. Escherichia coli "Marionette" strains with 12 highly optimized small-molecule sensors. Nat Chem Biol. 2019 Feb;15(2):196-204. doi: 10.1038/s41589-018-0168-3. Epub 2018 Nov 26. PMID: 30478458.
                            </a>
                        </cite>
        
        <cite>
        <a href="https://pubmed.ncbi.nlm.nih.gov/31488843/">
                                20.Huber, M., Faure, G., Laass, S., Kolbe, E., Seitz, K., Wehrheim, C., Wolf, Y.I., Koonin, E.V., Soppa, J., 2019. Translational coupling via termination-reinitiation in archaea and bacteria. Nat Commun 10, 4006. https://doi.org/10.1038/s41467-019-11999-9 [Accessed in June 2022]
                            </a>
                        </cite>
        
        <cite>
        <a href="https://pubmed.ncbi.nlm.nih.gov/19359587/">
                                21.Kudla G, Murray AW, Tollervey D, Plotkin JB. Coding-sequence determinants of gene expression in Escherichia coli. Science. 2009 Apr 10;324(5924):255-8. doi: 10.1126/science.1170160. PMID: 19359587; PMCID: PMC3902468.
                            </a>
                        </cite>
        
        <cite>
        <a href="https://pubmed.ncbi.nlm.nih.gov/16369541/">
                                22.Pédelacq JD, Cabantous S, Tran T, Terwilliger TC, Waldo GS. Engineering and characterization of a superfolder green fluorescent protein. Nat Biotechnol. 2006 Jan;24(1):79-88. doi: 10.1038/nbt1172. Epub 2005 Dec 20. Erratum in: Nat Biotechnol. 2006 Sep;24(9):1170. PMID: 16369541.
                            </a>
                        </cite>
        
        <cite>
        <a href="https://pubs.acs.org/doi/abs/10.1021/acssynbio.2c00139">
                                23.Duan Y, Zhang X, Zhai W, Zhang J, Zhang X, Xu G, Li H, Deng Z, Shi J, Xu Z. Deciphering the Rules of Ribosome Binding Site Differentiation in Context Dependence. ACS Synth Biol. 2022 Aug 19;11(8):2726-2740. doi: 10.1021/acssynbio.2c00139. Epub 2022 Jul 25. PMID: 35877551.
                            </a>
                        </cite>
        
        <cite>
        <a href="https://www.addgene.org/collections/tetracycline/">
                                24.https://www.addgene.org/collections/tetracycline/ [Accessed in June 2022]
                            </a>
                        </cite>
        
        <cite>
        <a href="https://www.nature.com/articles/s41467-020-17677-5">
                                25. Baumschlager, A., Rullan, M., Khammash, M., 2020. Exploiting natural chemical photosensitivity of anhydrotetracycline and tetracycline for dynamic and setpoint chemo-optogenetic control. Nat Commun 11, 3834. https://doi.org/10.1038/s41467-020-17677-5 [Accessed in June 2022]
        
                            </a>
                        </cite>
        
        <cite>
        <a href="https://www.frontiersin.org/articles/10.3389/fmicb.2018.01058/full">
                                26. Markley, J.L., Wencewicz, T.A., 2018. Tetracycline-Inactivating Enzymes. Front. Microbiol. 9, 1058. https://doi.org/10.3389/fmicb.2018.01058 [Accessed in June 2022]
                            </a>
                        </cite>
        
        <cite>
        <a href="https://pubmed.ncbi.nlm.nih.gov/24781324/">
                                27. Brophy JA, Voigt CA. Principles of genetic circuit design. Nat Methods. 2014 May;11(5):508-20. doi: 10.1038/nmeth.2926. PMID: 24781324; PMCID: PMC4230274.
                            </a>
                        </cite>
        
        <cite>
        <a href="https://pubmed.ncbi.nlm.nih.gov/23905721/">
                                28. Ang J, Harris E, Hussey BJ, Kil R, McMillen DR. Tuning response curves for synthetic biology. ACS Synth Biol. 2013 Oct 18;2(10):547-67. doi: 10.1021/sb4000564. Epub 2013 Sep 3. PMID: 23905721; PMCID: PMC3805330.
                            </a>
                        </cite> 
        
        <cite>
        <a href="https://pubmed.ncbi.nlm.nih.gov/10659857/">
                                29. Gardner TS, Cantor CR, Collins JJ. Construction of a genetic toggle switch in Escherichia coli. Nature. 2000 Jan 20;403(6767):339-42. doi: 10.1038/35002131. PMID: 10659857.
                            </a>
                        </cite>
        
        <cite>
        <a href="https://pubmed.ncbi.nlm.nih.gov/17905197/">
                                30. Sayut DJ, Kambam PK, Sun L. Noise and kinetics of LuxR positive feedback loops. Biochem Biophys Res Commun. 2007 Nov 23;363(3):667-73. doi: 10.1016/j.bbrc.2007.09.057. Epub 2007 Sep 24. PMID: 17905197.
                            </a>
                        </cite>
        
        <cite>
        <a href="https://www.pnas.org/doi/full/10.1073/pnas.0408507102">
                                31. Hooshangi S, Thiberge S, Weiss R. Ultrasensitivity and noise propagation in a synthetic transcriptional cascade. Proc Natl Acad Sci U S A. 2005 Mar 8;102(10):3581-6. doi: 10.1073/pnas.0408507102. Epub 2005 Feb 28. PMID: 15738412; PMCID: PMC552778.
                            </a>
                        </cite>
        
        <cite>
        <a href="https://pubmed.ncbi.nlm.nih.gov/18599624/">
                                32. Legewie S, Dienst D, Wilde A, Herzel H, Axmann IM. Small RNAs establish delays and temporal thresholds in gene expression. Biophys J. 2008 Oct;95(7):3232-8. doi: 10.1529/biophysj.108.133819. Epub 2008 Jul 3. PMID: 18599624; PMCID: PMC2547459.
                            </a>
                        </cite>
        
        <cite>
        <a href="https://pubmed.ncbi.nlm.nih.gov/19455136/">
                                33. Buchler NE, Cross FR. Protein sequestration generates a flexible ultrasensitive response in a genetic network. Mol Syst Biol. 2009;5:272. doi: 10.1038/msb.2009.30. Epub 2009 May 19. PMID: 19455136; PMCID: PMC2694680.
                            </a>
                        </cite>
        
        <cite>
        <a href="https://pubmed.ncbi.nlm.nih.gov/23089683/">
                                34. Chen D, Arkin AP. Sequestration-based bistability enables tuning of the switching boundaries and design of a latch. Mol Syst Biol. 2012;8:620. doi: 10.1038/msb.2012.52. PMID: 23089683; PMCID: PMC3501275.
                            </a>
                        </cite>
        
        
        
        <cite>
        <a href="https://www.nature.com/articles/s41467-020-19552-9/">
                                35. Wan X, Pinto F, Yu L, Wang B. Synthetic protein-binding DNA sponge as a tool to tune gene expression and mitigate protein toxicity. Nat Commun. 2020 Nov 24;11(1):5961. doi: 10.1038/s41467-020-19552-9. PMID: 33235249; PMCID: PMC7686491.
                            </a>
                        </cite>
        
        <cite>
        <a href="https://pubmed.ncbi.nlm.nih.gov/22453733/">
                                36. Lee TH, Maheshri N. A regulatory role for repeated decoy transcription factor binding sites in target gene expression. Mol Syst Biol. 2012 Mar 27;8:576. doi: 10.1038/msb.2012.7. PMID: 22453733; PMCID: PMC3322331.
                            </a>
                        </cite>
        
        <cite>
        <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0126264">
                                37.Zucca S, Pasotti L, Politi N, Casanova M, Mazzini G, Cusella De Angelis MG, Magni P. Multi-Faceted Characterization of a Novel LuxR-Repressible Promoter Library for Escherichia coli. PLoS One. 2015 May 26;10(5):e0126264. doi: 10.1371/journal.pone.0126264. PMID: 26010244; PMCID: PMC4444344.
                            </a>
                        </cite>
        
        <cite>
        <a href="https://pubmed.ncbi.nlm.nih.gov/33089779/">
                                38.Fei X, Bell TA, Barkow SR, Baker TA, Sauer RT. Structural basis of ClpXP recognition and unfolding of ssrA-tagged substrates. Elife. 2020 Oct 22;9:e61496. doi: 10.7554/eLife.61496. PMID: 33089779; PMCID: PMC7652416.
                            </a>
                        </cite>
        
        <cite>
        <a href="https://pubmed.ncbi.nlm.nih.gov/33089779/">
                                38.Fei X, Bell TA, Barkow SR, Baker TA, Sauer RT. Structural basis of ClpXP recognition and unfolding of ssrA-tagged substrates. Elife. 2020 Oct 22;9:e61496. doi: 10.7554/eLife.61496. PMID: 33089779; PMCID: PMC7652416.
                            </a>
                        </cite>
        
                    </div>
                </section>
            </div>
            </div>


</body>

<!--Loader-->   
{% include 'loader.html'%}

<!-- Footer -->
{% include 'footer.html' %}

</html>