<!doctype html>
<html>
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="shortcut icon" href="https://static.igem.wiki/teams/4294/wiki/favicon-white.png"/>
    <link rel="license" href="https://creativecommons.org/licenses/by/4.0/"/>
    
    <!-- Custom CSS -->
    <link href="{{ url_for('static', filename = 'css/engineering_style.css') }}" rel="stylesheet">

    <!-- Loader -->
    <script type= "text/javascript" src="../../athens/static/scripts/jquery.min.js"></script>

    <!--Custom menu CSS-->
    <link href="{{url_for('static',filename='css/menu_style_aspa.css')}}" rel="stylesheet">

    <!--Custom footer CSS-->
    <link href="{{url_for('static',filename='css/footer_style_aspa.css')}}" rel="stylesheet">

     <!--Custom to top button CSS-->
     <link href="{{url_for('static',filename='css/to_top_button.css')}}" rel="stylesheet">
    <title> Perspectives | Engineering</title>
</head>

<body>
<!-- Navigation -->
{% include 'menu_custom.html' %}

<!--To top button-->
{% include 'to_top_button.html'%}

<!--Loader-->   
{% include 'loader.html'%}  

<div class="page_header">
    <img class="img_header" src="https://static.igem.wiki/teams/4294/wiki/contribution/project-bg.gif" width="100%">
    <div class="heading">Engineering Success</div>    
</div>

<div class="buttons">
  <div class="button">
      <a href="#senders">Senders</a>
  </div>
  <div class="button">
      <a href="#receivers">Receivers</a>
  </div>
</div>

<!-- Summary -->

<div class="overview-section-senders" id="summary" >
  <h1>Engineering Success Summary</h1>
</div>

<div class="cc-summary">
  <div class="left-side">
      <div class="index-bar">
          <ul class="lateral-index">
              <li class="lateral-content">
                  <a href="#summary"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">Summary</span></a>
              </li>
              <li class="lateral-content">
                  <a href="#symmary-rec">Receivers </a>
              </li>
              <li class="lateral-content">
                  <a href="#summary-sen">Senders</a>
              </li>
              <li class="lateral-content">
                <a href="#summary-new-parts">New Parts</a>
              </li>
              <li class="lateral-content">
              <a href="#receivers"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">Receivers</span></a>
              </li>
              <li class="lateral-content">
                <a href="#senders"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">Senders</span></a>
              </li>
              <li class="lateral-content">
                <a href="#references"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">References</span></a>
            </li>
          </ul>
      </div>
  </div>
  
  <div class="right-side">

<!-- Overview  -->
          <div class="overview-summary">
              <h1>Overview</h1>
              <p>
              Our team has gone through several engineering cycles in order to establish a reliable characterization circuit for the Senders cells and a steep biological activation function in the Receiver Cells.
              <br><br>
              An overview of our successful constructs and parts is presented. For a detailed analysis of our engineering cycles, please push the respective buttons.
          
              </p>
          </div>

          <!-- Receivers Summary  -->
          <section class="symmary-rec">
            <a id="symmary-rec" class="anchorOffset"></a>
            <h1>Receivers </h1>
            <br>
            A circuit with a steeper response curve incorporating the LuxR and PhlF regulators was successfully engineered and tested. These circuits are described in detail in the “Design” tab.
            <br><br>

            <center>
            <img class="fig-a" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-a.png">
            <figcaption>
            Figure A: Comparison of OL colE1 or., OL p15 or. and PFr colE1 or. at 240 min. The uninduced cells are represented by the concentration value 0.0000001 for the diagram purposes.  
            </figcaption>
            </center>
            <br>           
                
          </section>

          <!-- Senders Summary  -->
          <section class="summary-sen">
            <a id="summary-sen" class="anchorOffset"></a>
            <h1>Senders </h1>
            <br>
            An efficient method for characterizing the relative strength of an RBS in the genetic context of LuxI synthase was built by fusing the first 36 nucleotides of the LuxI CDS with sfGFP.
            <br><br>

            <center>
            <img class="fig-b" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-b.png">
            <figcaption>
            Figure B:
            Illustration of the 36nt LuxI- sfGFP fusion reporter protein.
            </figcaption>
            </center>
            <br>      
                
          </section>

          <!-- New Parts Summary  -->
          <section class="summary-new-parts">
            <a id="summary-new-parts" class="anchorOffset"></a>
            <h1>New Parts</h1>
            <br>
            Among our new registered parts, there are two parts that, to our knowledge, have not been reported before and would be a valuable contribution to the iGEM Community.
            <br><br>
            <b><a href="http://parts.igem.org/Part:BBa_K4294207"><b>BBa_K4294207</b></a> 36nt LuxI - sfGFP fusion</b>
            <br>
            To our knowledge, the fusion of the 36 first nucleotides of the  LuxI Coding Sequence with sfGFP has not been reported before. These fusions make the characterisation of relative translation levels of LuxI synthase easier. LuxI is part of a widely used quorum sensing system and improving its characterisation strategies is important. For more information regarding the results generated with this part, refer to the <a href="https://2022.igem.wiki/athens/results">Results
            </a> pather or the <a href="https://2022.igem.wiki/athens/parts">Parts</a> registry page.
            <br><br>

            <b><a href="http://parts.igem.org/Part:BBa_K4294301"><b>BBa_K4294301</b></a> LuxR-Phlf controlled hybrid promoter</b>
            <br>
            This new promoter was used in our design attempts to create a genetic circuit with a steeper response curve. It contains a LuxR binding domain (lux box) upstream the -35 region and a Phlf operator in the core sequence that also overlaps with the -10 region. It is therefore positively regulated by LuxR and negatively by Phlf. For more information regarding the results generated with this part, refer to the Results tab or the part’s registry page.
            <br><br> 
            
            <b>Important Notes:</b>
            <br><br> 
            1. For more information about the decision of the plasmid vectors, bacterial strains and the whole genetic circuit illustrations and Genbank files, please refer to the “Design” tab.
            <br> 
            2. For the protocols mentioned below, please refer to the “Protocols” tab.
            <br> 
            3. For more detailed information about the results mentioned below, please refer to the “Results” tab.
            <br> 
            4. For the mentioned agarose gel electrophoresis results, the 1kb plus ladder of New England Biolabs was used if not noted otherwise.
            <br><br> 

            <center>
            <img class="es-fig-1" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-1.png">
            <figcaption>
            Figure 1: 1kb plus DNA ladder New England Biolabs (N3200L)
            </figcaption>
            </center>
          </section>


  </div>
</div>

<!-- Receivers  -->


<div class="overview-section-receivers" id="receivers">
  <h1>Receivers</h1>
</div>

<div class="cc-receivers">
  <div class="left-side">
    <div class="index-bar">
        <ul class="lateral-index">
            <li class="lateral-content">
                <a href="#summary"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">Summary</span></a>
            </li>
            <li class="lateral-content">
                <a href="#receivers"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">Receivers</span></a>
            </li>
            <li class="lateral-content">
              <a href="#steep-act">Activation Function</a>
            </li>
            <li class="lateral-content">
              <a href="#senders"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">Senders</span></a>
            </li>
            <li class="lateral-content">
              <a href="#references"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">References</span></a>
          </li>
        </ul>
    </div>
  </div>
  
  <div class="right-side">
<!-- Activation Function  -->
        <section class="steep-act">
          <a id="steep-act" class="anchorOffset"></a>
          <h1>Engineering a Steep Activation Function</h1>
          <br>
          One important aspect of our project was to imitate the activation function of a perceptron algorithm using a genetic circuit with a steep response curve. We focused on transcriptional regulation for the construction of a switch-like response and followed a step-by-step approach.
            <!-- 1st Cycle -->
            <div class="collapsible_in">
              <input type="checkbox" id="1st-cycle-steep-act">
              <label for="1st-cycle-steep-act">1st Cycle</label>
              <div class="collapsible-in-text">
              <h1>Design</h1>
              <p>
              Our step-by-step approach begins with the characterization of the LuxR activation system in its simplest form, an open loop circuit. 
              <br><br>
              <b>Model guided design</b>
              <br>
              In terms of dry lab simulations, before our team began with the OL construct characterization in the lab, we performed some parametric simulations with emphasis on three different design parameters; the construct’s copy number and the RBS variants used in the open loop circuit, regarding the production rates of LuxR and mNeonGreen. For that purpose, our modeling team used the whole-cell model for the OL construct, which is more analytically described in the receiver’s <a href="https://2022.igem.wiki/athens/model">Model</a> page.             
              <br>

              <center>
              <img class="fig-2" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-2.png">
              <figcaption>
              Figure 2: Parametric analysis for different Copy number values and RBS variants for the production of LuxR and mNeonGreen molecules. The abbreviations in the plot legends stand for: ‘30’=RBS BBa_B0030, ‘32’=RBS BBa_B0032, ‘34’=RBS_BBa_B0034, CN=Copy Number
              </figcaption>
              </center>
              <br>   

              From the graphs in figure 1, we can clearly observe the following
              <br><br>
              <ul>
               <li>
              Both the leakiness and final fluorescence values seem to be mainly dependent on the RBS of the mNeonGreen that would be used. So it would be wise to go with a moderate RBS variant in order to reduce the leakiness observed. As a result, we opted not to use the RBS BBa_B0030 to control the translation of LuxR.
              </li>
              <li>
              On the other hand, the RBS for the production of the LuxR seems to be directly influencing the initiation point of our function; the stronger the RBS controlling LuxR translation, the lower AHL value needed to cause induction. We noticed that the RBS ‘30’ construct is shifted to the left of the x-axis in comparison to ‘32’ and ‘34’. We decided therefore to exclude the RBS ‘30’ as a potential sequence for the production of LuxR since it might lead to a system with a response to very low OC6 concentrations, which could lead to the activation of the receivers by any combination of senders. 
              </li>
              <li>
              Finally the copy number is playing a hybrid role on our transfer functions, since it regulates not only the overall production values of mNeonGreen (including leakiness) but also leads to a minor increase in the AHL inducer concentration needed in order for the induction to start.
              </li>
              </ul>
              <br><br>
              From the analysis above, we decided to use the RBS BBa_0034 for both mNeonGreen and LuxR constructs and finalized our vector’s copy number choice with a preference towards the ColE1 origin of replication (medium CN). Both vectors will be tested to confirm the predictions. In that way, we’ve achieved a modest amount of leakiness, solid output fluorescence as well as a small shift of our graph towards the right axis. 
              <br><br>
              Unfortunately, due to the early stages of the 1st engineering cycle and a lack of experimental characterization data, a full simulation of our OL construct alongside the sender’s model as shown in the proof of concept and model pages was not implemented. Thus, that type of characterization would be still suboptimal for the entire system. However, later on, via our fitted sender’s and receiver’s models we were able to make a full simulation and show that a potential misfit of the transfer function would be overcome via a difference in the volumes of the sender’s and the receiver’s populations.
              <br><br>
                         
              <h1>Build</h1>
              <br>
                <ul>
                <li>
                We performed a Golden Gate Assembly with BsmBI restriction enzyme for each plasmid vector
                </li>
                    <ul>
                    <li>  
                    As vectors we used the BBa_K4294092 pTU2-A-RFP (colE1 origin) and the BBa_K4294093 pTU2-A-RFP (p15A origin). 
                    </li>
                    <li>
                    The parts used as inserts were the BBa_K4294806 (TU pLux Output for OL, PF, PFc) and the BBa_K4294808 (TU constitutive LuxR for OL)
                    </li>
                    </ul>
                <li>
                Transformation on <it>E. coli</it> DH5a at LB - Agar plates containing chloramphenicol. 
                </li>
                <li>
                Screening of colonies (white colonies contain the vector with the insert, red colonies the initial vector.
                </li>
                <li>
                Overnight liquid cultures from grown white colonies.
                </li>
                <li>
                Plasmid Miniprep.
                </li>
                Diagnostic Digest and PCR amplification (primers flanking the insert: Biobrick Prefix Forward Primer BBa_K4294021, pTU2 Reverse Primer  BBa_K4294032) for both Open Loop constructs.
                <li>
                Agarose gel electrophoresis.
                <br>
                Expected zones:
               
                  <ul>
                    <li>
                      Open-loop (pTU2-A-RFP colE1 origin) cut with BsaI & HindIII: 3 zones at 2027bp + 1092bp + 853bp
                    </li>
                    <li>
                      Open-loop (pTU2-A-RFP p15a origin) from PCR: 1 zone at 2045bp
                    </li>
                  </ul>
                </li>
                </ul>  
              <br>

              <center>
                <img class="fig-3" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-3.png">
                <figcaption>
                  Figure 3: Agarose Gel Electrophoresis<br>
                  L: Ladder 1kb plus<br>
                  1: Open-loop (pTU2-A-RFP colE1 origin) from the 2nd colony uncut<br>
                  2: Open-loop (pTU2-A-RFP colE1 origin) from the 1st colony cut with BsaI & HindIII<br>
                  3: Open-loop (pTU2-A-RFP colE1 origin) from the 2nd colony<br>
                  4: Open-loop (pTU2-A-RFP p15a origin) from PCR<br>
                  a: Agarose gel 1%<br>
                  b: Agarose gel 1%<br>
                  <br><br>
                  Figure 3 shows that our diagnostic digest and PCR on the receiver’s open-loop constructs resulted in the expected bands.
                </figcaption>
                </center>
                <br>              
              
              <center>
              <img class="fig-4" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-4.png">
              <figcaption>
                Figure 4:  Induction of OL constructs with different OC6 concentrations and in different timepoints. Comparison between p15A (a) origin and ColE1(b).
              </figcaption>
              </center>
              <br>

              <h1>Test</h1>
              <br>
              To test the aforementioned constructs, <it>E. coli</it> BL21 cells were transformed with the open loop constructs and were induced with a range of concentrations of Acyl Homoserine-Lactone (OC6). Cells were incubated in a shaking incubator and successive measurements of absorbance at 600nm and fluorescence were taken using the plate reader FlexStation® 3 for a time period of 4h. The results showed an increase in the RFU/OD600nm ratio and a sigmoidal response curve for both constructs.
              <br><br>

              <h1>Learn</h1>
              <br>
              These experiments prove that the ColE1 origin of replication is more appropriate for our system and, therefore, we use the medium copy number (ColE1 ori) pTU2 vector to engineer the rest of the constructs. A higher copy number provides a higher dynamic range with a minimal tradeoff regarding its leakiness. These first measurements were also important to detect the range of concentrations in which the system is inducible and update our models’ parameters.
              </p>
            </div>
            </div>

            <!-- 2nd Cycle -->
            <div class="collapsible_in">
              <input type="checkbox" id="2nd-cycle-steep-act">
              <label for="2nd-cycle-steep-act">2nd Cycle</label>
              <div class="collapsible-in-text">
              <h1>Design</h1>
              <br>
              <p>
              With the goal to build an ultrasensitive, switch-like response, and inspired by previous related work, two circuit designs in two variations regarding the basal LuxR expression were chosen. A LuxR transcriptional Positive Feedback loop (PF and PFc) and a circuit that incorporates an extra repressor (PhlF) that is regulated by LuxR (PFR, PFcR) resulting in a coherent feedforward loop with an embedded LuxR transcriptional Positive Feedback.
              <br><br>
              <b>Dry Lab Prediction</b>
              In terms of design simulations, we were able to perform a simulation of the PFR construct in parallel with the characterisation of OL constructs by the wet lab, valuable for the circuit design finalization, since a circuit incorporating such regulatory elements in the same topology has not, to our knowledge, been characterized. As analyzed in the receiver’s model section , we used some of the fitted parameters from the OpLo (OL) system and alongside the rest of the literature ones we received the following whole-cell transfer function:

              <center>
              <img class="fig-5" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-5.png">
              <figcaption>
              Figure 5: Theoretical model results for the PFR construct.
              </figcaption>
              </center>
              <br>
              
              The received curve for the PFR system displays an increased steepness and it would be the receiver’s design with the most potential.
              <br><br> 
              On the other hand, the simulations displayed for the PF and PFc results weren’t the expected ones and lead to similar results as in the case for the OL system. It should be mentioned that, for all the simulations in the PFR we used the RBS construct BBa_0034 and a medium copy number origin of replication (ColE1), from the 1st engineering cycle.
              <br><br> 
              Even though every circuit would be experimentally tested, the above simulation would aid a potential redesign of a circuit in parallel with the experiments in case of no functional circuits predicted. 
              <br><br> 
              <h1>Build</h1>
              <br>
              <ul>
                <li>
                  We performed 4 different Golden Gate Assemblies with BsmBI by using the vector BBa_K4294092 pTU2-A-RFP (colE1 origin) for all the constructs.                
                </li>
                    <ul>
                    <li>  
                      For the PF construct we used the parts:
                    </li>
                        <ul>
                          <li>
                            BBa_K4294806 (TU pLux Output for OL, PF, PFc)
                          </li>
                          <li>
                            BBa_K4294809 (TU pLux LuxR PF, PF+ R) 
                          </li>
                        </ul>

                    <li>  
                    For the PFR construct we used the parts:
                    </li>
                        <ul>
                          <li>
                            BBa_K4294812 (TU Output for PF+ R, PFc+ R)
                          </li>
                          <li>
                            BBa_K4294807 (TU pLuxr PhlF)
                          </li>
                          <li>
                            BBa_K4294809 (TU pLux LuxR PF, PF+ R) 
                          </li>
                        </ul>

                      <li>  
                        For the PFc construct we used the parts:
                      </li>
                            <ul>
                              <li>
                                BBa_K4294806 (TU pLux Output for OL, PF, PFc)
                              </li>
                              <li>
                                BBa_K4294810 (TU pLux LuxR PFc, PFc+ R)
                              </li>
                              <li>
                                BBa_K4294811 (TU constitutive LuxR for PFc, PFc+R) 
                              </li>
                            </ul>
                      <li>  
                        For the PFc+R construct we used the parts:
                      </li>
                            <ul>
                              <li>
                                BBa_K4294812 (TU Output for PF+ R, PFc+ R)
                              </li>
                              <li>
                                BBa_K4294807(TU pLuxr PhlF)
                              </li>
                              <li>
                                BBa_K4294811 (TU constitutive LuxR for PFc, PFc+R)
                              </li>
                              <li>
                                BBa_K4294810 (TU pLux LuxR PFc, PFc+ R)
                              </li>
                            </ul>


                <li>
                  Transformation on <it>E. coli<\it> DH5a at LB - Agar plates containing chloramphenicol. 
                </li>
                <li>
                  Choose the white colonies (plasmids with insert) over the red colonies (no insert in plasmid).
                </li>
                <li>
                  Overnight liquid cultures from grown white colonies
                </li>
                <li>
                  Plasmid Miniprep.
                </li>
                Diagnostic Digest and PCR amplification with primers binding upstream and downstream of the insert region for the Positive Feedback construct (pTU2-A-RFP colE1 origin), for Positive Feedback with constitutive production of LuxR construct (pTU2-A-RFP colE1 origin), for Positive Feedback + Repressor with constitutive production of LuxR construct (pTU2-A-RFP colE1 origin) and for Positive Feedback +Repressor construct (pTU2-A-RFP colE1 origin).
                <li>
                  Agarose gel electrophoresis.
                <br>
                Expected zones:
                </li>
                  <ul>
                    <li>
                      Positive Feedback construct  (pTU2-A-RFP colE1 origin) cut with BsaI: 2 zones at 2027bp + 1958bp.
                    </li>
                    <li>
                      Positive Feedback with constitutive production of LuxR construct (pTU2-A-RFP colE1 origin) cut with BsaI: 2 zones at 2959bp + 2027 bp.
                    </li>
                    <li>
                      Positive Feedback construct  (pTU2-A-RFP colE1 origin) cut with  BsaI and HindIII: 3 zones at 847 bp & 111 bp & 2027 bp.
                    </li>
                  </ul>
                </ul>  
              <br>
              
              <center>
              <img class="fig-6" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-6.png">
              <figcaption>
                Figure 6: Agarose Gel Electrophoresis<br>
                L: Ladder 1kb plus<br>
                1: PF (pTU2-A-RFP colE1 origin) cut with BsaI<br>
                2: PF (pTU2-A-RFP colE1 origin) cut with BsaI<br>
                3: PFc (pTU2-A-RFP colE1 origin) cut with BsaI<br>
                4: PFc (pTU2-A-RFP colE1 origin) cut with BsaI<br>
                5: PF (pTU2-A-RFP colE1 origin) cut with BsaI and HindIII<br>
                <br>
                Figure 6 shows that our diagnostic digest on the receiver’s PF and PFc constructs resulted in the expected bands. The zone of 111 bp is very faint in lane 5 (PF (pTU2-A-RFP colE1 origin) cut with BsaI and HindIII) due to its small size.
              </figcaption>
              </center>
              <br>

              Regarding the PFR and PFc+R constructs:<br>
                <br><br>
                Expected zones:
               
                <ul>
                  <li>
                    Positive Feedback Repressive with constitutive production of LuxR construct  (pTU2-A-RFP colE1 origin) cut with BsaI: 2 zones at 3914bp + 2027bp.
                  </li>
                  <li>
                    Positive Feedback Repressive construct (pTU2-A-RFP colE1 origin) cut with BsaI: 2 zones at 2913bp + 2027 bp.</ul>
                  </li>
                </ul>
              
              <center>
              <img class="fig-7" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-7.png">
              <figcaption>
              Figure 7: Agarose gel electrophoresis<br> 
              L: Ladder 1kb plus<br>
              1: PFR(pTU2-A-RFP colE1 origin) cut with BsaI<br>
              2: PFR(pTU2-A-RFP colE1 origin) cut with BsaI<br>
              3: Control pTU2-A-RFP colE1 origin uncut<br>
              4: Control pTU2-A-RFP colE1 origin cut with BsaI<br>
              5: PFc +R (pTU2-A-RFP colE1 origin) cut with BsaI<br>
              <br>
              Figure 7 shows that our diagnostic digest on the receiver’s PFR and PFc+R constructs resulted in the expected bands.
              </figcaption>
              </center>
              <br>

              <h1>Test</h1>
              <br>
              For the testing of the aforementioned constructs, we transformed<it>E. coli<\it> BL21 and induced them with a range of concentrations of Acyl Homoserine-Lactone OC6. Absorbance and Fluorescence Measurements were taken at specific time points using the plate reader FlexStation® 3 at a total time period of 4h. 
              <br><br>
              For the PF construct the concentration range 0.062μΜ-0.000121094μΜ was used for induction.
              <br><br>

              <center>
              <img class="fig-8" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-8.png">
              <figcaption>
              Figure 8: Induction of BL21 PF pTU2-RFP colE1 or. with the following OC6 concentrations (μΜ): 0.062, 0.031, 0.0155, 0.003875, 0.0019375, 0.00096875, 0.000484375, 0.000242188, 0.000121094. The uninduced cells are represented by the concentration value 0.00001 for diagram purposes. 
              </figcaption>
              </center>
              <br>  

              Regarding the PFc construct, the concentration range 10μΜ-0.0001μΜ was used for induction. Data presented in figure 9.
              <br><br> 

              <center>
              <img class="fig-9" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-9.png">
              <figcaption>
                Figure 9: Induction of BL21 PFc pTU2-RFP colE1 or. with the following OC6 concentrations (μΜ): 10, 1, 0.1, 0.01, 0.001, 0.0001. The uninduced cells are represented by the concentration value 0.000001 for diagram purposes. 
              </figcaption>
              </center>
              <br>

              For the PFr construct the concentration range 100μΜ-0.00001024μΜ was used for induction. The results that are presented in figure 10. 
              <br><br> 

              <center>
              <img class="fig-10" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-10.png">
              <figcaption>
                Figure 10: Induction of BL21 PFr pTU2-RFP colE1 or. with the following OC6 concentrations (μΜ):100, 20, 4, 0.16,    0.0064, 0.00128, 0.000256, 0.00001024. The uninduced cells are represented by the concentration value 0.000001 for diagram purposes. 
              </figcaption>
              </center>
              <br>

              For the PFc+R construct, the concentration range 10μΜ-0.0.00000512μΜ was used for induction. The results are presented in figure 11. 
              <br><br> 

              <center>
              <img class="fig-11" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-11.png">
              <figcaption>
                Figure 11: Induction of BL21 PFc+R pTU2-RFP colE1 or. with the following OC6 concentrations (μΜ): 10, 2, 0.4, 0.08, 0.016, 0.0032, 0.00064, 0.000128, 0.0000256, 0.00000512. The uninduced cells are represented by the concentration value 0.0000001 for diagram purposes.
              </figcaption>
              </center>
              <br>

              <h1>Learn</h1>
              <br>
              PF: According to the results that are presented in the figure 8 there is no significant increase in RFU/OD600nm ratio in comparison with other constructs. This circuit presents an uncommon response in comparison with similar circuits from the literature. 
              <br><br> 
              PFc: According to the results that are presented in figure 9. there is a slight increase in fluorescence expressed in arbitrary units, which could, by a rough approximation, resemble a sigmoidal response curve in the 0.1 to 1 μM of inducer but in general, seems more linear. The response curve does not fulfill our goals regarding steepness and dynamic range and this result is also considered unreliable due to its uncommon response in comparison with similar circuits from the literature. 
              <br><br> 
              PFR: This circuit is characterized by a response curve with low leakiness, high dynamic range and a steeper activation function. For the transition from Min to Max state a 100-fold increase in inducer’s concentration seems necessary. However, in the sigmoid part of the response curve, the slope of the curve is gradually increasing. This is an indicator that a repeat of these measurements with more OC6 concentrations within the Min to Max range will probably better reveal the increased steepness in a more narrow OC6 concentration range. Our model supports the hypothesis of this circuit’s steeper activation function (see Model section for more).
              <br><br> 
              PFcR: This circuit topology fails to be robustly induced. An approximal sigmoidal response curve is observed in the 0.0001μM to 0.001 μM inducer range, but the dynamic range is quite low. The only 10fold increase in OC6 concentration required for a response transition from Min to Max is an indicator of a switch-like response. However, on the grounds of this low Min to Max range, it is also probable that the higher response in 0.001 μM was an artifact.
              <br><br> 
              In conclusion, the PFR and PFcR circuit architectures may provide a steeper activation function to our biological perceptron. More specifically, PFR shows the most robust response and a further tuning of its components dynamics in a model driven approach could potentially improve its response. Potential solutions include PhlF tagging with a variety of degradation tags to estimate the ideal degradation rates, a LuxR repressible promoter of different strength and modifications of the LuxR dynamics (constitutive promoter strength in PFcR, RBS controlling its translation, potential tagging with degradation tags). 
              <br> 

              <center>
              <img class="fig-12" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-12.png">
              <figcaption>
              Figure 12:Comparison of OL colE1 or., OL p15 or. and PFR colE1 or. at 240 min. The uninduced cells are represented by the concentration value 0.0000001 for the diagram purposes.  
              </figcaption>
              </center>
              <br>
              </p>
            </div>
            </div>

              
          </section>

        
    </div>
  </div>


<!-- Senders -->


<div class="overview-section-senders" id="senders">
  <h1>Senders</h1>
</div>

<div class="cc-senders">
  <div class="left-side">
    <div class="index-bar">
        <ul class="lateral-index">
            <li class="lateral-content">
                <a href="#summary"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">Summary</span></a>
            </li>
            <li class="lateral-content">
                <a href="#receivers"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">Receivers</span></a>
            </li>
            <li class="lateral-content">
              <a href="#senders"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">Senders</span></a>
            </li>
            <li class="lateral-content">
              <a href="#circuit">Characterisation Circuit</a>
            </li>
            <li class="lateral-content">
                <a href="#strategy">Characterisation Strategy</a>
            </li>
            <li class="lateral-content">
              <a href="#constructs">Final Constructs</a>
            </li>
            <li class="lateral-content">
              <a href="#crosstalk">Crosstalk Reduction</a>
            </li>
            <li class="lateral-content">
              <a href="#references"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">References</span></a>
          </li>
        </ul>
    </div>
  </div>
  
  <div class="right-side">

          <!-- Circuit  -->
          <section class="circuit">
          <a id="circuit" class="anchorOffset"></a>
          <h1>Sender Characterization Circuit</h1>
            
            <!-- 1st Cycle -->
            <div class="collapsible_in">
              <input type="checkbox" id="1st-cycle-circuit">
              <label for="1st-cycle-circuit">1st Cycle</label>
              <div class="collapsible-in-text">
              <h1>Design</h1>
              <br><br>
              <p>
              </p>

              <h1>Build</h1>
              <br><br>
              <p>
              </p>

              <h1>Test</h1>
              <br><br>
              <p>
              </p>

              <h1>Learn</h1>
              <br><br>
              <p>
              </p>
            </div>
            </div>

            <!-- 2nd Cycle -->
            <div class="collapsible_in">
              <input type="checkbox" id="2nd-cycle-circuit">
              <label for="2nd-cycle-circuit">2nd Cycle</label>
              <div class="collapsible-in-text">
              <h1>Design</h1>
              <br><br>
              <p>
              </p>

              <h1>Build</h1>
              <br><br>
              <p>
              </p>

              <h1>Test</h1>
              <br><br>
              <p>
              </p>

              <h1>Learn</h1>
              <br><br>
              <p>
              </p>
            </div>
            </div>

            <!-- 3rd Cycle -->
            <div class="collapsible_in">
              <input type="checkbox" id="3rd-cycle-circuit">
              <label for="3rd-cycle-circuit">3rd Cycle</label>
              <div class="collapsible-in-text">
              <h1>Design</h1>
              <br><br>
              <p>
              </p>

              <h1>Build</h1>
              <br><br>
              <p>
              </p>

              <h1>Test</h1>
              <br><br>
              <p>
              </p>

              <h1>Learn</h1>
              <br><br>
              <p>
              </p>
            </div>
            </div>
              
          </section>

          <!-- Strtegy -->

          <section class="strategy">
          <a id="strategy" class="anchorOffset"></a>
          <h1>Final Characterisation Strategy</h1>
          
          </section>

          <!-- Contructs  -->
          <section class="constructs">
          <a id="constructs" class="anchorOffset"></a>
          <h1>Final Senders’ constructs</h1>
            
            <!-- Constructs Cycle -->
            <div class="collapsible_in">
              <input type="checkbox" id="constructs-cycle-circuit">
              <label for="constructs-cycle-circuit">Engineering Cycle</label>
              <div class="collapsible-in-text">
              <h1>Design</h1>
              <br><br>
              <p>
              </p>

              <h1>Build</h1>
              <br><br>
              <p>
              </p>

              <h1>Test</h1>
              <br><br>
              <p>
              </p>

            </div>
            </div>
  
          </section>

          <!-- Crosstalk -->

          <section class="crosstalk">
          <a id="constructs" class="anchorOffset"></a>
          <h1>Reduction of crosstalk</h1>
            
            <!-- crosstalk Cycle -->
            <div class="collapsible_in">
            <input type="checkbox" id="crosstalk-cycle-circuit">
            <label for="crosstalk-cycle-circuit">Engineering Cycle</label>
            <div class="collapsible-in-text">
            <h1>Design</h1>
            <br><br>
            <p>
            </p>

            <h1>Build</h1>
            <br><br>
            <p>
            </p>

            <h1>Test</h1>
            <br><br>
            <p>
            </p>

            </div>
            </div>
          
          </section>



        <!-- Contructs  -->
        <section class="constructs">
        
        </section>

        <!-- Crosstalk -->

        <section class="crosstalk">
        
        </section>


      <!-- References -->

      <section class="collapsible" id="references">
        <input type="checkbox" id="collapsible-head">
            <label for ="collapsible-head">References</label>    
            <div class="collapsible-content">
              <cite>
                <a href="https://pubmed.ncbi.nlm.nih.gov/31488843/">
                    1.Huber M, Faure G, Laass S, Kolbe E, Seitz K, Wehrheim C, Wolf YI, Koonin EV, Soppa J. Translational coupling via termination-reinitiation in archaea and bacteria. Nat Commun. 2019 Sep 5;10(1):4006. doi: 10.1038/s41467-019-11999-9. PMID: 31488843; PMCID: PMC6728339.
      
                </a>
            </cite>
      <cite>
                <a href="https://pubmed.ncbi.nlm.nih.gov/19279094/">
                    2. Hatta M, Kohlmeier CK, Hatta Y, Ozawa M, Kawaoka Y. Region required for protein expression from the stop-start pentanucleotide in the M gene of influenza B virus. J Virol. 2009 Jun;83(11):5939-42. doi: 10.1128/JVI.00180-09. Epub 2009 Mar 11. PMID: 19279094; PMCID: PMC2681956.
      
                </a>
            </cite>
      <cite>
                <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.0020096">
                    3. Price MN, Arkin AP, Alm EJ. The life-cycle of operons. PLoS Genet. 2006 Jun;2(6):e96. doi: 10.1371/journal.pgen.0020096. Epub 2006 Jun 23. Erratum in: PLoS Genet. 2006 Jul;2(7):e126. PMID: 16789824; PMCID: PMC1480536.
                </a>
            </cite>
      
      
            <cite>
                <a href="https://pubmed.ncbi.nlm.nih.gov/30843313/">
                    4. Tutol, J., Kam, H., Dodani, S., Identification of mNeonGreen as a pH-Dependent, Turn-On Fluorescent Protein Sensor for Chloride. 2019 Jul 15;20(14):1759-1765. doi: 10.1002/cbic.201900147. Epub 2019 May 8. PMID: 30843313 PMCID: PMC6663633 DOI: 10.1002/cbic.201900147
      
      
      
                </a>
            </cite>
      
            <cite>
                <a href="https://www.nature.com/articles/s41576-021-00417-w">
                    5.  Wright BW, Molloy MP, Jaschke PR. Overlapping genes in natural and engineered genomes. Nat Rev Genet. 2022 Mar;23(3):154-168. doi: 10.1038/s41576-021-00417-w. Epub 2021 Oct 5. PMID: 34611352; PMCID: PMC8490965.
                </a>
            </cite>
      
      
      <cite>
                <a href="https://pubmed.ncbi.nlm.nih.gov/23474465/">
                    6.   Mutalik VK, Guimaraes JC, Cambray G, Lam C, Christoffersen MJ, Mai QA, Tran AB, Paull M, Keasling JD, Arkin AP, Endy D. Precise and reliable gene expression via standard transcription and translation initiation elements. Nat Methods. 2013 Apr;10(4):354-60. doi: 10.1038/nmeth.2404. Epub 2013 Mar 10. PMID: 23474465.
      
                </a>
            </cite>
      
      <cite>
                <a href="https://2018.igem.org/Team:Tsinghua/Results)">
                    7.  Team Tsinghua 2018
      
                </a>
            </cite>
      
      <cite>
                <a href="https://pubmed.ncbi.nlm.nih.gov/30478458/">
                    8.   Meyer AJ, Segall-Shapiro TH, Glassey E, Zhang J, Voigt CA. Escherichia coli "Marionette" strains with 12 highly optimized small-molecule sensors. Nat Chem Biol. 2019 Feb;15(2):196-204. doi: 10.1038/s41589-018-0168-3. Epub 2018 Nov 26. PMID: 30478458.
      
                </a>
            </cite>
      <cite>
      <a href="https://www.semanticscholar.org/paper/Primerize%E2%80%902D%3A-automated-primer-design-for-RNA-Tian-Das/acbbd0e1161646b1216143b6f304936cbe6fa78a">
                    9. Siqi Tian, Rhiju Das, Primerize-2D: automated primer design for RNA multidimensional chemical mapping, Bioinformatics, Volume 33, Issue 9, 1 May 2017, Pages 1405–1406
                </a>
            </cite>
      
      
      
      
      <cite>
                <a href="https://academic.oup.com/nar/article/43/W1/W522/2467987">
                    10. Siqi Tian, Joseph D. Yesselman, Pablo Cordero, Rhiju Das, Primerize: automated primer assembly for transcribing non-coding RNA domains, Nucleic Acids Research, Volume 43, Issue W1, 1 July 2015, Pages W522–W526
      
      
                </a>
            </cite>
      
      
            </div>
        </section>
        </div>
      </div>





<!-- Footer -->
{% include 'footer.html' %}

<!--Loader-->   
{% include 'loader.html'%}  

</html>
