<!doctype html>
<html>
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="shortcut icon" href="https://static.igem.wiki/teams/4294/wiki/favicon-white.png"/>
    <link rel="license" href="https://creativecommons.org/licenses/by/4.0/"/>
    
    <!-- Custom CSS -->
    <link href="{{ url_for('static', filename = 'css/engineering_style.css') }}" rel="stylesheet">

    <!-- Loader -->
    <script type= "text/javascript" src="../../athens/static/scripts/jquery.min.js"></script>

    <!--Custom menu CSS-->
    <link href="{{url_for('static',filename='css/menu_style_aspa.css')}}" rel="stylesheet">

    <!--Custom footer CSS-->
    <link href="{{url_for('static',filename='css/footer_style_aspa.css')}}" rel="stylesheet">

     <!--Custom to top button CSS-->
     <link href="{{url_for('static',filename='css/to_top_button.css')}}" rel="stylesheet">
    <title> Perspectives | Engineering</title>
</head>

<body>
<!-- Navigation -->
{% include 'menu_custom.html' %}

<!--To top button-->
{% include 'to_top_button.html'%}

<!--Loader-->   
{% include 'loader.html'%}  

<div class="page_header">
    <img class="img_header" src="https://static.igem.wiki/teams/4294/wiki/contribution/project-bg.gif" width="100%">
    <div class="heading">Engineering Success</div>    
</div>

<div class="buttons">
  <div class="button">
      <a href="#senders">Senders</a>
  </div>
  <div class="button">
      <a href="#receivers">Receivers</a>
  </div>
</div>

<!-- Summary -->

<div class="overview-section-senders" id="summary" >
  <h1>Engineering Success Summary</h1>
</div>

<div class="cc-summary">
  <div class="left-side">
      <div class="index-bar">
          <ul class="lateral-index">
              <li class="lateral-content">
                  <a href="#summary"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">Summary</span></a>
              </li>
              <li class="lateral-content">
                  <a href="#symmary-rec">Receivers </a>
              </li>
              <li class="lateral-content">
                  <a href="#summary-sen">Senders</a>
              </li>
              <li class="lateral-content">
                <a href="#summary-new-parts">New Parts</a>
              </li>
              <li class="lateral-content">
              <a href="#receivers"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">Receivers</span></a>
              </li>
              <li class="lateral-content">
                <a href="#senders"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">Senders</span></a>
              </li>
              <li class="lateral-content">
                <a href="#references"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">References</span></a>
            </li>
          </ul>
      </div>
  </div>
  
  <div class="right-side">

<!-- Overview  -->
          <div class="overview-summary">
              <h1>Overview</h1>
              <p>
              Our team has gone through several engineering cycles in order to establish a reliable characterization circuit for the Senders cells and a steep biological activation function in the Receiver Cells.
              <br><br>
              An overview of our successful constructs and parts is presented. For a detailed analysis of our engineering cycles, please push the respective buttons.
          
              </p>
          </div>

          <!-- Receivers Summary  -->
          <section class="symmary-rec">
            <a id="symmary-rec" class="anchorOffset"></a>
            <h1>Receivers </h1>
            <br>
            A circuit with a steeper response curve incorporating the LuxR and PhlF regulators was successfully engineered and tested. These circuits are described in detail in the “Design” tab.
            <br><br>

            <center>
            <img class="fig-a" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-a.png">
            <figcaption>
            Figure A: Comparison of OL colE1 or., OL p15 or. and PFr colE1 or. at 240 min. The uninduced cells are represented by the concentration value 0.0000001 for the diagram purposes.  
            </figcaption>
            </center>
            <br>           
                
          </section>

          <!-- Senders Summary  -->
          <section class="summary-sen">
            <a id="summary-sen" class="anchorOffset"></a>
            <h1>Senders </h1>
            <br>
            An efficient method for characterizing the relative strength of an RBS in the genetic context of LuxI synthase was built by fusing the first 36 nucleotides of the LuxI CDS with sfGFP.
            <br><br>

            <center>
            <img class="fig-b" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-b.png">
            <figcaption>
            Figure B:
            Illustration of the 36nt LuxI- sfGFP fusion reporter protein.
            </figcaption>
            </center>
            <br>      
                
          </section>

          <!-- New Parts Summary  -->
          <section class="summary-new-parts">
            <a id="summary-new-parts" class="anchorOffset"></a>
            <h1>New Parts</h1>
            <br>
            Among our new registered parts, there are two parts that, to our knowledge, have not been reported before and would be a valuable contribution to the iGEM Community.
            <br><br>
            <b><a href="http://parts.igem.org/Part:BBa_K4294207"><b>BBa_K4294207</b></a> 36nt LuxI - sfGFP fusion</b>
            <br>
            To our knowledge, the fusion of the 36 first nucleotides of the  LuxI Coding Sequence with sfGFP has not been reported before. These fusions make the characterisation of relative translation levels of LuxI synthase easier. LuxI is part of a widely used quorum sensing system and improving its characterisation strategies is important. For more information regarding the results generated with this part, refer to the <a href="https://2022.igem.wiki/athens/results">Results
            </a> pather or the <a href="https://2022.igem.wiki/athens/parts">Parts</a> registry page.
            <br><br>

            <b><a href="http://parts.igem.org/Part:BBa_K4294301"><b>BBa_K4294301</b></a> LuxR-Phlf controlled hybrid promoter</b>
            <br>
            This new promoter was used in our design attempts to create a genetic circuit with a steeper response curve. It contains a LuxR binding domain (lux box) upstream the -35 region and a Phlf operator in the core sequence that also overlaps with the -10 region. It is therefore positively regulated by LuxR and negatively by Phlf. For more information regarding the results generated with this part, refer to the Results tab or the part’s registry page.
            <br><br> 
            
            <b>Important Notes:</b>
            <br><br> 
            1. For more information about the decision of the plasmid vectors, bacterial strains and the whole genetic circuit illustrations and Genbank files, please refer to the “Design” tab.
            <br> 
            2. For the protocols mentioned below, please refer to the “Protocols” tab.
            <br> 
            3. For more detailed information about the results mentioned below, please refer to the “Results” tab.
            <br> 
            4. For the mentioned agarose gel electrophoresis results, the 1kb plus ladder of New England Biolabs was used if not noted otherwise.
            <br><br> 

            <center>
            <img class="es-fig-1" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-1.png">
            <figcaption>
            Figure 1: 1kb plus DNA ladder New England Biolabs (N3200L)
            </figcaption>
            </center>
          </section>


  </div>
</div>

<!-- Receivers  -->


<div class="overview-section-receivers" id="receivers">
  <h1>Receivers</h1>
</div>

<div class="cc-receivers">
  <div class="left-side">
    <div class="index-bar">
        <ul class="lateral-index">
            <li class="lateral-content">
                <a href="#summary"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">Summary</span></a>
            </li>
            <li class="lateral-content">
                <a href="#receivers"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">Receivers</span></a>
            </li>
            <li class="lateral-content">
              <a href="#steep-act">Activation Function</a>
            </li>
            <li class="lateral-content">
              <a href="#senders"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">Senders</span></a>
            </li>
            <li class="lateral-content">
              <a href="#references"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">References</span></a>
          </li>
        </ul>
    </div>
  </div>
  
  <div class="right-side">
<!-- Activation Function  -->
        <section class="steep-act">
          <a id="steep-act" class="anchorOffset"></a>
          <h1>Engineering a Steep Activation Function</h1>
          <br>
          One important aspect of our project was to imitate the activation function of a perceptron algorithm using a genetic circuit with a steep response curve. We focused on transcriptional regulation for the construction of a switch-like response and followed a step-by-step approach.
            <!-- 1st Cycle -->
            <div class="collapsible_in">
              <input type="checkbox" id="1st-cycle-steep-act">
              <label for="1st-cycle-steep-act">1st Cycle</label>
              <div class="collapsible-in-text">
              <h1>Design</h1>
              <p>
              Our step-by-step approach begins with the characterization of the LuxR activation system in its simplest form, an open loop circuit. 
              <br><br>
              <b>Model guided design</b>
              <br>
              In terms of dry lab simulations, before our team began with the OL construct characterization in the lab, we performed some parametric simulations with emphasis on three different design parameters; the construct’s copy number and the RBS variants used in the open loop circuit, regarding the production rates of LuxR and mNeonGreen. For that purpose, our modeling team used the whole-cell model for the OL construct, which is more analytically described in the receiver’s <a href="https://2022.igem.wiki/athens/model">Model</a> page.             
              <br>

              <center>
              <img class="fig-2" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-2.png">
              <figcaption>
              Figure 2: Parametric analysis for different Copy number values and RBS variants for the production of LuxR and mNeonGreen molecules. The abbreviations in the plot legends stand for: ‘30’=RBS BBa_B0030, ‘32’=RBS BBa_B0032, ‘34’=RBS_BBa_B0034, CN=Copy Number
              </figcaption>
              </center>
              <br>   

              From the graphs in figure 1, we can clearly observe the following
              <br><br>
              <ul>
               <li>
              Both the leakiness and final fluorescence values seem to be mainly dependent on the RBS of the mNeonGreen that would be used. So it would be wise to go with a moderate RBS variant in order to reduce the leakiness observed. As a result, we opted not to use the RBS BBa_B0030 to control the translation of LuxR.
              </li>
              <li>
              On the other hand, the RBS for the production of the LuxR seems to be directly influencing the initiation point of our function; the stronger the RBS controlling LuxR translation, the lower AHL value needed to cause induction. We noticed that the RBS ‘30’ construct is shifted to the left of the x-axis in comparison to ‘32’ and ‘34’. We decided therefore to exclude the RBS ‘30’ as a potential sequence for the production of LuxR since it might lead to a system with a response to very low OC6 concentrations, which could lead to the activation of the receivers by any combination of senders. 
              </li>
              <li>
              Finally the copy number is playing a hybrid role on our transfer functions, since it regulates not only the overall production values of mNeonGreen (including leakiness) but also leads to a minor increase in the AHL inducer concentration needed in order for the induction to start.
              </li>
              </ul>
              <br><br>
              From the analysis above, we decided to use the RBS BBa_0034 for both mNeonGreen and LuxR constructs and finalized our vector’s copy number choice with a preference towards the ColE1 origin of replication (medium CN). Both vectors will be tested to confirm the predictions. In that way, we’ve achieved a modest amount of leakiness, solid output fluorescence as well as a small shift of our graph towards the right axis. 
              <br><br>
              Unfortunately, due to the early stages of the 1st engineering cycle and a lack of experimental characterization data, a full simulation of our OL construct alongside the sender’s model as shown in the proof of concept and model pages was not implemented. Thus, that type of characterization would be still suboptimal for the entire system. However, later on, via our fitted sender’s and receiver’s models we were able to make a full simulation and show that a potential misfit of the transfer function would be overcome via a difference in the volumes of the sender’s and the receiver’s populations.
              <br><br>
                         
              <h1>Build</h1>
              <br>
                <ul>
                <li>
                We performed a Golden Gate Assembly with BsmBI restriction enzyme for each plasmid vector
                </li>
                    <ul>
                    <li>  
                    As vectors we used the BBa_K4294092 pTU2-A-RFP (colE1 origin) and the BBa_K4294093 pTU2-A-RFP (p15A origin). 
                    </li>
                    <li>
                    The parts used as inserts were the BBa_K4294806 (TU pLux Output for OL, PF, PFc) and the BBa_K4294808 (TU constitutive LuxR for OL)
                    </li>
                    </ul>
                <li>
                Transformation on <it>E. coli</it> DH5a at LB - Agar plates containing chloramphenicol. 
                </li>
                <li>
                Screening of colonies (white colonies contain the vector with the insert, red colonies the initial vector.
                </li>
                <li>
                Overnight liquid cultures from grown white colonies.
                </li>
                <li>
                Plasmid Miniprep.
                </li>
                Diagnostic Digest and PCR amplification (primers flanking the insert: Biobrick Prefix Forward Primer BBa_K4294021, pTU2 Reverse Primer  BBa_K4294032) for both Open Loop constructs.
                <li>
                Agarose gel electrophoresis.
                <br>
                Expected zones:
               
                  <ul>
                    <li>
                      Open-loop (pTU2-A-RFP colE1 origin) cut with BsaI & HindIII: 3 zones at 2027bp + 1092bp + 853bp
                    </li>
                    <li>
                      Open-loop (pTU2-A-RFP p15a origin) from PCR: 1 zone at 2045bp
                    </li>
                  </ul>
                </li>
                </ul>  
              <br>

              <center>
                <img class="fig-3" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-3.png">
                <figcaption>
                  Figure 3: Agarose Gel Electrophoresis<br>
                  L: Ladder 1kb plus<br>
                  1: Open-loop (pTU2-A-RFP colE1 origin) from the 2nd colony uncut<br>
                  2: Open-loop (pTU2-A-RFP colE1 origin) from the 1st colony cut with BsaI & HindIII<br>
                  3: Open-loop (pTU2-A-RFP colE1 origin) from the 2nd colony<br>
                  4: Open-loop (pTU2-A-RFP p15a origin) from PCR<br>
                  a: Agarose gel 1%<br>
                  b: Agarose gel 1%<br>
                  <br><br>
                  Figure 3 shows that our diagnostic digest and PCR on the receiver’s open-loop constructs resulted in the expected bands.
                </figcaption>
                </center>
                <br>              
              
              <center>
              <img class="fig-4" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-4.png">
              <figcaption>
                Figure 4:  Induction of OL constructs with different OC6 concentrations and in different timepoints. Comparison between p15A (a) origin and ColE1(b).
              </figcaption>
              </center>
              <br>

              <h1>Test</h1>
              <br>
              To test the aforementioned constructs, <it>E. coli</it> BL21 cells were transformed with the open loop constructs and were induced with a range of concentrations of Acyl Homoserine-Lactone (OC6). Cells were incubated in a shaking incubator and successive measurements of absorbance at 600nm and fluorescence were taken using the plate reader FlexStation® 3 for a time period of 4h. The results showed an increase in the RFU/OD600nm ratio and a sigmoidal response curve for both constructs.
              <br><br>

              <h1>Learn</h1>
              <br>
              These experiments prove that the ColE1 origin of replication is more appropriate for our system and, therefore, we use the medium copy number (ColE1 ori) pTU2 vector to engineer the rest of the constructs. A higher copy number provides a higher dynamic range with a minimal tradeoff regarding its leakiness. These first measurements were also important to detect the range of concentrations in which the system is inducible and update our models’ parameters.
              </p>
            </div>
            </div>

            <!-- 2nd Cycle -->
            <div class="collapsible_in">
              <input type="checkbox" id="2nd-cycle-steep-act">
              <label for="2nd-cycle-steep-act">2nd Cycle</label>
              <div class="collapsible-in-text">
              <h1>Design</h1>
              <br>
              <p>
              With the goal to build an ultrasensitive, switch-like response, and inspired by previous related work, two circuit designs in two variations regarding the basal LuxR expression were chosen. A LuxR transcriptional Positive Feedback loop (PF and PFc) and a circuit that incorporates an extra repressor (PhlF) that is regulated by LuxR (PFR, PFcR) resulting in a coherent feedforward loop with an embedded LuxR transcriptional Positive Feedback.
              <br><br>
              <b>Dry Lab Prediction</b>
              In terms of design simulations, we were able to perform a simulation of the PFR construct in parallel with the characterisation of OL constructs by the wet lab, valuable for the circuit design finalization, since a circuit incorporating such regulatory elements in the same topology has not, to our knowledge, been characterized. As analyzed in the receiver’s model section , we used some of the fitted parameters from the OpLo (OL) system and alongside the rest of the literature ones we received the following whole-cell transfer function:

              <center>
              <img class="fig-5" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-5.png">
              <figcaption>
              Figure 5: Theoretical model results for the PFR construct.
              </figcaption>
              </center>
              <br>
              
              The received curve for the PFR system displays an increased steepness and it would be the receiver’s design with the most potential.
              <br><br> 
              On the other hand, the simulations displayed for the PF and PFc results weren’t the expected ones and lead to similar results as in the case for the OL system. It should be mentioned that, for all the simulations in the PFR we used the RBS construct BBa_0034 and a medium copy number origin of replication (ColE1), from the 1st engineering cycle.
              <br><br> 
              Even though every circuit would be experimentally tested, the above simulation would aid a potential redesign of a circuit in parallel with the experiments in case of no functional circuits predicted. 
              <br><br> 
              <h1>Build</h1>
              <br>
              <ul>
                <li>
                  We performed 4 different Golden Gate Assemblies with BsmBI by using the vector BBa_K4294092 pTU2-A-RFP (colE1 origin) for all the constructs.                
                </li>
                    <ul>
                    <li>  
                      For the PF construct we used the parts:
                    </li>
                        <ul>
                          <li>
                            BBa_K4294806 (TU pLux Output for OL, PF, PFc)
                          </li>
                          <li>
                            BBa_K4294809 (TU pLux LuxR PF, PF+ R) 
                          </li>
                        </ul>

                    <li>  
                    For the PFR construct we used the parts:
                    </li>
                        <ul>
                          <li>
                            BBa_K4294812 (TU Output for PF+ R, PFc+ R)
                          </li>
                          <li>
                            BBa_K4294807 (TU pLuxr PhlF)
                          </li>
                          <li>
                            BBa_K4294809 (TU pLux LuxR PF, PF+ R) 
                          </li>
                        </ul>

                      <li>  
                        For the PFc construct we used the parts:
                      </li>
                            <ul>
                              <li>
                                BBa_K4294806 (TU pLux Output for OL, PF, PFc)
                              </li>
                              <li>
                                BBa_K4294810 (TU pLux LuxR PFc, PFc+ R)
                              </li>
                              <li>
                                BBa_K4294811 (TU constitutive LuxR for PFc, PFc+R) 
                              </li>
                            </ul>
                      <li>  
                        For the PFc+R construct we used the parts:
                      </li>
                            <ul>
                              <li>
                                BBa_K4294812 (TU Output for PF+ R, PFc+ R)
                              </li>
                              <li>
                                BBa_K4294807(TU pLuxr PhlF)
                              </li>
                              <li>
                                BBa_K4294811 (TU constitutive LuxR for PFc, PFc+R)
                              </li>
                              <li>
                                BBa_K4294810 (TU pLux LuxR PFc, PFc+ R)
                              </li>
                            </ul>


                <li>
                  Transformation on <it>E. coli<\it> DH5a at LB - Agar plates containing chloramphenicol. 
                </li>
                <li>
                  Choose the white colonies (plasmids with insert) over the red colonies (no insert in plasmid).
                </li>
                <li>
                  Overnight liquid cultures from grown white colonies
                </li>
                <li>
                  Plasmid Miniprep.
                </li>
                Diagnostic Digest and PCR amplification with primers binding upstream and downstream of the insert region for the Positive Feedback construct (pTU2-A-RFP colE1 origin), for Positive Feedback with constitutive production of LuxR construct (pTU2-A-RFP colE1 origin), for Positive Feedback + Repressor with constitutive production of LuxR construct (pTU2-A-RFP colE1 origin) and for Positive Feedback +Repressor construct (pTU2-A-RFP colE1 origin).
                <li>
                  Agarose gel electrophoresis.
                <br>
                Expected zones:
                </li>
                  <ul>
                    <li>
                      Positive Feedback construct  (pTU2-A-RFP colE1 origin) cut with BsaI: 2 zones at 2027bp + 1958bp.
                    </li>
                    <li>
                      Positive Feedback with constitutive production of LuxR construct (pTU2-A-RFP colE1 origin) cut with BsaI: 2 zones at 2959bp + 2027 bp.
                    </li>
                    <li>
                      Positive Feedback construct  (pTU2-A-RFP colE1 origin) cut with  BsaI and HindIII: 3 zones at 847 bp & 111 bp & 2027 bp.
                    </li>
                  </ul>
                </ul>  
              <br>
              
              <center>
              <img class="fig-6" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-6.png">
              <figcaption>
                Figure 6: Agarose Gel Electrophoresis<br>
                L: Ladder 1kb plus<br>
                1: PF (pTU2-A-RFP colE1 origin) cut with BsaI<br>
                2: PF (pTU2-A-RFP colE1 origin) cut with BsaI<br>
                3: PFc (pTU2-A-RFP colE1 origin) cut with BsaI<br>
                4: PFc (pTU2-A-RFP colE1 origin) cut with BsaI<br>
                5: PF (pTU2-A-RFP colE1 origin) cut with BsaI and HindIII<br>
                <br>
                Figure 6 shows that our diagnostic digest on the receiver’s PF and PFc constructs resulted in the expected bands. The zone of 111 bp is very faint in lane 5 (PF (pTU2-A-RFP colE1 origin) cut with BsaI and HindIII) due to its small size.
              </figcaption>
              </center>
              <br>

              Regarding the PFR and PFc+R constructs:<br>
                <br><br>
                Expected zones:
               
                <ul>
                  <li>
                    Positive Feedback Repressive with constitutive production of LuxR construct  (pTU2-A-RFP colE1 origin) cut with BsaI: 2 zones at 3914bp + 2027bp.
                  </li>
                  <li>
                    Positive Feedback Repressive construct (pTU2-A-RFP colE1 origin) cut with BsaI: 2 zones at 2913bp + 2027 bp.</ul>
                  </li>
                </ul>
              
              <center>
              <img class="fig-7" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-7.png">
              <figcaption>
              Figure 7: Agarose gel electrophoresis<br> 
              L: Ladder 1kb plus<br>
              1: PFR(pTU2-A-RFP colE1 origin) cut with BsaI<br>
              2: PFR(pTU2-A-RFP colE1 origin) cut with BsaI<br>
              3: Control pTU2-A-RFP colE1 origin uncut<br>
              4: Control pTU2-A-RFP colE1 origin cut with BsaI<br>
              5: PFc +R (pTU2-A-RFP colE1 origin) cut with BsaI<br>
              <br>
              Figure 7 shows that our diagnostic digest on the receiver’s PFR and PFc+R constructs resulted in the expected bands.
              </figcaption>
              </center>
              <br>

              <h1>Test</h1>
              <br>
              For the testing of the aforementioned constructs, we transformed<it>E. coli</it> BL21 and induced them with a range of concentrations of Acyl Homoserine-Lactone OC6. Absorbance and Fluorescence Measurements were taken at specific time points using the plate reader FlexStation® 3 at a total time period of 4h. 
              <br><br>
              For the PF construct the concentration range 0.062μΜ-0.000121094μΜ was used for induction.
              <br><br>

              <center>
              <img class="fig-8" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-8.png">
              <figcaption>
              Figure 8: Induction of BL21 PF pTU2-RFP colE1 or. with the following OC6 concentrations (μΜ): 0.062, 0.031, 0.0155, 0.003875, 0.0019375, 0.00096875, 0.000484375, 0.000242188, 0.000121094. The uninduced cells are represented by the concentration value 0.00001 for diagram purposes. 
              </figcaption>
              </center>
              <br>  

              Regarding the PFc construct, the concentration range 10μΜ-0.0001μΜ was used for induction. Data presented in figure 9.
              <br><br>

              <center>
              <img class="fig-9" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-9.png">
              <figcaption>
                Figure 9: Induction of BL21 PFc pTU2-RFP colE1 or. with the following OC6 concentrations (μΜ): 10, 1, 0.1, 0.01, 0.001, 0.0001. The uninduced cells are represented by the concentration value 0.000001 for diagram purposes. 
              </figcaption>
              </center>
              <br>

              For the PFr construct the concentration range 100μΜ-0.00001024μΜ was used for induction. The results that are presented in figure 10. 
              <br><br> 

              <center>
              <img class="fig-10" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-10.png">
              <figcaption>
                Figure 10: Induction of BL21 PFr pTU2-RFP colE1 or. with the following OC6 concentrations (μΜ):100, 20, 4, 0.16,    0.0064, 0.00128, 0.000256, 0.00001024. The uninduced cells are represented by the concentration value 0.000001 for diagram purposes. 
              </figcaption>
              </center>
              <br>

              For the PFc+R construct, the concentration range 10μΜ-0.0.00000512μΜ was used for induction. The results are presented in figure 11. 
              <br><br>

              <center>
              <img class="fig-11" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-11.png">
              <figcaption>
                Figure 11: Induction of BL21 PFc+R pTU2-RFP colE1 or. with the following OC6 concentrations (μΜ): 10, 2, 0.4, 0.08, 0.016, 0.0032, 0.00064, 0.000128, 0.0000256, 0.00000512. The uninduced cells are represented by the concentration value 0.0000001 for diagram purposes.
              </figcaption>
              </center>
              <br>

              <h1>Learn</h1>
              <br>
              PF: According to the results that are presented in the figure 8 there is no significant increase in RFU/OD600nm ratio in comparison with other constructs. This circuit presents an uncommon response in comparison with similar circuits from the literature. 
              <br><br> 
              PFc: According to the results that are presented in figure 9. there is a slight increase in fluorescence expressed in arbitrary units, which could, by a rough approximation, resemble a sigmoidal response curve in the 0.1 to 1 μM of inducer but in general, seems more linear. The response curve does not fulfill our goals regarding steepness and dynamic range and this result is also considered unreliable due to its uncommon response in comparison with similar circuits from the literature. 
              <br><br> 
              PFR: This circuit is characterized by a response curve with low leakiness, high dynamic range and a steeper activation function. For the transition from Min to Max state a 100-fold increase in inducer’s concentration seems necessary. However, in the sigmoid part of the response curve, the slope of the curve is gradually increasing. This is an indicator that a repeat of these measurements with more OC6 concentrations within the Min to Max range will probably better reveal the increased steepness in a more narrow OC6 concentration range. Our model supports the hypothesis of this circuit’s steeper activation function (see Model section for more).
              <br><br> 
              PFcR: This circuit topology fails to be robustly induced. An approximal sigmoidal response curve is observed in the 0.0001μM to 0.001 μM inducer range, but the dynamic range is quite low. The only 10fold increase in OC6 concentration required for a response transition from Min to Max is an indicator of a switch-like response. However, on the grounds of this low Min to Max range, it is also probable that the higher response in 0.001 μM was an artifact.
              <br><br> 
              In conclusion, the PFR and PFcR circuit architectures may provide a steeper activation function to our biological perceptron. More specifically, PFR shows the most robust response and a further tuning of its components dynamics in a model driven approach could potentially improve its response. Potential solutions include PhlF tagging with a variety of degradation tags to estimate the ideal degradation rates, a LuxR repressible promoter of different strength and modifications of the LuxR dynamics (constitutive promoter strength in PFcR, RBS controlling its translation, potential tagging with degradation tags). 
              <br> <br> 

              <center>
              <img class="fig-12" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-12.png">
              <figcaption>
              Figure 12:Comparison of OL colE1 or., OL p15 or. and PFR colE1 or. at 240 min. The uninduced cells are represented by the concentration value 0.0000001 for the diagram purposes.  
              </figcaption>
              </center>
              <br>
              </p>
            </div>
            </div>

              
          </section>

        
    </div>
  </div>


<!-- Senders -->


<div class="overview-section-senders" id="senders">
  <h1>Senders</h1>
</div>

<div class="cc-senders">
  <div class="left-side">
    <div class="index-bar">
        <ul class="lateral-index">
            <li class="lateral-content">
                <a href="#summary"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">Summary</span></a>
            </li>
            <li class="lateral-content">
                <a href="#receivers"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">Receivers</span></a>
            </li>
            <li class="lateral-content">
              <a href="#senders"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">Senders</span></a>
            </li>
            <li class="lateral-content">
              <a href="#circuit">Characterisation Circuit</a>
            </li>
            <li class="lateral-content">
                <a href="#strategy">Characterisation Strategy</a>
            </li>
            <li class="lateral-content">
              <a href="#constructs">Final Constructs</a>
            </li>
            <li class="lateral-content">
              <a href="#crosstalk">Crosstalk Reduction</a>
            </li>
            <li class="lateral-content">
              <a href="#references"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">References</span></a>
          </li>
        </ul>
    </div>
  </div>
  
  <div class="right-side">

          <!-- Circuit  -->
          <section class="circuit">
          <a id="circuit" class="anchorOffset"></a>
          <h1>Sender Characterization Circuit</h1>
          <br>
          Our project involves the use of  Ribosome Binding Sites to obtain a translation rate gradient of the Acyl-homoserine-lactone synthase, (<i>luxI</i> gene). This translation rate corresponds to the different weights of the activation of each sender subpopulation. 
          <br><br>
          We should figure out a way to measure the translation rate that each RBS variant provides. For that purpose, we added a C-terminal hexahistidine tag (6*His tag) to the LuxI synthase, suitable for protein isolation and quantification using affinity chromatography or anti-histidine antibodies for Western Blotting. However, the cost and time needed to perform such quantifications is a considerable issue of the aforementioned methods and, therefore, we decided to characterise our parts using fluorescent proteins.   

            <!-- 1st Cycle -->
            <div class="collapsible_in">
              <input type="checkbox" id="1st-cycle-circuit">
              <label for="1st-cycle-circuit">1st Cycle</label>
              <div class="collapsible-in-text">
              <h1>Design</h1>
              <br>
              <p>
              The goal of the sender characterisation circuit was the characterisation of the different translation rates of the LuxI synthase provided by our Ribosome Binding Site collection. Our first design incorporated a bicistronic module including the <it> luxI </it> coding sequence followed by the mNeonGreen coding sequence. The two coding sequences are separated by the stop-start pentanucleotide TAATG, resulting in the translation coupling of Acyl-homoserine-lactone synthase and mNeonGreen. Relative fluorescent units would be an indirect indicator of Acyl-homoserine-lactone synthase production levels and RBS strength[1,2,3].
              <br><br>

              <center>
              <img class="fig-13" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-13.png">
              <figcaption>
              Figure 13: First characterization circuit overview. The translational coupling of the LuxI CDS with the mNeonGreen CDS via a stop-start pentanucleotide.
              </figcaption>
              </center>
              <br> 

              <h1>Build</h1>
              <br>
              <b>Level 1 constructs:</b>
              <br><br>
              <ul>
                <li>
                  We performed Golden Gate Assemblies with BsaI using for every construct :
                </li>
                    <ul>
                    <li>  
                      As vector we used the  BBa_K4294091 (pTU1-A-lacΖ).
                    </li>
                    <li>
                      As inserts we used the parts: BBa_K3424010 (TetR promoter), BBa_K4294210 (LuxI and mNeonGreen coding sequences separated by the stop-start pentanucleotide "TAATG"), BBa_B0015 (Double Terminator).
                    </li>
                    <li>
                      And an RBS that was different for each construct. The RBSs was BBa_K4294004 (S4), BBa_K4294005 (S5), BBa_K4294010 (S10), BBa_K4294011 (S11), BBa_K4294012 (S12), BBa_K4294013 (S13), BBa_J428034 (BCD1),BBa_K4294102 (BCD2), BBa_K2680561 (BCD8), BBa_K2680529 (BCD12), BBa_K4294114 (BCD14).
                    </li>
                    </ul>
                <li>
                  Transformation of the recombinant plasmid on <it>E. coli</it> DH5a at LB - Agrar - Ampicillin plates whith IPTG and x-Gal for Blue-White screening.
                  </li>
                <li>
                  Selection of the white colonies (plasmids with insert) over the blue colonies (no insert in plasmid).
                </li>
                <li>
                  Overnight liquid cultures from grown white colonies.
                </li>
                <li>
                Plasmid Miniprep.
                </li>
                <li>
                Diagnostic Digest for all of the aforementioned level 1 constructs.
                </li>
                <li>
                Agarose gel electrophoresis.
                <br>
                Expected zones:
               
                  <ul>
                    <li>
                      Constructs with S4,S5,S10,S11,S13 (pTU1-A-lacZ) cut with NotI &PstI: 2 zones at 2061bp + 1682bp.
                    </li>
                    <li>
                      Constructs with  S12 RBS (pTU1-A-lacZ) cut with NotI &PstI: 2 zones at 2061bp + 1681bp.
                    </li>
                    <li>
                      Constructs with BCD1, BCD8, BCD12 (pTU1-A-lacZ) cut with NotI &PstI: 2 zones at 2061bp + 1737bp.
                    </li>
                    <li>
                      Constructs with BCD2, BCD14 (pTU1-A-lacZ) cut with NotI & PstI: 2 zones at 2061bp + 1741bp.
                    </li>
                  </ul>
                </li>
              </ul>  
              <br>

              <center>
                <img class="fig-14" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-14.png">
                <figcaption>
                  Figure 14:<br>
                  L: Ladder 1kb plus<br>
                  1: BCD2 level 1 cut with NotI & PstI<br>
                  2: BCD14 level 1 cut with NotI & PstI<br>
                  3: S4 level 1 cut with NotI & PstI<br>
                  4: S5 level 1cut with NotI & PstI<br>
                  5: S10 level 1 cut with NotI & PstI<br>
                  6: S11 level 1 cut with NotI & PstI<br>
                  7: S12 level 1 cut with NotI & PstI<br>
                  8: S13 level 1 cut with NotI & PstI<br>
                  9: Control - BCD2 level 1 uncut<br>
                  10: Control - S4 level 1 uncut<br>
                  11: Control pTU1 uncut<br>
                  12: BCD1 level 1 cut with NotI & PstI<br> 
                  13: BCD8 level 1 cut with NotI & PstI<br>
                  14: BCD12 level 1 cut with NotI & PstI<br>
                  15: Control - BCD1 level 1 uncut<br>
                  16: Control - BCD1 level 1 cut with BsaI<br>
                  17: Control - pTU1-A-lacz uncut<br>
                  18: Control - pTU1-A-lacz cut with BsaI<br>
                  19: Control - pTU1-A-lacz cut with NotI & PstI<br>
                  a: Agarose gel 1,2%<br>
                  b: Agarose gel 1,2%<br>
                  <br><br>
                </figcaption>
                </center>
                <br>  
                Figure 14 shows that our diagnostic digest on the sender’s constructs resulted in the expected bands. The diagnostic digest and the electrophoresis for the samples with faint bands were repeated to ensure they are correct.
                <br><br>   

                <b>Level 2 constructs:
                </b>
                <ul>
                  <li>
                    We performed Golden Gate Assemblies with BsmBI using for every construct:
                  </li>
                      <ul>
                      <li>  
                        As vector we used  BBa_K4294093 (pTU2-A-RFP (p15A origin)).
                      </li>
                      <li>
                        As inserts we used :The composite part BBa_K4294780 (TetR TU).
                      </li>
                      <li>
                        And the composite parts that resulted from level 1 Golden Gate Assembly and were isolated by plasmid minipreps. Each of these was used to build a new level 2 construct. For S4 level 2 the BBa_K4294981, for S5 level 2 the BBa_K4294982, for S10 level 2 the BBa_K4294983, for S11 level 2 the BBa_K4294984, for S12 level 2 the BBa_K4294985, for S13 level 2 the BBa_K4294986, for BCD1 Level 2 the BBa_K4294987, for BCD2 level 2 the BBa_K4294988, for BCD8 level 2 the BBa_K4294989, for BCD12 level 2 the BBa_K4294990 and for BCD14 level 2 the BBa_K4294991.
                      </li>
                      </ul>
                  <li>
                    Transformation of the recombinant plasmids on <it>E. coli</it> DH5a at LB - Chloramphenicol plates.
                    </li>
                  <li>
                    Selection of the white colonies (plasmids with insert) over the red colonies (no insert in plasmid).
                  </li>
                  <li>
                    Overnight liquid cultures from grown white colonies.
                  </li>
                  <li>
                  Plasmid Miniprep.
                  </li>
                  Diagnostic Digest and PCR amplification for all of the constructs.
                  <li>
                  Agarose gel electrophoresis.
                  <br>
                  Expected zones:
                 
                    <ul>
                      <li>
                        Constructs with S4,S5,S10,S11,S13  - pTU2-A-RFP (p15a origin)
                        cut with BsaI: 2 zones at 2474bp + 2173bp.
                                              </li>
                      <li>
                        Constructs with  S12 RBS  - (pTU2-A-RFP (p15a origin) cut with BsaI: 2 zones at 2474bp + 2172bp.

                      </li>
                      <li>
                        Constructs with BCD1, BCD8, BCD12  - pTU2-A-RFP (p15a origin)
                        cut with BsaI:  2 zones at 2474bp + 2228bp.
                                              </li>
                      <li>
                        Constructs with BCD2, BCD14 -  pTU2-A-RFP (p15a origin)
                        cut with BsaI: 2 zones at 2474bp + 2232bp.
                                              </li>
                    </ul>
                  </li>
                   
                <br>
  
                <center>
                  <img class="fig-14" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-14.png">
                  <figcaption>
                    Figure 15:<br>
                    L: Ladder 1kb plus<br>
                    1: S4 level 2 cut with BsaI<br>
                    2: S5 level 2 cut with BsaI<br>
                    3: S10 level 2 cut with BsaI<br>
                    4: S11 level 2 cut with BsaI<br>
                    5: S12 level 2 cut with BsaI<br>
                    6: S13 level 2 cut with BsaI<br>
                    7: Control - S13 uncut<br>
                    8: Control - S5 cut with BsmBI<br>
                    9: Control - pTU2-A-RFP (p15a origin) uncut<br>
                    10: Control - pTU2-A-RFP (p15a origin) cut with BsaI<br>
                    11: Control S4 uncut <br>
                    12: Control S4 cut with HindIII<br>
                    13: BCD1 cut with BsaI<br>
                    14: BCD2 cut with BsaI<br>
                    15: BCD8 cut with BsaI<br>
                    16: BCD12 cut with BsaI<br>
                    17: BCD14 11: cut with BsaI<br>
                    18: Control - BCD1 uncut<br>
                    19: Control pTU2-A-RFP cut with BsaI<br>
                  </figcaption>
                  </center>
                  <br>  
                  Figure 15 shows that our diagnostic digest on the level 2 sender’s constructs resulted in the expected bands. The diagnostic digest and the electrophoresis for the samples with faint bands were repeated to ensure they are correct. 
                  <br><br>   
                  <li>
                    Transformation of <it>E. coli</it> BL21with the miniprep of level 2 recombinant plasmid on plates containing chloramphenicol.
                  </li>
                </ul> 
                <br>
              <h1>Test</h1>
              <br>
              For the testing of the aforementioned Level 2 constructs, we selected white <it>E. coli</it> BL21 colonies (plasmid with insert), and no red colonies (plasmid without insert), from Petri dishes and we made overnight liquid cultures. The next day, we diluted them 1:30, and then when they reached OD: 0,3-0,6 we induced them with multiple concentrations of inducer Anhydrotetracycline hydrochloride- aTc. Cells were incubated in a shaking incubator for 4 hours. At different time points within the 4 hours, a sample of each culture was centrifuged, washed and resuspended in PBS.200 μL of each sample was transferred into a 96-well plate (black with clear bottom). By using the plate reader FlexStation® 3 (Molecular Devices), absorption (OD600nm) and the fluorescence measurements were taken. Τhe ratio of RFU/ OD600nm did not increase significantly over time. Measurements of mNeonGreen protein fluorescence were performed with excitation and emission filters set at 476nm and 547nm respectively, as recommended by the plate reader manufacturer for fluorescent proteins with significant overlap regarding their emission and excitation wavelengths. According to the fluorescent protein database, mNeonGreen shows maximum excitation at 506nm and emission at 517nm.
              <br><br>
              A potential malfunction of our circuit would lie in the overlap of transcriptional units in our bicistronic design (LuxI CDS - TAATG stop/start pentanucleotide - mNeonGreen CDS). To rule out a suspected TetR induction system malfunction (which was also latter revealed during our troubleshooting), we transformed DH5a-z1 cells (they produce TetR constitutively encoded in their genome) with our pTU1 Level 1 recombinant plasmids, which only contain the Ptet regulated transcriptional. Fluorescence did not increase in this way either, pointing towards not functional characterization circuit. (Figure 16)
              <br><br>

              <center>
              <img class="fig-16" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-16.png">
              <figcaption>
                Figure 16: Testing senders DH5a-z1 pTU1- luxI mNeonGreen with multiple aTc concentrations (2.16μΜ,0.4μΜ,0.09μΜ). This induction resulted in a very low fluorescence signal, a maximum of 340 RFU (ratio) 20h post-induction.
              </figcaption>
              </center>
              <br> 

              It is also important to mention that according to Tutol et al. (2019) chloride has an effect on mNeonGreen folding . We therefore tried changing the buffer that we resuspended the pellets for measurement to rule out this, quite improbable, scenario. The results were consistent with every buffer as indicated in Figure 17 [4].
              <br><br>

              <center>
              <img class="fig-17" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-17.png">
              <figcaption>
                Figure 17:Testing the effect of chloride in the resuspension buffer on mNeonGreen Fluorescence. No significant effect of the buffers was observed indicating to the absent expression of mNeonGreen.
              </figcaption>
              </center>
              <br>

              After that, we also assessed the production of LuxI and mNeonGreen with SDS-PAGE analysis and in-gel fluorescence of cell lysates in two cell types BL21(DE3) (level2 construct) and DH5a-z1 (level 1 construct) after a 3h induction with aTc, centrifugation and sonication of the cells. The in-gel fluorescence did not produce any results except for the positive control (OL ColE1 - induced). That was the confirming sign that the bicistronic design wasn't working. The SDS-PAGE confirmed that our constructs did not work in BL21 cells but worked in a leaky way in DH5a-z1. This was the first sign that made us realize there was probably a problem with the chosen promoter for the TetR repressor gene as well, since LuxI was not produced in neither the induced nor the uninduced BL21 cells containing the plasmid-encoded TetR repressor.
              <br><br>

              <center>
                <img class="fig-18" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-18.png">
                <figcaption>
                  Figure 18: SDS-PAGE: OL ColEI for positive control for mNeonGreen. S4- DH5a-z1 produces luxI but not mNeon Green both at induced and uninduced states. S4-BL21 is not producing luxI or mNeonGreen with or without induction.
                </figcaption>
                </center>
                <br>

              Last but not least, in an attempt to fully comprehend the multiple issues and troubleshoot, we tested the potential expression of LuxI by the LuxI-mNeonGreen CDS overlap construct by observing the effect of the supernatant of induced (after a 3h induction with aTc) BL21 sender cells with the LuxI-mNeonGreen CDS overlap on the output of receiver BL21 cells transformed with the OL-ColE1 construct. This experiment proved that the LuxI TU is functional and OC6 is produced to activate the receivers. Receiver’s induction was nevertheless relatively low (Figure 19) (only a 4fold increase in contrast with , correlating with the results of SDS-PAGE, judging by the RFU for mNeonGreen produced by the receivers when induced externally with pure OC6) and made us realize the problem is also located in the bicistronic design with mNeonGreen.
              <br><br>

              <center>
                <img class="fig-19" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-19.png">
                <figcaption>
                  Figure 19: We tested the effect of the supernatant of BL21 luxI-mNeonGreen for S4 RBS after 3h induction with aTc on the Receivers construct OL-ColE1 (BL21). This experiment proved that the LuxI TU is functional and can produce OC6 to activate the receivers. 
                </figcaption>
                </center>
                <br>


                Please refer to the <a href="https://2022.igem.wiki/athens/results">Results</a> page in order to see more details on the aforementioned experiments.
                <br><br>
              <h1>Learn</h1>
              <br>

              Our first characterization design was proven to be unable to function. Possible changes that could be made are different stop-start codons, such as ATGA or TGATG [1] , and the introduction of an intragenic Shine Dalgarno sequence at the end of the LuxI coding sequence[5]. However, such changes are quite complex from a design and a modelling aspect. Therefore, we came up with a simpler solution as described in the 2nd Cycle.
              <br><br>
              Furthermore, a strong suspicion that the TetR induction system was not functioning properly arose during the 1st cycle’s experiments.
              
              </p>
            </div>
            </div>

            <!-- 2nd Cycle -->
            <div class="collapsible_in">
              <input type="checkbox" id="2nd-cycle-circuit">
              <label for="2nd-cycle-circuit">2nd Cycle</label>
              <div class="collapsible-in-text">
              <h1>Design</h1>
              <br>
              <p>
              Ribosome binding sites are quite variable parts and their behaviour is strongly influenced by the upstream and downstream genetic context. For example, a binding interaction with the first nucleotides of the coding sequence might trap the RBS inside a stem-loop, hindering Ribosome binding and translation initiation. This means that characterizing our synthetic RBS collection simply by using a fluorescent protein’s CDS would not be the right approach, since the genetic context in such characterisation would be different from the genetic context of the LuxI coding region. 
              <br><br>
              In the research paper by Mutalik et. al [6], bicistronic-like translation initiation elements (the BCDs, that we have also included in our parts) were evaluated for their performance variability regarding the genetic context at the 5’UTR:CDS junction. Different genetic contexts were introduced by assembling a test panel of 14 chimeric reporter genes of interest (GOIs) by fusing the first 36 nucleotides from the coding sequences of several transcription factors or enzymes in-frame to the second codon of a gene encoding GFP or RFP. These 36 nucleotides are considered sufficient to imitate the genetic context of the CDS of origin and at the same time do not alter the function of the sfGFP, making these constructs a reliable way to characterise RBS performance in different genetic contexts.
              <br><br>
              We deployed this strategy by creating a new fusion. The 36 first nucleotides of the LuxI synthase were fused with the sfGFP coding sequence, creating a new characterisation part. 
              <br><br> 

              <center>
              <img class="fig-20" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-20.png">
              <figcaption>
                Figure 20:
                Illustration of the 36nt LuxI- sfGFP fusion reporter protein.
              </figcaption>
              </center>
              <br>

              <h1>Build</h1>
              <br>
              To construct the 36nt LuxI- sfGFP CDS, the CDS of sfGFP was ordered as a gBlock from IDT. Primers (BBa_K4294082, BBa_K4294083) were used to add overhangs to the sfGFP CDS containing BsaI cut sites. The 5’ overhang would be 5’-CAAA-3’ to attach with the 36nt LuxI Double Stranded Oligo and the 3’ overhang would be 5’-GCTT-3’ to attach with the Terminator.
              <br><br>
              The small 36nt double stranded oligo was constructed via primer annealing and extension following the protocol recommended from the iGEM registry (Knight:Annealing and primer extension with Taq polymerase). Primers BBa_K4294061 and BBa_K4294062 were used to construct the final oligo. The oligo includes BsaI cut sites on both sites. The 5’ overhang would be 5’ AATG 3’ to attach with the RBS and the 3’ overhang would be 5’CAAA3’ to attach with the sfGFP CDS.
              <br><br>

              <b>Level 1 constructs:</b>
              <br><br>
              <ul>
                <li>
                  We performed Golden Gate Assemblies with BsaI using for every construct :
                </li>
                    <ul>
                    <li>  
                      The vector BBa_K4294091 (pTU1-A-lacZ).
                    </li>
                    <li>
                      The parts BBa_K3424010 (TetR promoter), BBa_K4294210 (LuxI codon optimised for E. Coli (+6His Tag)), BBa_K4294207 (36nt LuxI - sfGFP), BBa_B0015 (Double Terminator).
                    </li>
                    <li>
                      An RBS that was different for each construct. The RBSs was BBa_K4294004 (S4), BBa_K4294005 (S5), BBa_K4294010 (S10), BBa_K4294011 (S11), BBa_K4294012 (S12), BBa_K4294013 (S13), BBa_J428034 (BCD1),BBa_K4294102 (BCD2), BBa_K2680561 (BCD8), BBa_K2680529 (BCD12), BBa_K4294114 (BCD14).
                    </li>
                    </ul>
                <li>
                  Transformation of the recombinant plasmid on <it>E. coli</it> DH5a at LB - Ampicillin plates with IPTG and xGal for Blue-White screening.
                  </li>
                <li>
                  Selection of the white colonies (plasmids with insert) over the blue colonies (no insert in plasmid).
                  <center>
                    <img class="fig-21" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-21.png">
                    <figcaption>
                      Figure 21: Agar plate with Ampicillin, X-Gal and IPTG for blue-white screening with transformed <it>E. Coli <\it> DH5a colonies with BCD8 level 1 luxI36nt-sfGFP construct on pTU1-A-lacZ. We observe it under UV light.
                    </figcaption>
                    </center>
                </li>
                <li>
                  Overnight liquid cultures from grown white colonies from bacteria transformed with BCD2, BCD8, BCD14, S11, S5, and S13 level 1 constructs.
                </li>
                <li>
                  Plasmid Miniprep from the above liquid cultures.
                </li>
                <li>
                  Standard PCR and Colony PCR with Taq polymerase for all of the constructs by using the primers:<br>
                  BBa_K4294031 (pTU1 Reverse).<br>
                  BBa_K4294021 (Biobrick Prefix Forward).<br>                
                </li>
                <li>
                Agarose gel electrophoresis.
                <br>
                Expected zones:
               
                  <ul>
                    <li>
                      PCR products from constructs with S4,S5,S10,S11,S13 RBS (pTU1-A-lacZ): 1 band at 1134bp.
                    </li>
                    <li>
                      PCR products from constructs with  S12 RBS (pTU1-A-lacZ): 1 band at 1133bp.
                    </li>
                    <li>
                      PCR products from constructs with BCD1, BCD8, and BCD12 RBS (pTU1-A-lacZ): 1 band at 1189bp.
                    </li>
                    <li>
                      PCR products from constructs with BCD2, BCD14 RBS: 1 band at 1193bp.
                    </li>
                  </ul>
                </li>
              </ul>  
              <br>

              <center>
                <img class="fig-22" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-22.png">
                <figcaption>
                  Figure 22:<br>
                  L: Ladder 1kb plus<br>
                  1: pTU1-A-lacz amplified by standard PCR<br>
                  2: BCD2 36nt LuxI-sfGFP construct amplified by standard PCR<br>
                  3: BCD8 36nt LuxI-sfGFP construct amplified by standard PCR<br>
                  4: BCD8 36nt LuxI-sfGFP construct amplified by standard PCR<br>
                  5: BCD14 36nt LuxI-sfGFP construct amplified by standard PCR<br>
                  6: BCD2 36nt LuxI-sfGFP construct amplified by standard PCR<br>
                  7: S11 36nt LuxI-sfGFP construct amplified by standard PCR<br>
                  8: S4 36nt LuxI-sfGFP construct amplified by standard PCR<br>
                  9: S5 36nt 36nt LuxI-sfGFP construct amplified by standard PCR<br>
                  10: S5 36nt 36nt LuxI-sfGFP construct amplified by standard PCR<br>
                  11: S13 36nt LuxI- sfGFP construct amplified by standard PCR<br>
                  12: Control - Negative<br>                       
                </figcaption>
                </center>
                <br>  

              From the solid culture of S12, BCD8, S5, BCD12 and S10 we applied colony PCR.
              <br>  <br> 

              <center>
                <img class="fig-23" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-23.png">
                <figcaption>
                  Figure 23:<br>
                  L: Ladder 1kb plus<br>
                  1: S12 36nt LuxI-sfGFP construct amplified by colony PCR<br>
                  2: S12 36nt LuxI-sfGFP construct amplified by colony PCR<br>
                  3: S12 36nt LuxI-sfGFP construct amplified by colony PCR<br>
                  4: BCD8 36nt LuxI-sfGFP construct amplified by colony PCR<br>
                  5: BCD8 36nt LuxI-sfGFP construct amplified by colony PCR<br>
                  6: BCD8 36nt LuxI-sfGFP construct amplified by colony PCR<br>
                  7: S5 36nt LuxI-sfGFP construct amplified by colony PCR<br>
                  8: S5 36nt LuxI-sfGFP construct amplified by colony PCR<br>
                  9: S5 36nt LuxI-sfGFP construct amplified by colony PCR<br>
                  10: BCD12 36nt LuxI-sfGFP construct amplified by colony PCR<br>
                  11: BCD12 36nt LuxI-sfGFP construct amplified by colony PCR<br>
                  12: S10 36nt LuxI-sfGFP construct amplified by colony PCR<br>
                  13: S10 36nt LuxI-sfGFP construct amplified by colony PCR<br>
                  14: S10 36nt LuxI-sfGFP construct amplified by colony PCR<br>
                  15: Control - Negative <br>            
                </figcaption>
                </center>
                <br>  
                As shown in Figures 22 and 23, our colony PCR on the level 1 sender’s constructs resulted in the expected bands for some colonies, that we subsequently picked to start liquid cultures and isolate their plasmids (miniprep).
                <br><br>   
                <li>
                  So, we did plasmid minipreps using all the bacteria strains that contained the desired constructs and we continued with the construction of level 2 plasmids.
                </li>
              <br><br>
              
            
            </ul> 
                <b>Level 2 constructs:
                </b>
                <ul>
                  <li>
                    We performed Golden Gate Assemblies with BsmBI restriction enzyme, for every construct we used:
                  </li>
                      <ul>
                      <li>  
                        The vector BBa_K4294093 (pTU2-A-RFP (p15A origin)).
                      </li>
                      <li>
                        The composite part BBa_K4294780 (TetR TU).
                      </li>
                      <li>
                        The composite parts that resulted from level 1 Golden Gate Assembly and were isolated by plasmid minipreps. Each of these was used to build a new level 2 construct. For S4 level 2 we used the BBa_K4294754, for S5 level 2 we used the BBa_K4294755, for S10 level 2 we used the BBa_K4294760, for S11 level 2 we used the BBa_K4294761, for S12 level 2 we used the BBa_K4294762, for S13 level 2 we used the BBa_K4294763, for BCD1 Level 2  we used the BBa_K4294771, for BCD2 level 2 we used the BBa_K4294772, for BCD8 level 2 we used the BBa_K4294773, for BCD12 level 2 we used the BBa_K4294774 and for BCD14 level 2 we used the BBa_K4294775.
                      </li>
                      </ul>
                  <li>
                    Transformation of the recombinant plasmids on <i> E. coli </i> DH5a at LB - Chloramphenicol plates.                    
                    <center>
                      <img class="fig-24a" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-24a.png">
                      <img class="fig-24b" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-24b.png">
                      <figcaption>
                        (a)Agar plate with Chloramphenicol. We can observe transformed <it> E. coli </it>DH5a colonies with BCD1 level 2 LuxI36nt-sfGFP construct on pTU2-A-RFP.<br>
                          (b) Agar plate with Chloramphenicol. We can observe transformed <it>E. Coli </it> DH5a colonies with BCD2 level 2 LuxI36nt-sfGFP construct on pTU2-A-RFP under UV light.
                      </figcaption>
                      </center>
                  </li>

                  <li>
                    Selection of the white colonies (plasmids with insert) over the red colonies (no insert in plasmid).
                  </li>
                  Colony PCR amplification for all of the constructs by using the primers:<br>
                  BBa_K4294032 (pTU2 Reverse).<br>
                  BBa_K4294021 (Biobrick Prefix Forward).<br>       
                  <li>
                  Agarose gel electrophoresis.
                  <br>
                  Expected zones:
                 
                    <ul>
                      <li>
                        PCR products from constructs with S4,S5,S10,S11,S13 RBS (pTU2-A-RFP) 1 band at 2042bp.
                      </li>
                      <li>
                        PCR products from constructs with  S12 RBS (pTU2-A-RFP) 1 band at 2041bp.                        
                      </li>
                      <li>
                        PCR products from constructs with BCD1, BCD8, BCD12 RBS (pTU2-A-RFP) 1 band at 2097bp.
                      </li>
                      <li>
                        PCR products from constructs with BCD2, BCD14 RBS (pTU2-A-RFP) 1 band at 2101bp.
                      </li>
                    </ul>
                  </li>
                   
                <br>
  
                <center>
                  <img class="fig-25" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-25.png">
                  <figcaption>
                    Figure 25:<br>
                    L: Ladder 1kb plus<br>
                    1: BCD8 level 2 36nt LuxI-sfGFP construct amplified by colony PCR<br>
                    2: BCD2 level 2 36nt LuxI-sfGFP construct amplified by colony PCR<br>
                    3: S12 level 2 36nt LuxI-sfGFP construct amplified by colony PCR<br>
                    4: BCD1 level 2 36nt LuxI-sfGFP construct amplified by colony PCR<br>
                    5: S10 level 2 36nt LuxI-sfGFP construct amplified by colony PCR<br>
                  </figcaption>
                  </center>
                  <br>  

                </li>
                  As shown in Figures above, our electrophoresis on PCR products for the level 2 senders’ constructs resulted in the expected bands.  
                  <br><br>   
                  <li>
                    Performed plasmid minipreps from all the aforementioned level 2 senders’ constructs.
                  </li>
                  <li>
                    Transform  <it>E. Coli </it>  BL21 with the recombinant level 2 plasmids from the minipreps on plates containing chloramphenicol.
                  </li>
                </ul> 
                <br><br>

                We were happy to observe our BL21 colonies on agar plates to be fluorescent, indicating the production of a functional fluorescent protein.
                <br><br>
                <center>
                  <img class="fig-26" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-26.png">
                  <figcaption>
                    Figure 26: Agar plate with Chloramphenicol in which they grew transformed E. coli BL21 colonies with BCD2 level 2 construct on pTU2-A-RFP (p15a origin).
                  </figcaption>
                  </center>
                  <br>

              <h1>Test</h1>
              <br>
              For the testing of the aforementioned Level 2 constructs, we selected white <i>E. coli </i> BL21 colonies (plasmid with insert), and no red colonies (plasmid without insert), from Petri dishes and we made overnight liquid cultures. The next day, we diluted them 1:30, and then when they reached OD: 0,3-0,6 we induced them with multiple concentrations of inducer Anydrotetracycline - aTc. Then, we transferred 200 μL of each concentration of inducer with the liquid culture from each construct into a 96-well plate (black with clear bottom) and incubated them in a shaking incubator for 4 hours (180rpm,37oC). By using the plate reader FlexStation® 3, we measured the absorption and the fluorescence of the cells. The RFU/OD600nm ratio did not increase. Measurements for sfGFP protein were performed with an excitation wavelength 485nm and emission wavelength 510nm according to the fluorescence protein database. 
              <br><br>

              Τhen, we hypothesized that the BL21 that we had at our disposal might not be the appropriate strain for our constructs to be expressed. For this reason, we induced DH5a cells with the level 2 constructs of 36ntLuxI-sfGFP. However, this did not produce any positive results (Figure 27).
              <br><br>

              As a result, we concluded that the green fluorescent colour (Figure 26) of the colonies is due to the leakiness of the pTet promoter or negligible repression by TetR due to unsuitable dynamics or a loss of function error in its coding sequence.
              <br><br>

              <center>
                <img class="fig-27" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-27.png">
                <figcaption>
                  Figure 27: Induction experiment of BL21 luxI 36nt -sfGFP. Example for BCD2. The cells produced a stable amount of fluorescence expressed in RFU that did not change after induction.
                </figcaption>
                </center>
                <br>

              <h1>Learn</h1>
              <br>
              The results pointed out that the 36ntluxI-sfGFP fusion is fluorescent and can be measured, but there is an issue regarding the TetR induction system. One possible aspect is that TetR is excessively produced (it is placed downstream of the medium strength Anderson constitutive promoter J23105). Indeed, repressors’ dosage can alter the response to the inducer and even hinder it completely, as was described by team Tsinghua 2018 for the LacI repressor [7] . We had a last solution in mind; to change the promoter responsible for the constitutive TetR production.

              <p>
              </p>
            </div>
            </div>

            <!-- 3rd Cycle -->
            <div class="collapsible_in">
              <input type="checkbox" id="3rd-cycle-circuit">
              <label for="3rd-cycle-circuit">3rd Cycle</label>
              <div class="collapsible-in-text">
              <h1>Design</h1>
              <br>
              <p>
              The paper on the <it>E. coli</it> Marionette Strains [8], provides a detailed characterisation and directed evolution optimisation of many common induction systems. Marionette strains originate from BL21 strains. Therefore, we decided to deploy their system for TetR production in our BL21 cells, since it was already tested and proven functional. TetR is under the control of the constitutive PlacI promoter and a synthetic RBS. 
              <br><br>

              <center>
              <img class="fig-28" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-2.png">
              <figcaption>
              Figure 28: Alternative TetR protein generator
              </figcaption>
              </center>
              <br>

              <h1>Build</h1>
              <br>

              <ul>
                <li>
                  We constructed PlacI-tet1 sequence with flanking BsmBI cut sites using the Primerize [9,10] online software for PCR assembly with the primers BBa_K4294041, BBa_K4294042, BBa_K4294043, BBa_K4294044 designed by the same software.
               </li>
                <li>
                Agarose gel electrophoresis.
                <br>
                  The expected band is at 109bp.
                  <center>
                    <img class="fig-29" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-29.png">
                    <figcaption>
                      Figure 29:<br>
                      L: Ladder 1kb plus<br>
                      1: Control - negative<br>
                      2 and 3: PlacI-tet1 amplified by PCR<br>
                                             
                    </figcaption>
                    </center>
                    <br>
                    As shown in Figure 29, our electrophoresis on PlacI-tet1 resulted in the expected bands.   
                </li>
                <li>
                  Primers with BsmBI overhangs were used to isolate TetR CDS (BBa_K4294051, BBa_K4294052) and the following terminator from an already assembled Level 2 plasmid so as to add the suitable overhangs for assembly with the PlacI-tet1 part and the pTU2 vector in the following Golden Gate Assembly. The TetR CDS - Terminator was isolated with gel extraction.
                </li>
                <li>
                  We performed Golden Gate Assemblies with BsmBI using for every construct:
                </li>
                <ul>
                  <li>
                    The vector pTU2-A-RFP (p15a origin).
                  </li>
                  <li>
                    The BBa_K4294998 (PlacI-tet1).
                  </li>
                  <li>
                    The  BBa_K4294215 (TetR CDS).
                  </li>
                  <li>
                    The composite parts that resulted from the level 1 Golden Gate Assembly of the level 2 engineering cycle and were isolated by plasmid minipreps. Each of these was used to build a new level 2 construct. For S4 level 2 we used the BBa_K4294754, for S5 level 2 we used the BBa_K4294755, for S10 level 2 we used the BBa_K4294760, for S11 level 2 we used the BBa_K4294761, for S12 level 2 we used the BBa_K4294762, for S13 level 2 we used the BBa_K4294763, for BCD1 Level 2  we used the BBa_K4294771, for BCD2 level 2 we used the BBa_K4294772, for BCD8 level 2 we used the BBa_K4294773, for BCD12 level 2 we used the BBa_K4294774 and for BCD14 level 2 we used the BBa_K4294775.
                  </li>
                </ul>
              <li>
                Transformation of the recombinant plasmids on <i>E. coli<\i> DH5a at LB - Chloramphenicol plates.
                  <center>
                  <img class="fig-30" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-30.png">
                  <figcaption>
                    Figure 30:
                    Agar plate with Chloramphenicol. We can observe transformed <i>E. coli<\i> DH5a colonies with BCD12 LuxI36nt-sfGFP with the new promoter level 2 construct on pTU2-A-RFP, under UV light.                
                  </figcaption>
                  </center>
                    <br>
              </li>
              <li>
                Selection of the white colonies (plasmids with insert) over the red colonies (no insert in plasmid).

              </li>
              <li>
                Colony PCR amplification for all the constructs by using the primers:<br>
                BBa_K4294032 (pTU2 Reverse).<br>
                BBa_K4294021 (Biobrick Prefix Forward).<br>

              </li>
              <li>
                Agarose gel electrophoresis.
                <br>
                Expected zones:
               
                  <ul>
                    <li>
                      PCR products from constructs with S4,S5,S10,S11,S13 RBS (pTU2-A-RFP): 
                      1 band at 2067bp.
                    </li>
                    <li>
                      PCR products from constructs with  S12 RBS (pTU2-A-RFP) 
                      1 band at 2066bp.
                    </li>
                    <li>
                      PCR products from constructs with BCD1, BCD8, BCD12 RBS (pTU2-A-RFP)
                      1 band at 2122bp.
                    </li>
                    <li>
                      PCR products from constructs with BCD2, BCD14 RBS (pTU2-A-RFP) 1 band at 2126bp.
                    </li>
                  </ul>
                  <center>
                    <img class="fig-31" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-31.png">
                    <figcaption>
                      Figure 31:<br>
                      L: Ladder 1kb plus<br>
                      1: BCD12 level 2 new promoter 36nt LuxI-sfGFP construct amplified by colony PCR<br>
                      2: BCD2 level 2 new promoter 36nt LuxI-sfGFP construct amplified by colony PCR<br>
                      3: S5 level 2 new promoter 36nt LuxI-sfGFP construct amplified by colony PCR<br>
                      4: S10 level 2 new promoter 36nt LuxI-sfGFP construct amplified by colony PCR<br>
                      5: S12 level 2 new promoter 36nt LuxI-sfGFP construct amplified by colony PCR<br>              
                    </figcaption>
                    </center>
                    <br>
                    As shown in Figure 31, our electrophoresis on PCR products for the level 2 sender’s constructs resulted in the expected bands for most constructs. 
                </li>
                <li>
                  Plasmid minipreps from all the correct aforementioned level 2 sender’s constructs.
                </li>
                <li>
                  Transform <it> E. coli </it> BL21 with the recombinant level 2 plasmids from the minipreps on LB-Agar plates containing chloramphenicol.
                </li>
              <br>

              <h1>Test</h1>
              <br>
              For the testing of the aforementioned Level 2 constructs, we selected white <it>E. coli</it> BL21 colonies (plasmid with insert), and no red colonies (plasmid without insert), from Petri dishes and we made overnight liquid cultures. The next day, we diluted them 1:30, and when OD reached 0.3-0.6 cells were induced with multiple concentrations of aTc. Then,  200 μL of each sample was transferred into a 96-well plate (black with clear bottom) and incubated in a shaking incubator for 3 hours. By using the plate reader FlexStation® 3, absorption and fluorescence of the cells was measured at different time points. The RFU/OD600nm ratio did not increase (Figure 32). sfGFP protein measurements were performed with excitation and emission filters set at 485 nm and 510 nm respectively. 
              <br><br>
              <center>
                <img class="fig-32" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-32.png">
                <figcaption>
                  Figure 32: Induction experiment of DH5a cells with pLacI promoter for the expression of TetR. The adaptation of the promoter and RBS of TetR to a previously well characterised system did not lead to a functional regulatory system.         
                </figcaption>
                </center>
                <br>

              <h1>Learn</h1>
              <br>
              There should be an error in the design we could not figure out. Maybe TetR needed a degradation tag, since a LVA tag is often added to repressors and TetR as well, as in part BBa_C0040. The need of a greater TetR production for the deployed cells is a possible issue, but not really likely given the fact that TetR has a low association constant K, meaning that it can form functional dimers at relatively lower concentrations compared to other repressors [8].
              <br><br>
              Since we had limited time left to characterise our parts, our instructor urged us to use the DH5a-Z1 strain with the high copy Level 1 pTU1 plasmid for the part characterization. This strain possesses genome-encoded TetR expression and its inducibility with aTc had been proven in our instructor’s lab
              <br><br>
              A preliminary experiment was conducted to determine the inducibility of the system. DH5a-z1 cells transformed with a Level 1 plasmid containing the LuxI-only-CDS (the construction of these circuits is analyzed in the “FInal Senders’ Constructs” section) were induced with aTc. Then, they were centrifuged and the supernatant was mixed with receiver cells containing an already characterized and functional system, the open loop (ColE1) circuit. The results indicated that the DH5a-z1 cells are successfully induced.  The sigmoidal response curve of the receivers in relation with the aTc concentration is, in a rough approximation, an indirect indicator of a similar response curve regarding the LuxI production in senders (Figure 33).
              <br><br>
              <center>
                <img class="fig-33" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-33.png">
                <figcaption>
                  Figure 33: Receiver (OL - ColE1) response curves after resuspension in the supernatant of S10 - LuxI only CDS containing  senders, which were induced for 3h (aTc concentration range 2μΜ - 5,12 *10^(-6)μM ). Measurements are presented until the time point of 240min (4h) when receivers have reached a steady state.        
                </figcaption>
                </center>
                <br>
              </p>
            </div>
            </div>
              
          </section>

          <!-- Strtegy -->

          <section class="strategy">
          <a id="strategy" class="anchorOffset"></a>
          <h1>Final Characterisation Strategy</h1>
          <br>
          Level 1 pTU1 plasmids encoding the 36nt LuxI- sfGFP fusion were used to characterize the relative strengths of our RBS library. In order to achieve that, we transformed <it>E. coli</it>  DH5a-Z1 cells and grew them in LB- Agar- Ampicillin plates.
          <br><br>
          <center>
            <img class="fig-34a" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-34a.png">
            <img class="fig-34b" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-34b.png">
            <figcaption>
              Figure 34: 
              (a) Agar plates with Ampicillin in which they grew transformed <it>E. coli<\it> DH5a-z1 colonies with BCD8 level 1 LuxI36nt-sfGFP construct on pTU1-A-lacZ <br>
              (b)Agar plates with Ampicillin in which they grew transformed <it>E. coli<\it> DH5a-z1 colonies with BCD2 level 1 LuxI36nt-sfGFP construct on pTU1-A-lacZ under UV light.
            </figcaption>
            </center>
            <br>

            The induction of DH5a-Z1 with aTc was successful and we were able to characterize our RBS variants (Figure 35). The potential variability regarding the copy number of pTU1 seems not to shadow the differences in RBS strength. A higher copy number leads to greater noise due to a more extensive deviation of the copy number at a given time and was a major consideration when designing the circuits. Since our Level 2 constructs could not function, this characterization was the only possible way. An extra transformation step for both the senders and the receivers, which would introduce an empty vector offering resistance to chloramphenicol and ampicillin respectively, would be necessary whether an experiment involving mixing the two populations was conducted. 
            <br><br>

            For the testing of the aforementioned constructs, white <it>E. coli</it> DH5a-z1 colonies (plasmid with insert), and no blue colonies (plasmid without insert), were selected from Petri dishes and inoculated in liquid cultures. The next day, we diluted them 1:30 in LB medium with Ampicillin, and when OD reached 0.3-0.6, they were induced with multiple concentrations of aTc (ranging from 5.12 * 10^-6μΜ to 2μΜ). Measurements of OD and fluorescence were taken for a total time period of 4h. The RFU/OD600nm ratio was increasing and formed a sigmoidal response curve in its steady-state, proving our characterization approach successful. Measurements for sfGFP fluorescence were performed with excitation and emission filters set at 485 nm and 510 nm respectively.

            <center>
              <img class="fig-35" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-35.png">
              <figcaption>
              Figure 35: Comparison of all 11 RBS  as characterized by our measurements. Please refer to the results tab for more information.
             </figcaption>
              </center>

          </section>

          <!-- Contructs  -->
          <section class="constructs">
          <a id="constructs" class="anchorOffset"></a>
          <h1>Final Senders' Constructs</h1>
          <br>
          After a long way to determine a functional characterization strategy, the selected RBS variants were characterized in the pTU1 plasmid vector in <it>E.coli</it> DH5a-z1 cells. 
          <br><br>
          It was time to test our system for its initial goal; that is input pattern recognition. We had already constructed Level 1 TUs containing only the LuxI CDS (LuxI-only TU) with each different RBS variant so as to be used in this step. Level 2 plasmids with the LuxI only TU and the original TetR repressor system in pTU2 plasmids were constructed as well in a similar way to the Level 2 characterization circuits. The only difference was the Level 1 CDS being the LuxI-only-TU. However, since these constructs were malfunctioning during our characterization experiments, there is not an actual point to describe them in detail.
          <br><br>          
            
            <!-- Constructs Cycle -->
            <div class="collapsible_in">
              <input type="checkbox" id="constructs-cycle-circuit">
              <label for="constructs-cycle-circuit">Engineering Cycle</label>
              <div class="collapsible-in-text">
              <h1>Design</h1>
              <br>
              <p>
              For the senders’ basic circuit we designed 11 different sender populations, each one with a different RBS variant defining the translation rate of LuxI, namely with a different weight in determining the final systems’ output. The goal was to determine the right combination of senders populations that would perform the desired computation task. 
              <br>
              <h1>Build</h1>
              <br>

              <b>Level 1 constructs:</b>
              <br><br>
              <ul>
                <li>
                  We performed Golden Gate Assemblies with BsaI restriction enzyme, for every construct with the LuxI TU we used:
                </li>
                    <ul>
                    <li>  
                    The vector BBa_K4294091 (pTU1-A-lacZ).
                    </li>
                    <li>
                    The parts BBa_K3424010 (TetR promoter), BBa_K4294210 (LuxI codon optimised for <it>E. coli<\it> (+6His Tag)), BBa_B0015 (Double Terminator).
                    </li>
                    <li>
                    An RBS that was different for each construct. The RBSs was BBa_K4294004 (S4), BBa_K4294005 (S5), BBa_K4294010 (S10), BBa_K4294011 (S11), BBa_K4294012 (S12), BBa_K4294013 (S13), BBa_J428034 (BCD1),BBa_K4294102 (BCD2), BBa_K2680561 (BCD8), BBa_K2680529 (BCD12), BBa_K4294114 (BCD14).
                    </li>
                    </ul>
                <li>
                  Transformation of the recombinant plasmid on <it>E. coli</it> DH5a at LB - Agar - Ampicillin plates whith IPTG 0.1M and x-Gal 2% for Blue-White screening.
                  <center>
                    <img class="fig-36" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-368.png">
                    <figcaption>
                      Figure 36: Agar plate with Ampicillin, X-Gal and IPTG for blue-white screening with transformed <it>E. coli<\it> DH5a colonies with BCD8 level 1 LuxI TU construct on pTU1-A-lacZ. 
                    </figcaption>
                    </center>
                  </li>
                <li>
                  Selection of the white colonies (plasmids with insert) over the blue colonies (no insert in plasmid).
                </li>
                <li>
                  Overnight liquid cultures from grown white colonies.
                </li>
                <li>
                  Plasmid Minipreps.
                </li>
                <li>
                  Diagnostic Digest for all constructs.
                </li>
                <li>
                Agarose gel electrophoresis.
                <br>
                Expected zones:
               
                  <ul>
                    <li>
                      Constructs with S4,S5,S10,S11,S13 (pTU1-A-lacz) cut with NotI &PstI: 2 zones at 2045bp + 919bp.

                    </li>
                    <li>
                      Constructs with  S12 RBS (pTU1-A-lacz) cut with NotI &PstI: 2 zones at 2045bp + 918bp.
                    </li>
                    <li>
                      Constructs with BCD1, BCD8, BCD12 (pTU1-A-lacz) cut with NotI &PstI: 2 zones at 2045bp + 974bp
                    </li>
                    <li>
                      Constructs with BCD2, BCD14 (pTU1-A-lacΖ) cut with NotI & PstI: 2 zones at 2045bp + 978bp.
                    </li>
                  </ul>
                </li>
              </ul>  
              <br>

              <center>
                <img class="fig-39" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-39.png">
                <figcaption>
                  Figure 37:<br>

                  L: Ladder 1kb plus<br>
                  1: BCD2 (pTU1-A-lacZ) cut with NotI & PstI<br>
                  2: BCD14 (pTU1-A-lacZ) cut with NotI & PstI<br>
                  3: BCD1 (pTU1-A-lacZ) cut with NotI & PstI<br>
                  4: BCD8 (pTU1-A-lacZ) cut with NotI & PstI<br>
                  5: BCD12 (pTU1-A-lacZ) cut with NotI & PstI<br>
                  6: S4 (pTU1-A-lacZ) cut with NotI & PstI<br>
                  7: S5 (pTU1-A-lacZ) cut with NotI & PstI<br>
                  8: S10 (pTU1-A-lacZ) cut with NotI & PstI<br>
                  9: S11 (pTU1-A-lacZ) cut with NotI & PstI<br>
                  10: S12 (pTU1-A-lacZ) cut with NotI & PstI<br>
                  11: S13 (pTU1-A-lacZ) cut with NotI & PstI<br>
                  12: Control - pTU1-A-lacZ uncut<br>
                  13: Control - pTU1-A-lacZ cut<br>
                  14: Control - BCD8 (pTU1-A-lacZ) uncut<br>
                  15: Control - S13 (pTU1-A-lacZ) uncut<br>
                                      
                </figcaption>
                </center>
                <br>  

                As shown in Figure 37, our diagnostic digest on the level 1 sender’s constructs resulted in the expected bands (in the 1st, the 2nd and the 7th samples the zone at 978bp and 919bp respectively has very low brightness. However, we observe that there is a high concentration of uncut recombinant plasmid in the samples.              
                <br><br>



              <h1>Test</h1>
              <br>
              Figure 33 (“Final Characterization Circuit - Learn) represents the first trial of a Sender - Receiver interaction. More information regarding such experiments can be found in the <a href="https://2022.igem.wiki/athens/results">Results</a> and <a href="https://2022.igem.wiki/athens/proof-of-concept">Proof of Concept</a> pages.
              </p>

            </div>
            </div>
  
          </section>

          <!-- Crosstalk -->

          <section class="crosstalk">
          <a id="crosstalk" class="anchorOffset"></a>
          <h1>Reduction of crosstalk</h1>
            
            <!-- crosstalk Cycle -->
            <div class="collapsible_in">
            <input type="checkbox" id="crosstalk-cycle-circuit">
            <label for="crosstalk-cycle-circuit">Engineering Cycle</label>
            <div class="collapsible-in-text">
            <h1>Design</h1>
            <br>
            <p>
            After deciding that the input patterns would be generated by one chemical inducer, a problem that emerged was keeping the supposedly “0” senders in the pattern uninduced. Even with washes before the senders are mixed, residual aTc might result in unwanted activation of a sender subpopulation and mess up the initial pattern. We decided to incorporate TetX, a tetracycline inactivation enzyme that breaks down aTc, in our cross-talk tackling strategy. As mentioned in the Design section, this construct was designed with pTKEI-Dest as its backbone, because its ori (incompatibility group A) would not interfere with the replication of the senders’ final construct (incompatibility group B). pTKEI would be co-transformed in the senders we wished to keep uninduced in the specific pattern.

            <h1>Build</h1>
            <br>
            <b>Level 1 constructs:</b>
              <br><br>
              <ul>
                <li>
                  We performed Golden Gate Assembly with BsmBI using the:
                </li>
                    <ul>
                    <li>  
                      BBa_K4294094 (pTKEI - Dest vector).
                    </li>
                    <li>
                      BBa_K4294790 (TetX TU).
                    </li>
                    </ul>
                <li>
                  Transformation of the recombinant plasmid on <it>E. coli</it> DH5a at LB - Kanamycin plates whit IPTG and xGal for Blue-White screening.
                </li>
                <li>
                  Selection of the white colonies (plasmids with insert) over the blue colonies (no insert in plasmid).
                  <center>
                    <img class="fig-38" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-38.png">
                    <figcaption>
                      Figure 38: Agar plates with Kanamycin in which they grew transformed E. coli DH5a colonies with pTKEI-Dest TetX construct.
                    </figcaption>
                    </center>
                </li>
                <li>
                  Overnight liquid cultures from grown white colonies.
                </li>
                <li>
                  Plasmid Minipreps.
                </li>
                <li>
                  Diagnostic Digest of the recombinant plasmid.         
                </li>
                <li>
                Agarose gel electrophoresis.
                <br>
                Expected zones:
               
                  <ul>
                    <li>
                      TetX (pTKEI - Dest) cut with PstI: 2 zones (3718bp + 1526bp).
                    </li>
                  </ul>
                </li>
              </ul>  
              <br>

              <center>
                <img class="fig-39" src="https://static.igem.wiki/teams/4294/wiki/engineering-success/es-figure-39.png">
                <figcaption>
                  Figure 39:<br>
                  L: Ladder 1kb plus<br>
                  1: Control - pTKEI uncut<br>
                  2: Control - pTKEI cut with PstI<br>
                  3: pTKEI - TetX cut with PstI<br>                     
                </figcaption>
                </center>
                <br>  

              Figure 39 shows that our diagnostic digest on the pTKEI - TetX construct resulted in the expected bands.
              <br><br>
            

            <h1>Test</h1>
            <br>

            Unfortunately, due to a lack of time, we did not manage to test it. However, to test the construct we had to:
            <br><br>
            1. Transform the already transformed senders bacteria (with the pTU2-A-RFP recombinant plasmid containing the fluorescent marker for characterization), by making the competent again to accept the new plasmid.<br>

            2. Cultivate them in LB - Agar plates containing Kanamycin and Chloramphenicol.<br>

            3. Make induction of our senders with the inducer aTc and at some point adjust IPTG in order to activate the TetX transcription.<br>

            4. After that, a decrease in RFU/ absorption ratio is to be expected.<br>
            <br>
            <br>
            Due to lack of time, we only reached step 2 of this workflow and did not have time to test the induction of these cells, but the transformation was successful. <br>
            



            </p>

            </div>
            </div>
          
          </section>



      <!-- References -->

      <section class="collapsible" id="references">
        <input type="checkbox" id="collapsible-head">
            <label for ="collapsible-head">References</label>    
            <div class="collapsible-content">
              <cite>
                <a href="https://pubmed.ncbi.nlm.nih.gov/31488843/">
                    1.Huber M, Faure G, Laass S, Kolbe E, Seitz K, Wehrheim C, Wolf YI, Koonin EV, Soppa J. Translational coupling via termination-reinitiation in archaea and bacteria. Nat Commun. 2019 Sep 5;10(1):4006. doi: 10.1038/s41467-019-11999-9. PMID: 31488843; PMCID: PMC6728339.
      
                </a>
            </cite>
      <cite>
                <a href="https://pubmed.ncbi.nlm.nih.gov/19279094/">
                    2. Hatta M, Kohlmeier CK, Hatta Y, Ozawa M, Kawaoka Y. Region required for protein expression from the stop-start pentanucleotide in the M gene of influenza B virus. J Virol. 2009 Jun;83(11):5939-42. doi: 10.1128/JVI.00180-09. Epub 2009 Mar 11. PMID: 19279094; PMCID: PMC2681956.
      
                </a>
            </cite>
      <cite>
                <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.0020096">
                    3. Price MN, Arkin AP, Alm EJ. The life-cycle of operons. PLoS Genet. 2006 Jun;2(6):e96. doi: 10.1371/journal.pgen.0020096. Epub 2006 Jun 23. Erratum in: PLoS Genet. 2006 Jul;2(7):e126. PMID: 16789824; PMCID: PMC1480536.
                </a>
            </cite>
      
      
            <cite>
                <a href="https://pubmed.ncbi.nlm.nih.gov/30843313/">
                    4. Tutol, J., Kam, H., Dodani, S., Identification of mNeonGreen as a pH-Dependent, Turn-On Fluorescent Protein Sensor for Chloride. 2019 Jul 15;20(14):1759-1765. doi: 10.1002/cbic.201900147. Epub 2019 May 8. PMID: 30843313 PMCID: PMC6663633 DOI: 10.1002/cbic.201900147
      
      
      
                </a>
            </cite>
      
            <cite>
                <a href="https://www.nature.com/articles/s41576-021-00417-w">
                    5.  Wright BW, Molloy MP, Jaschke PR. Overlapping genes in natural and engineered genomes. Nat Rev Genet. 2022 Mar;23(3):154-168. doi: 10.1038/s41576-021-00417-w. Epub 2021 Oct 5. PMID: 34611352; PMCID: PMC8490965.
                </a>
            </cite>
      
      
      <cite>
                <a href="https://pubmed.ncbi.nlm.nih.gov/23474465/">
                    6.   Mutalik VK, Guimaraes JC, Cambray G, Lam C, Christoffersen MJ, Mai QA, Tran AB, Paull M, Keasling JD, Arkin AP, Endy D. Precise and reliable gene expression via standard transcription and translation initiation elements. Nat Methods. 2013 Apr;10(4):354-60. doi: 10.1038/nmeth.2404. Epub 2013 Mar 10. PMID: 23474465.
      
                </a>
            </cite>
      
      <cite>
                <a href="https://2018.igem.org/Team:Tsinghua/Results)">
                    7.  Team Tsinghua 2018
      
                </a>
            </cite>
      
      <cite>
                <a href="https://pubmed.ncbi.nlm.nih.gov/30478458/">
                    8.   Meyer AJ, Segall-Shapiro TH, Glassey E, Zhang J, Voigt CA. Escherichia coli "Marionette" strains with 12 highly optimized small-molecule sensors. Nat Chem Biol. 2019 Feb;15(2):196-204. doi: 10.1038/s41589-018-0168-3. Epub 2018 Nov 26. PMID: 30478458.
      
                </a>
            </cite>
      <cite>
      <a href="https://www.semanticscholar.org/paper/Primerize%E2%80%902D%3A-automated-primer-design-for-RNA-Tian-Das/acbbd0e1161646b1216143b6f304936cbe6fa78a">
                    9. Siqi Tian, Rhiju Das, Primerize-2D: automated primer design for RNA multidimensional chemical mapping, Bioinformatics, Volume 33, Issue 9, 1 May 2017, Pages 1405–1406
                </a>
            </cite>
      
      
      
      
      <cite>
                <a href="https://academic.oup.com/nar/article/43/W1/W522/2467987">
                    10. Siqi Tian, Joseph D. Yesselman, Pablo Cordero, Rhiju Das, Primerize: automated primer assembly for transcribing non-coding RNA domains, Nucleic Acids Research, Volume 43, Issue W1, 1 July 2015, Pages W522–W526
      
      
                </a>
            </cite>
      
      
            </div>
        </section>
        </div>
      </div>





<!-- Footer -->
{% include 'footer.html' %}

<!--Loader-->   
{% include 'loader.html'%}  

</html>
