<!doctype html>
<html>
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="shortcut icon" href="https://static.igem.wiki/teams/4294/wiki/favicon-white.png"/>
    <link rel="license" href="https://creativecommons.org/licenses/by/4.0/"/>
    
    <!-- Custom CSS -->
    <link href="{{ url_for('static', filename = 'css/implementation_style.css') }}" rel="stylesheet">

    <!-- Loader -->
    <script type= "text/javascript" src="../../athens/static/scripts/jquery.min.js"></script>

    <!--Custom menu CSS-->
    <link href="{{url_for('static',filename='css/menu_style_aspa.css')}}" rel="stylesheet">

    <!--Custom footer CSS-->
    <link href="{{url_for('static',filename='css/footer_style_aspa.css')}}" rel="stylesheet">

     <!--Custom to top button CSS-->
     <link href="{{url_for('static',filename='css/to_top_button.css')}}" rel="stylesheet">
    <title>Perspectives | Implementation</title>
</head>

<body>
<!-- Navigation -->
{% include 'menu_custom.html' %}

<!--To top button-->
{% include 'to_top_button.html'%}

<div class="page_header">
    <img class="img_header" src="https://static.igem.wiki/teams/4294/wiki/contribution/project-bg.gif" width="100%">
    <div class="heading">Implementation</div>    
</div>


<div class="cc" id="cc-implementaion">
  <div class="left-side">
      <div class="index-bar">
          <ul class="lateral-index">
              <li class="lateral-content">
                  <a href="#intro">Introduction</a>
              </li>
              <li class="lateral-content">
                  <a href="#health">Health</a>
              </li>
              <li class="lateral-content">
                <a href="#environment">Environment</a>
              </li>
              <li class="lateral-content">
                <a href="#biocomputing">Biocomputing</a>
              </li>
              <li class="lateral-content">
                <a href="#conclusion">Conlcusions</a>
              </li>
              <li class="lateral-content">
                  <a href="#references">References</a>
              </li>
          </ul>
      </div>
  </div>
  
  <div class="right-side">
      <section class="introduction">
          <a id="intro" class="anchorOffset"></a>
          <br><br>
          <h1>Introduction</h1>
          <br>
          <p> “PERspectives” can have a wide range of implementations, some of which are examined and proposed in this section. From medicine, to the environment and even energy consumption, it can be the foundation of groundbreaking advances in each of the aforementioned fields and even more. As a universal tool used by  future iGEM teams and generally researchers, PERspectives can provide enhanced genetic stability, reliability and long-term functionality in a variety of projects. Our system utilizes a cell consortia format in order to create a novel biosensor that allows multiple inputs recognition and activation according to their significance. The work compartmentalization to multiple bacterial populations reduces the metabolic burden that each cell would carry. Additionally, by using the quorum sensing mechanism for communication among bacterial populations, misinterpretation of the output resulting from cellular noise is reduced. The reduction of biological noise is the major advantage of “PERspectives”, in combination with its various potential implementations. Our system can be versatile to different problems, since it can be used for different genes, as presented in Figure 1. Some of the applications that will be presented in the section below can be directly plugged into our system and some of them require further editing as described in each case.</p>
          <br>
          <img class="imple-img-1" src="https://static.igem.wiki/teams/4294/wiki/implementation/imple1.png">
          <img class="imple-img-2" src="https://static.igem.wiki/teams/4294/wiki/implementation/imple-2.png">
          <h5 style="text-align: center;">Figure 1: Representation of the versatile nature of “PERspectives” to solve multiple problems.<br> (Created in Biorender.com)</h5>
      </section>

      <section class="health">
        <a id="health" class="anchorOffset"></a>
        <br><br>
        <h1>Health</h1>
        <br>
        <p> Our project’s  goal is to create a <span style="color: #3aa9c5;">universal tool</span> that will be able to solve real-time multifactorial problems. Our bacterial perceptron will have special value in cases where the problem is too hard to reach, taking for example the human body. 
            <br><br>
            What about a <span style="color: #3aa9c5;">living therapeutic</span> that could detect the problem in real-time and decide if a medicine is needed or not?
        </p>
        <br>

        <div class="collapsible_in">
            <input type="checkbox" id="theranostic">
            <label for="theranostic">Cancer - Targeted drug delivery - Theranostic systems +</label>
            <div class="collapsible-in-text">
            <p>Non-pathogenic bacteria have been proposed and even tested in clinical trials as a potential drug delivery system. The iGEM Competition has even dedicated a project track named “Living Therapeutics”, including projects aiming to utilise a living organism as a drug delivery system. Living therapeutics are without a doubt a popular research field and the engineered  genetic circuits are constantly evolving.
                <br><br>
            Nowadays, the field of precision medicine and targeted drug delivery is flourishing, promising more effective therapies and fewer side effects for patients. 
            <br><br>
            How could our project ideation and design be incorporated to take this field to the next level?
            <br><br>
            As far as cancer is concerned, it is proven that tumors are not homogenous. This diversity often makes one-drug therapeutic schedules ineffective. Every tumor type includes various subtypes and even the environment inside a single tumor shows variations. 
            <br><br>
            One important classification of tumors refers to the activity of the immune system inside them. Roughly explained, tumors that host an activated immune system and inflammation are characterized as immune “hot” tumors, whereas tumors with suppressed immune cells that are not inflamed are characterized as immune “cold” tumors. The immune status of a specific tumor plays a vital role regarding its response to immunotherapy, and more specifically to Immune Checkpoint Inhibitors (ICIs). These are monoclonal antibodies that bind and block immune checkpoints, like PD-L1 and CTLA-4, which are naturally expressed by cells to hinder immune systems’ overactivity. In some cancer types, these inhibitors are overexpressed and thus cancer cells escape immune surveillance. Immune checkpoint inhibitors block immune checkpoint’s activity and permit immune cells to attack the malignant cells. However, tumors are often immune excluded or exhausted, meaning that they do not host inflammatory immune cell phenotypes. Even though these tumors might express immune checkpoint inhibitors, they do not respond sufficiently to ICIs. One promising strategy is to fire up cold tumor microenvironments by combining these immunotherapeutic drugs with proinflammatory agents, that will promote infiltration of the tumor by immune cells and then, their antitumoral activity will be enhanced by the action of ICIs [12,13].
            <br><br>
            Tumour-invading bacteria are pathogenically incompetent bacterial strains, which are able to reside and multiply in the necrotic tumour environment and have been described as a potential drug delivery system to tumours [14,15]. They could potentially take the mentioned “cocktail” strategies [16] to the next level. Probiotics have been engineered to express immune checkpoint inhibiting nanobodies and release them inside tumors after synchronized lysis when they have reached a certain density, which is sensed via quorum sensing signals. In the same research, bacteria that express a proinflammatory agent were intratumorally injected together with the blockade nanobodies producing bacteria to enhance their antitumor action, similarly with the drug cocktail strategies but promising less side effects since the drugs are released inside the tumor and not systemically delivered[17]. A perceptron-inspired approach could revolutionize this strategy. The suitable therapeutic genes will be engineered into receiver cells for both hot and cold tumors. Sender cells will be engineered to sense signs of active inflammation inside the tumor and respond by producing a quorum sensing molecule. Receiver cells according to the local summation of the lactone will switch their response between the production of a proinflammatory agent or an immune blockade inhibiting nanobody. Such a system, rather complex, might provide an immunotherapeutic strategy able to adapt to each tumor's immune status and even to the immune status of each tumor’s site (necrotic immune inactive center versus a more immune active periphery). Analyses of the different metabolism between macrophage phenotypes ( inflammatory, classically activated M1 phenotype and immunosuppressive, tumor promoting alternatively activated M2 phenotype[18,19]), very important players in the tumor microenvironment, and mathematical modeling of tumor-immune cells interactions and of the different intratumoral macrophage subtypes regarding the production of small molecules already exist[20,21].  Bacteria capable of recognising small products of inflammatory cells, such as nitric oxide [22] and even the proteins Interferon-γ and Tumor Necrosis Factor-a [23] have been engineered and the collection of new bacterial sensing modules grows.
            <br><br>
            All the above are paving the way towards datasets of biochemical analyses and markers that could be used to simulate and develop a probiotic perceptron that would be able to target solid tumors, classify them and produce the more suitable therapeutic substance. 
            <br><br>
            Beyond this example, such neural-like inspired biological systems would revolutionize therapies on the grounds that many diseases are multifactorial and many times difficult to be distinguished from one another. Incorporating weighted processing of inputs would lead to efficient theranostic systems promising more effective therapies and less side effects.
            <br><br>
            <center>
            <img class="imple-img-4" src="https://static.igem.wiki/teams/4294/wiki/implementation/imple-4.png">
            <h5 style="text-align: center;">Figure 3: Tumor-invading bacterial perceptron system. Sender populations (blue bacteria) sense inputs from 
                their surroundings and respond by producing a different or either by producing or not producing quorum sensing molecules (concentric 
                circles indicate the different signaling state in different tumor microenvironments). Receiver populations respond to these signals by 
                either producing an ICI (black bacteria) or a proinflammatory agent (red bacteria).<br> (Created in Biorender.com)
            </h5>
            </center>
            </p>
            </div>
        </div>


        <div class="collapsible_in">
            <input type="checkbox" id="Gut">
            <label for="Gut">Gut Disbalances + </label>
            <div class="collapsible-in-text">
            <p>The human body is cohabited by microorganisms, this symbiosis is frequently referred to as the microbiome. The most studied microbiome is the one of the human gut, known as the gut microbiome[1].  Up to 1000 different bacterial species are found in the human gut. Each one of these species is essential for the function of the human gut, as it contributes to processes such as food digestion and immunity. In case that the balance between the different species gets distracted (called dysbiosis), multiple diseases could  emerge. Gut disbalances are linked to diseases such as inflammatory bowel diseases (IBD), irritable bowel syndrome (IBS), diabetes, obesity, cancer, and cardiovascular and central nervous system disorders. [2]. According to the World Health Organization (WHO), probiotics are “live strains of strictly selected microorganisms which, when administered in adequate amounts, confer a health benefit on the host” [3,4]. In order to help with the treatment of gut disbalances we plan to design a bacterial perceptron that will have miRNAs or gDNAs detected in the human gut as inputs, the levels of which have been linked to gut disbalances in individuals with the disease. Regarding the gDNAs, <it>B.subtitlis</it> has the natural competence ability to detect unique DNA sequences present in the environment. So we plan to use this ability in order to program <it>B.subtilis</it> to detect different species in the human gut [5]. In fact, <it>B. Subtilis</it> can uptake environmental DNA and integrate it using specific sequences in each genome homologous recombination. (Dubnau, D. Genetic competence in Bacillus subtilis. Microbiological Reviews (1991)). The sequence’s length and similarity affect the recombination efficiency. By engineering these two factors in the genome of, <it>B.subtilis</it> for different  strains found in the human gut, we will be able to incorporate the weights and control the expression of the quorum sensing molecule, in a similar way that the different Ribosome Binding Sites control the expression of the quorum sensing molecule in our system. If the quorum sensing molecule concentration surpasses a certain threshold, the production of a certain substance that helps with gut disbalances will be activated. This substance could be reuterin which has been proven to restore the microbiome balance. [6,7,8]. Regarding the miRNAs approach, there are multiple databases that correlate levels of miRNAs to a specific gut disbalance disorder [9]. After bioinformatic processing, we will be able to classify these miRNAs by importance and frequency in the population. Then, we will set the weights of our living therapeutic by adjusting the complementarity of our miRNA to a specific locus.  In this case, our weights will be engineered based on miRNA interference [10,11].


                <center>
                <img class="imple-img-gut" src="https://static.igem.wiki/teams/4294/wiki/implementation/gut-img.png">
                <h5 style="text-align: center;">Figure 2: <it>B. Subtilis</it> detects different microorganisms in the gut microbiome environment and because of its natural competence, internalizes their g.DNA. This leads to Homologous Recombination and production of a OC6 or other quorum sensing molecule that will activate the receivers to produce reuterin.<br> (Created in Biorender.com)
                </h5>
                </center>
             </p>
            </div>
        </div>
        


        

         <div class="collapsible_in">
            <input type="checkbox" id="bacviral">
            <label for="bacviral">Bacterial or Viral Infection - Classification +</label>
            <div class="collapsible-in-text">
            <p>Identifying the cause in acute infections constitutes a common medical problem. The symptoms' similarity between viral and bacterial infections makes it difficult for health specialists to identify the optimal treatment in urgent situations. This situation often leads to the overuse of antibiotics [24] and contributes to antibiotic resistance. Research has shown that discriminating between viral and bacterial infections requires simultaneous testing for multiple factors, some of which belong to the individual's immune response and not to the pathogen itself [25]. What if we had a bacterial artificial intelligence classification system or a cell-free system following a similar principle that would simultaneously test for three different factors and classifiy for the best treatment for the patient? Our project “Perspectives” seems like a perfect candidate for this task because it can evaluate different inputs, assigning each of them the appropriate significance depending on if it recognizes the bacterial or the viral infections [26]. Three molecules that could be recognized according to literature are: NF-related apoptosis-inducing ligand (TRAIL) [27] which is upregulated in viral infection patients, IP-10 which is higher in viral patients compared to bacterial patients, and CRP (C-Reactive Protein) which is mainly encountered in bacterial infections but has been correlated to some viral strains' infection as well. In this way, our circuit could be used as a diagnostic tool, taking into account that huge research has been done on how human samples affect bacterial biosensors [28].
            </p>
            </div>
        </div>

         <div class="collapsible_in">
            <input type="checkbox" id="diabetic">
            <label for="diabetic">Prevention of Diabetic Ketoacidosis +</label>
            <div class="collapsible-in-text">
            <p>Poor control of type one diabetes may lead to diabetic ketoacidosis, which could be lethal. Diabetic ketoacidosis is characterized by pH lower than 7.35, high blood sugar levels and low insulin levels. Following the lead of David Ausländer [29] who designed a pH-biosensor using microencapsulated HEK293 cell line batches, a biological perceptron embedded in a living therapeutic could incorporate multiple input signals, like the aforementioned pH level, glucose level and insulin level to classify in real time the patient's status and produce a suitable amount of insulin.
            </p>
            </div>
        </div>

         <div class="collapsible_in">
            <input type="checkbox" id="pharma">
            <label for="pharma">Pharmaceutical Industry - RnD +</label>
            <div class="collapsible-in-text">
            <p>
            As it’s known, developing new, efficient, and safe drugs is extremely time-consuming, expensive, and can take even 12 years on average.  [30] The first stage of that discovery begins in the laboratory. According to the FDA, thousands of compounds may be potential candidates for development as a medical treatment at this stage. After early testing, however, only a small number of compounds seems promising and calls for further study [31].  Also, many diseases have a very complicated phenotype, often related to a complicated genotype. Therefore, the existence of a biosensing artificial neural network system could be extremely beneficial to the Research and Development (RnD) phase. [32] In the last few years, a lot of research has been done to reveal the connection between cancer and miRNA profiles[33,34]. According to them, many miRNAs are associated with cancer types, but some miRNAs have a stronger correlation to a specific type of cancer. Also, the ANNs appear as a promising tool for pharmaceutical scientists [35]. What about a biological system that could classify the new tested molecules, according to the provoked change in the phenotype of cell lines, between the possible new drugs and the molecules that are not effective or safe? 
            <br><br>
            Our system’s design could play a leading role in the RnD domain in this framework. Let’s take as an example, colorectal cancer. Firstly, we 
            could gather a significant amount of patient samples as well as a significant amount of cancer-free individual samples and analyze them in
             quantity and quality for miRNAs. After that, with the use of an Artificial Neural Network, we can select the most important miRNA molecules 
             and calculate the weights of all of them[36]. Then, we could translate those weights into RBS strengths by using the same weight promoters. 
             By using that knowledge, we could design a novel diagnostic tool for colorectal cancer. The most fascinating part though could be if we also 
             collect a significant number of samples of patients during the course of therapy and analyze them also with the usage of an Artificial Neural 
             Network [37]. 
            <br><br>
            After that, the potential candidate molecules will be added to colorectal cancer cell lines. By giving to our system the lysate of these cells or a portion of their medium we could have a classification between promising molecules and molecules with no efficiency or safety. In the framework of the above experiments, the scientist can use either mammalian cells [38] or bacterial cells that detect miRNAs. Toehold switches could be engineered to bind to input miRNA and respond in a weighted manner by an output production. The same principle of work could be used also for the discovery of new antidepressants [39], new anticancer drugs for Hodgkin lymphoma [40]and new anticancer drugs for breast cancer [41,42]. 
            </p>
            </div>
        </div>


         <div class="collapsible_in">
            <input type="checkbox" id="ptherapy">
            <label for="ptherapy">Personalized Pharmacotherapy +</label>
            <div class="collapsible-in-text">
            <p>
                According to CDC: “Pharmacogenomics is an important example of the field of precision medicine, which aims to tailor medical treatment to each person or to a group of people” [43]. By pharmacogenomics, scientists reduce the likelihood of adverse drug reactions and optimise therapeutic efficacy [44]. As an example, in cancer research, several in vitro investigations describe altered patterns of miRNA expression in cells resistant to anticancer drugs. For instance, in a study by Berkers et al., they analysed miRNA expression in clear cell renal carcinoma tissue samples from patients who received sunitinib. The results showed a significant decrease in the expression of miR-141 in patients with progressive disease (that indicates maybe a high weight of that miRNA for a perceptron algorithm). Also, a subset of other miRNAs, differentially regulated between the patients who responded to sunitinib treatment versus the non-responders, included miR-520g, miR-155, miR-526b (all upregulated) and miR-144 and miR-376b (both downregulated) (that indicates maybe a medium or low weight of those miRNAs for a perceptron algorithm).
                <br><br>
                Some of the mechanisms that have been proposed for that resistance are presented below. MicroRNAs contribute to cancer drug resistance via :
                <br></p>
            <ul>    
                <li>
                    enhanced drug efflux
                </li>
                <li>
                    altered drug metabolism
                </li>
                <li>
                    overexpression of target molecules
                </li>
                <li>
                    enhanced survival anti-apoptosis pathways
                </li>
            </ul> 
            <br>
            <p>
                In conclusion, by analysing samples of patients with miRNAs connected with the aforementioned resistance pathways for specific drugs and using a system similar to “PERspectives” we could predict if a patient will be cured by a specific medication or not[45]. In the framework of the above experiments, the scientist can use either mammalian cells [46] or bacteria cells that detect miRNAs. In either case, toehold switches appear a very promising structure for the sensing part of miRNAs. In this way, scientists could save a lot of people or improve their quality of life.
            </p>
         </div> 
        </div>

      </section>

      <section class="environment">
          <a id="environment" class="anchorOffset"></a>
          <br><br>
          <h1>Environment</h1>

          <div class="collapsible_in">
            <input type="checkbox" id="water">
            <label for="water">Water Quality +</label>
            <div class="collapsible-in-text">
                <p>
                Inspired by the project S-POP [47], we propose a future implementation in order to make the future goals of S-POP become possible, by implementing our bacterial perceptron to detect water Persistent Organic Pollutants (POPs). The senders (<it>E.coli</it>) will be able to recognize different pollutants in the environment, using promoters of similar strength and adjusting the weights by different context independent RBS sequences, as proved by Mutalik et al. [48,49] for the BCDs. When the specific pollutant is recognized, senders will produce OC6, by expressing the luxI gene, and will activate the receivers (<it>S. oneidensis</it>) to produce electricity. In this way, our system will  be tuned to display oscillations with distinct periods depending on the pollutant detected and will allow us to make a useful diagnostic tool for water pollution. The electricity will be detected by a special device developed by Stockholm 2020 team.
            </div>
          </div>

            <div class="collapsible_in">
                <input type="checkbox" id="soil">
                <label for="soil">Health of soil for classification of samples + </label>
                <div class="collapsible-in-text">
                    <p>
                    Another implementation of our project could be the evaluation of soil health. Soil health is very important in means of plant health and crop yield, but also affects human and animal health [50]. There are different kinds of indicators to assess soil health: Chemical Indicators, Physical Indicators and Biological Indicators. Our system could be redesigned to detect such indicators:  Nitrogen found in the soil via protein degradation is an indicator of its availability for plant assimilation, there are bacterial nitrogen transporters that could be used to set Nitrogen as an input to our system [51],[52],[53]. The pH [54] of the soil is also very important in determining soil health. There are already several pH responsive promoters that could be incorporated in our system to evaluate the pH. [55]. Last but not the least, salinity is important for some environments and could be detected by salt-activated promoters engineered in our system [56]. Our system could be adjusted for the weights of each input to respond in several soil compositions. This will be a huge help for people that work in the agricultural sector to optimize their crop production. 
                    </p>
                </div>
            </div>


      </section>

      <section class="biocomputing">
          <a id="biocomputing" class="anchorOffset"></a>
          <br>
          <h1>Biocomputing</h1>
          <div class="collapsible_in">
            <input type="checkbox" id="biocomputing-d">
            <label for="biocomputing-d">A novel approach to efficient embedded Machine Learning</label>
            <div class="collapsible-in-text">
                <p>
                “PERspectives” comes as an alternative, environmentally friendly method for implementing machine learning algorithms. As it is well-known, s
                tate-of-the art computational systems require a large amount of data in order to be modelled and fine-tuned. Currently, the data are stored in 
                data centres. The operation of such centers is estimated to handle enormous amounts of data every day, as in the case of IBM’S The Weather 
                Company [57], which processes about 400 Terabytes of data per day in order to predict the weather in advance around the globe. Such data 
                mining, storage and processing flow corresponds to energy consumption of around 80MegaWatts-more than enough to power 80.000 US Households. 
                <br><br>
                Another example of effective but energy-costly use of machine learning and-in particular-neural networks comes in the training process of 
                Natural Language Processing Software [58], where the usage of big data and specialized hardware leads to a significant increase in Carbon 
                Footprint Emissions. As an example, training the big “Transformer”[59] of Google-a novel neural network architecture for machine translation 
                tasks-required a total of 100.000 steps or 3.5 days and 8 NVIDIA P100 GPUs, leading to an estimate of 626.000 lbs of CO2 emission. As it can 
                be shown in Figure 1, such an emission total is 5 times more than the average emissions of a conventional car!
                <br><br>
                <img class="imple-img-5" src="https://static.igem.wiki/teams/4294/wiki/implementation/imple-5.png">
                <h5 style="text-align: center;">Figure 4: Comparison of Carbon Emissions produced for training neural networks and their 
                    every-day counterparts (in lbs of CO2). Source:
                     (<a href="https://www.researchgate.net/publication/335778882_Energy_and_Policy_Considerations_for_Deep_Learning_in_NLP">
                        https://www.researchgate.net/publication/335778882_Energy_and_Policy_Considerations_for_Deep_Learning_in_NLP
                     </a>)
                </h5>
                
                <br><br>
                On the other hand, the demand for integrated systems that use machine learning algorithms to recognize patterns and perform complicated tasks 
                is ever-increasing; a variety of such projects has been implemented, from gender recognition hardware [60]. to real-time evaluation of 
                prosthetic hand characteristics[61] and sensor fault detection[62]. Such a concept of using machine learning could revolutionize every-day 
                life, offering high accuracy computing systems which would assist people in a variety of tasks, especially when immediate high-accuracy 
                predictions are required.
                <br><br>
                In that context, an in-vivo implementation of the perceptron comes as a low-power and cost-effective alternative to the current wave of neural 
                networks, since it could be the first step in the right direction in order to limit the ever-increasing carbon footprint that AI algorithms 
                leave on the environment.  Moreover, it is on-par with the trend of using embedded AI systems, and combining its versatile nature, it has the 
                potential to become the basis of a multipurpose classification system used in the environment, health and pharmaceutics domains, as analyzed 
                in the previous sections. Energy demands and carbon footprint could be reduced if such system is incorporated into photosynthetic cells, which harness light's energy and some of them can be easily cultured.
                </p>
                </div>
            </div>
      </section> 

      <section class="introduction">
        <a id="conclusion" class="anchorOffset"></a>
        <h1> Conclusions </h1>
        <br>
        <p> Concluding, there is a vast field for implementation for our project. In our opinion the first steps that should be taken in order for 
            some of these applications to come true are:
            <br>
            <br>

            1. Introducing more different inputs and more layers to the system. <br>
            2. Introducing other intermediate molecules to reduce cross-talk in case of living therapeutics. <br>
            3. Introducing the system to yeast cells using the pheromone system of communication. [63] <br>
            4. Introducing the system to human cell lines using the tryptophan synthase for communication. [63] <br>
            5. Incorporating the kill switch and the negative weights as mentioned in the safety section of the implementations. <br>
            </p>
            <br><br>
        </section>

      <section class="collapsible">
          <a id="references" class="anchorOffset"></a>
          <input type="checkbox" id="collapsible-head">
              <label for ="collapsible-head">References</label>    
              <div class="collapsible-content">
                <cite>
                    <a href="https://www.nature.com/articles/d41586-020-00194-2">
                        1. Herb Brody, The gut microbiome, OUTLOOK,29 January 2020. 
                       https://www.nature.com/articles/d41586-020-00194-2
                    </a>
                </cite>
                <cite>
                    <a href="https://pubmed.ncbi.nlm.nih.gov/30535609/">
                        2. Belizarion,J., Faintuch,J. Microbiome and Gut Dysbiosis. Volume 213, 104608 (2018). 
                      https://pubmed.ncbi.nlm.nih.gov/30535609/
                    </a>
                </cite>
                <cite>
                    <a href="https://4cau4jsaler1zglkq3wnmje1-wpengine.netdna-ssl.com/wp-content/uploads/2019/04/probiotic_guidelines.pdf">
                        3. Joint FAO/WHO Working Group Report on Drafting Guidelines for the Evaluation of Probiotics in Food London, Ontario, Canada, April 30 and May 1, 2002 .                 https://4cau4jsaler1zglkq3wnmje1-wpengine.netdna-ssl.com/wp-content/uploads/2019/04/probiotic_guidelines.pdf
                    </a>
                </cite>
                <cite>
                    <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5622781/#B5-nutrients-09-01021">
                        4. Markowiak , P., Śliżewska, K. Effects of Probiotics, Prebiotics, and Synbiotics on Human Health ,Sep, 2017, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5622781/#B5-nutrients-09-01021
                    </a>
                </cite>
                <cite>
                <a href="https://www.biorxiv.org/content/10.1101/2022.03.10.483875v1.full">
                    5. Cheng , Y., Chen, Z,Cao, X. et.al. Programming bacteria to sense environmental DNA for multiplexed pathogen detection, https://www.biorxiv.org/content/10.1101/2022.03.10.483875v1.full
                </a>
                </cite>
                <cite>
                <a href="https://www.tandfonline.com/doi/abs/10.3109/08910608909140211">
                6. Chung ,T.C.,Axellson, L.,Lindgran, E., Dobrogosz, W.J.In Vitro Studies on Reuterin Synthesis by Lactobacillus reuteri,Pages 137-144,  Jul ,2009,
                https://www.tandfonline.com/doi/abs/10.3109/08910608909140211
                </a>
                </cite>
                <cite>
                <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5770549/#:~:text=Bacterial%20formation%20of%20acrolein%20in%20the%20human%20intestine.&text=Glycerol%20is%20a%20common%20additive,the%20small%20intestine%20">
                7. Zhang,J.,Sturla, S.,Lacroix, C.,Schwab, C.Gut Microbial Glycerol Metabolism as an Endogenous Acrolein Source
                ,  Jan ,2018,
                https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5770549/#:~:text=Bacterial%20formation%20of%20acrolein%20in%20the%20human%20intestine.&text=Glycerol%20is%20a%20common%20additive,the%20small%20intestine%20
                </a>
                </cite>
                <cite>
                <a href="https://www.sciencedirect.com/science/article/pii/S0023643822003565">
                8. Xu ,Y.,Wang, Y.,Ding, X., et.al. Inhibitory effects of reuterin on biofilm formation, quorum sensing and virulence genes of Clostridium perfringens
                ,Volume 162,  June ,2022,
                https://www.sciencedirect.com/science/article/pii/S0023643822003565
                </a>
                </cite>

                <cite>
                                    <a href="https://ibdmdb.org/">
                                        9. The Inflammatory Bowel Disease Multi'omics Database. [Accessed 10/10/2022] 
                https://ibdmdb.org/
                                    </a>
                                </cite>

                <cite>
                                    <a href="https://www.frontiersin.org/articles/10.3389/fmicb.2020.604462/full">
                                        10. Zhao,Y., Zeng, Y., Zeng, D. et.al. Probiotics and MicroRNA: Their Roles in the Host–Microbe Interactions, Jan, 2021
                https://www.frontiersin.org/articles/10.3389/fmicb.2020.604462/full
                                    </a>
                                </cite>

                <cite>
                                <a href="https://bpspubs.onlinelibrary.wiley.com/doi/epdf/10.1111/bph.15421?saml_referrer">
                                        11. Pozo-Acebo,L., Hazas, M.C.., Marcolles, A., et.al. Eating microRNAs: pharmacological opportunities for cross-kingdom regulation and implications in host gene and gutmicrobiota modulation, Feb, 2021
                https://bpspubs.onlinelibrary.wiley.com/doi/epdf/10.1111/bph.15421?saml_referrer
                                    </a>
                                </cite>

                <cite>
                                <a href="https://pubmed.ncbi.nlm.nih.gov/32610070/">
                                        12. Duan Q, Zhang H, Zheng J, Zhang L. Turning Cold into Hot: Firing up the Tumor Microenvironment. Trends Cancer. 2020 Jul;6(7):605-618. doi: 10.1016/j.trecan.2020.02.022. Epub 2020 Mar 21. PMID: 32610070.)
                https://pubmed.ncbi.nlm.nih.gov/32610070/
                                    </a>
                                </cite>

                <cite>
                                <a href="https://pubmed.ncbi.nlm.nih.gov/33859752/">
                                        13. Liu YT, Sun ZJ. Turning cold tumors into hot tumors by improving T-cell infiltration. Theranostics. 2021 Mar 11;11(11):5365-5386. doi: 10.7150/thno.58390. PMID: 33859752; PMCID: PMC8039952.)
                https://pubmed.ncbi.nlm.nih.gov/33859752/
                                    </a>
                                </cite>

                <cite>
                                <a href="https://pubmed.ncbi.nlm.nih.gov/31863567/">
                                        14. Yu X, Lin C, Yu J, Qi Q, Wang Q. Bioengineered Escherichia coli Nissle 1917 for tumour-targeting therapy. Microb Biotechnol. 2020 May;13(3):629-636. doi: 10.1111/1751-7915.13523. Epub 2019 Dec 21. PMID: 31863567; PMCID: PMC7111071.
                https://pubmed.ncbi.nlm.nih.gov/31863567/
                                    </a>
                                </cite>

                <cite>
                                <a href=”https://pubmed.ncbi.nlm.nih.gov/31827064/">
                                        15. Duong MT, Qin Y, You SH, Min JJ. Bacteria-cancer interactions: bacteria-based cancer therapy. Exp Mol Med. 2019 Dec 11;51(12):1-15. doi: 10.1038/s12276-019-0297-0. PMID: 31827064; PMCID: PMC6906302.)
                https://pubmed.ncbi.nlm.nih.gov/31827064/
                                    </a>
                                </cite>

                <cite>
                                <a href=”https://pubmed.ncbi.nlm.nih.gov/33964366/">
                                        16. Li X, Luo L, Jiang M, Zhu C, Shi Y, Zhang J, Qin B, Luo Z, Guo X, Lu Y, Shan X, Liu Y, Du Y, Ling P, You J. Cocktail strategy for 'cold' tumors therapy via active recruitment of CD8+ T cells and enhancing their function. J Control Release. 2021 Jun 10;334:413-426. doi: 10.1016/j.jconrel.2021.05.002. Epub 2021 May 6. PMID: 33964366.
                https://pubmed.ncbi.nlm.nih.gov/33964366/
                                    </a>
                                </cite>

                <cite>
                                <a href=”https://pubmed.ncbi.nlm.nih.gov/32051224/">
                                        17. Gurbatri CR, Lia I, Vincent R, Coker C, Castro S, Treuting PM, Hinchliffe TE, Arpaia N, Danino T. Engineered probiotics for local tumor delivery of checkpoint blockade nanobodies. Sci Transl Med. 2020 Feb 12;12(530):eaax0876. doi: 10.1126/scitranslmed.aax0876. PMID: 32051224; PMCID: PMC7685004.
                https://pubmed.ncbi.nlm.nih.gov/32051224/
                                    </a>
                                </cite>

                <cite>
                                <a href=”https://pubmed.ncbi.nlm.nih.gov/31186085/">
                                        18. Thapa B, Lee K. Metabolic influence on macrophage polarization and pathogenesis. BMB Rep. 2019 Jun;52(6):360-372. doi: 10.5483/BMBRep.2019.52.6.140. PMID: 31186085; PMCID: PMC6605523
                https://pubmed.ncbi.nlm.nih.gov/31186085/
                                    </a>
                                </cite>

                <cite>
                                <a href=”https://pubmed.ncbi.nlm.nih.gov/28360914/">
                                        19. Geeraerts X, Bolli E, Fendt SM, Van Ginderachter JA. Macrophage Metabolism As Therapeutic Target for Cancer, Atherosclerosis, and Obesity. Front Immunol. 2017 Mar 15;8:289. doi: 10.3389/fimmu.2017.00289. PMID: 28360914; PMCID: PMC5350105.]
                https://pubmed.ncbi.nlm.nih.gov/28360914/
                                    </a>
                                </cite>

                <cite>
                                <a href=”https://pubmed.ncbi.nlm.nih.gov/29382395/">
                                        20. Mahlbacher G, Curtis LT, Lowengrub J, Frieboes HB. Mathematical modeling of tumor-associated macrophage interactions with the cancer microenvironment. J Immunother Cancer. 2018 Jan 30;6(1):10. doi:.
                https://pubmed.ncbi.nlm.nih.gov/29382395/
                                    </a>
                                </cite>

                <cite>
                                <a href=”https://pubmed.ncbi.nlm.nih.gov/30836073/">
                                        21. Mahlbacher GE, Reihmer KC, Frieboes HB. Mathematical modeling of tumor-immune cell interactions. J Theor Biol. 2019 May 21;469:47-60. doi: 10.1016/j.jtbi.2019.03.002. Epub 2019 Mar 2. PMID: 30836073; PMCID: PMC6579737.
                https://pubmed.ncbi.nlm.nih.gov/30836073/
                                    </a>
                                </cite>

                <cite>
                                <a href=”https://pubmed.ncbi.nlm.nih.gov/12529359/">
                                        22. Gardner AM, Gessner CR, Gardner PR. Regulation of the nitric oxide reduction operon (norRVW) in Escherichia coli. Role of NorR and sigma54 in the nitric oxide stress response. J Biol Chem. 2003 Mar 21;278(12):10081-6. doi: 10.1074/jbc.M212462200. Epub 2003 Jan 15. PMID: 12529359.
                https://pubmed.ncbi.nlm.nih.gov/12529359/
                                    </a>
                                </cite>

                <cite>
                                <a href=”https://pubmed.ncbi.nlm.nih.gov/26370067/">
                                        23. Aurand TC, March JC. Development of a synthetic receptor protein for sensing inflammatory mediators interferon-γ and tumor necrosis factor-α. Biotechnol Bioeng. 2016 Mar;113(3):492-500. doi: 10.1002/bit.25832. Epub 2016 Jan 15. PMID: 26370067.
                https://pubmed.ncbi.nlm.nih.gov/26370067/
                                    </a>
                                </cite>

                <cite>
                                <a href=”https://pubmed.ncbi.nlm.nih.gov/24252483/">
                                        24. Laxminarayan, R., Duse, A., Wattel, C. et.al. Antibiotic resistance-the need for global solutions, Nov, 2013.
                https://pubmed.ncbi.nlm.nih.gov/24252483/

                                    </a>
                                </cite>

                <cite>
                                <a href=”https://pubmed.ncbi.nlm.nih.gov/11271398/">
                                        25. Lacour, A.G., Gervaix, A., Zamora, S.A. . Procalcitonin, IL-6, IL-8, IL-1 receptor antagonist and C-reactive protein as identificators of serious bacterial infections in children with fever without localising signs Eur J Pediatr
                . 2001 Feb;160(2):95-100. doi: 10.1007/s004310000681. 
                https://pubmed.ncbi.nlm.nih.gov/11271398/  </a>
                                </cite>

                <cite>
                                <a href=”https://www.sciencedirect.com/science/article/pii/S1369527420300953">
                                        26. Tanna,T., Ramachanderan, R.,Platt,R. Engineered bacteria to report gut function: technologies and implementation. Volume 59, February 2021, Pages 24-33
                https://www.sciencedirect.com/science/article/pii/S1369527420300953  </a>
                                </cite>
                <cite>
                                <a href=”https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0120012">
                                        27. Oved,K.,Cohen,A.,Boico,O.  A Novel Host-Proteome Signature for Distinguishing between Acute Bacterial and Viral Infections. https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0120012  </a>
                                </cite>
                <cite>
                                <a href=”https://www.science.org/doi/10.1126/scitranslmed.aaa3601?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed">
                                        28. COURBET, A., ENDY,D., RENARD, E., MOLINA,F., BONNET,J. Detection of pathological biomarkers in human clinical samples via amplifying genetic switches and logic gates 27 May 2015
                Vol 7, Issue 289 p. 289ra83 DOI: 10.1126/scitranslmed.aaa360
                https://www.science.org/doi/10.1126/scitranslmed.aaa3601?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed
                </a>
                                </cite>

                <cite>
                                <a href=”https://www.cell.com/molecular-cell/fulltext/S1097-2765(14)00489-4?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1097276514004894%3Fshowall%3Dtrue#%20">
                                        29. Ausländer,D., Ausländer,S., Charpin-El Hamri, G.,Hierlemann,A., Stelling,J., Fussenegger,M. A Synthetic Multifunctional Mammalian pH Sensor and CO2 Transgene-Control Device
                July 10, 2014DOI:https://doi.org/10.1016/j.molcel.2014.06.007
                https://www.cell.com/molecular-cell/fulltext/S1097-2765(14)00489-4?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1097276514004894%3Fshowall%3Dtrue#%20
                </a>
                                </cite>
                <cite>
                                <a href=”https://pubmed.ncbi.nlm.nih.gov/31320117/">
                                        30. Chan,S. ,Shan,H. , Dahoun,T. , Vogel,H. ,  Yuan,S. Advancing Drug Discovery via Artificial Intelligence. PMID: 31320117 DOI: 10.1016/j.tips.2019.06.004 
                https://pubmed.ncbi.nlm.nih.gov/31320117/  </a>
                                </cite>

                <cite>
                                <a href=”https://www.fda.gov/patients/drug-development-process/step-1-discovery-and-development">
                                        31. Step 1: Discovery and Development [Accessed 10/10/2022]
                https://www.fda.gov/patients/drug-development-process/step-1-discovery-and-development  </a>
                                </cite>

                <cite>
                                <a href=”https://pubmed.ncbi.nlm.nih.gov/31320117/">
                                        32. H C Stephen Chan , Hanbin Shan , Thamani Dahoun , Horst Vogel , Shuguang Yuan. Advancing Drug Discovery via Artificial Intelligence. PMID: 31320117 DOI: 10.1016/j.tips.2019.06.004 
                https://pubmed.ncbi.nlm.nih.gov/31320117/
                </a>
                                </cite>
                <cite>
                                <a href=”https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3632255/">
                                        33. Leva,G., Croce,C. miRNA profiling of cancer. Published online 2013 Mar 4. doi: 10.1016/j.gde.2013.01.004 
                https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3632255/
                </a>
                                </cite>

                <cite>
                                <a href=”https://pubmed.ncbi.nlm.nih.gov/26481951/">
                                        34. Mishra,S., Yadav,T., Rani,V. Exploring miRNA based approaches in cancer diagnostics and therapeutics. PMID: 26481951 DOI: 10.1016/j.critrevonc.2015.10.003 
                https://pubmed.ncbi.nlm.nih.gov/26481951/
                </a>
                                </cite>

                <cite>
                                <a href=”https://pubmed.ncbi.nlm.nih.gov/30449189/">
                                        35. Xu,Υ.,Yao,Η., Lin,Κ. An overview of neural networks for drug discovery and the inputs used. PMID: 30449189 DOI: 10.1080/17460441.2018.1547278
                https://pubmed.ncbi.nlm.nih.gov/30449189/
                </a>
                                </cite>

                <cite>
                                <a href=”https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6462295/">
                                        36. Afshar, S., Afshar,S., Warden,E., Manochehri,H., Saidijam,M. Application of Artificial Neural Network in miRNA Biomarker Selection and Precise Diagnosis of Colorectal Cancer. 2019 May; 23(3): 175–183.
                doi: 10.29252/.23.3.175 
                https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6462295/
                </a>
                                </cite>

                <cite>
                                <a href=”https://pubmed.ncbi.nlm.nih.gov/34540669/">
                                        37. Cervena,K., Novosadova,V., Pardini,B., Naccarati,A.,Opattova,A., Horak,J., Vodenkova,S.,Buchler,T.,Skrobanek,P.,Levy M., Vodicka,P., Vymetalkova,V. Analysis of MicroRNA Expression Changes During the Course of Therapy In Rectal Cancer Patients. PMID: 34540669 PMCID: PMC8444897 DOI: 10.3389/fonc.2021.702258 
                https://pubmed.ncbi.nlm.nih.gov/34540669/
                </a>
                                </cite>

                <cite>
                                <a href=”https://pubs.acs.org/doi/10.1021/acssynbio.8b00530">
                                        38. Shue Wang, Nicholas J. Emery, and Allen P. Liu. A Novel Synthetic Toehold Switch for MicroRNA Detection in Mammalian Cells. https://pubs.acs.org/doi/10.1021/acssynbio.8b00530
                </a>
                                </cite>

                <cite>
                                <a href=”https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6939173/">
                                        39. Ya-Yun Xu, Qian-Hui Xia, Qing-Rong Xia, Xu-Lai Zhang,  Jun Liang. MicroRNA-Based Biomarkers in the Diagnosis and Monitoring of Therapeutic Response in Patients with Depression. Published online 2019 Dec 27. doi: 10.2147/NDT.S237116.
                https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6939173/
                </a>
                                </cite>

                <cite>
                                <a href=”https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5111516/">
                                        40. Monique A.J. van Eijndhoven, Josée M. Zijlstra, Nils J. Groenewegen etc. Plasma vesicle miRNAs for therapy response monitoring in Hodgkin lymphoma patients. Published online 2016 Nov 17. doi: 10.1172/jci.insight.89631
                https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5111516/
                </a>
                                </cite>

                <cite>
                                <a href=”https://pubmed.ncbi.nlm.nih.gov/30072748/">
                                        41.Kahraman,M., Röske,A.,Laufer,T. etc. MicroRNA in diagnosis and therapy monitoring of early-stage triple-negative breast cancer. PMID: 30072748 PMCID: PMC6072710 DOI: 10.1038/s41598-018-29917-2
                https://pubmed.ncbi.nlm.nih.gov/30072748/
                </a>
                                </cite>

                <cite>
                                <a href=”https://pubmed.ncbi.nlm.nih.gov/31402562/">
                                        42.Evita Maria Lindholm , Miriam Ragle Aure, Mads Haugland Haugen. miRNA expression changes during the course of neoadjuvant bevacizumab and chemotherapy treatment in breast cancer. PMID: 31402562 PMCID: PMC6763780 DOI: 10.1002/1878-0261.12561
                https://pubmed.ncbi.nlm.nih.gov/31402562/
                </a>
                                </cite>

                <cite>
                                <ahref=” https://www.cdc.gov/genomics/disease/pharma.htm">
                                        43.Pharmacogenomics: What does it mean for your health? [Accessed 10/10/2022] https://www.cdc.gov/genomics/disease/pharma.htm
                </a>
                                </cite>

                <cite>
                                <a href=”https://pubmed.ncbi.nlm.nih.gov/31950879/">
                                        44.Dragan Primorac , Lidija Bach-Rojecky , Dalia Vađunec , Alen Juginović , Katarina Žunić , Vid Matišić , Andrea Skelin. Pharmacogenomics at the center of precision medicine: challenges and perspective in an era of Big Data. PMID: 31950879 DOI: 10.2217/pgs-2019-0134
                https://pubmed.ncbi.nlm.nih.gov/31950879/
                </a>
                                </cite>

                <cite>
                                <a href=”https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5712454/">
                                        45.Igor Koturbash, William H Tolleson, Lei Guo,2 Dianke Yu, Si Chen, Huixiao Hong, William Mattes, Baitang Ning. microRNAs as pharmacogenomic biomarkers for drug efficacy and drug safety assessment. Published online 2015 Oct 26. doi: 10.2217/bmm.15.89
                https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5712454/
                </a>
                                </cite>

                <cite>
                                <a href=”https://pubs.acs.org/doi/10.1021/acssynbio.8b00530">
                                        46.Shue Wang, Nicholas J. Emery, and Allen P. Liu. A Novel Synthetic Toehold Switch for MicroRNA Detection in Mammalian Cell. 
                https://pubs.acs.org/doi/10.1021/acssynbio.8b00530
                </a>
                                </cite> 
                <cite>
                                    <a href="https://2020.igem.org/Team:Stockholm">
                                        47. Team Stockholm 2020
                                    </a>
                                </cite>

                <cite>
                                <a href=”https://www.nature.com/articles/nmeth.2404#citeas">
                                        48.Mutalik,V.L., Joao C Guimaraes, Guillaume Cambray, Colin Lam, Marc Juul Christoffersen, Quynh-Anh Mai, Andrew B Tran, Morgan Paull, Jay D Keasling, Adam P Arkin. Precise and reliable gene expression via standard transcription and translation initiation elements. Published: 10 March 2013.
                https://www.nature.com/articles/nmeth.2404#citeas
                </a>
                                </cite>

                <cite>
                                <a href=”https://pubs.acs.org/doi/10.1021/acssynbio.2c00139">
                                        49.Yanting DuanYanting Duan
                , Xiaojuan Zhang, Weiji Zhai, Jinpeng Zhang, Xiaomei Zhang, Guoqiang Xu, Hui Li, Zhaohong Deng, Jinsong Shi, and Zhenghong Xu. Deciphering the Rules of Ribosome Binding Site Differentiation in Context Dependence. ACS Synth. Biol. 2022, 11, 8, 2726–2740. 
                https://pubs.acs.org/doi/10.1021/acssynbio.2c00139
                </a>
                                </cite>

                <cite>
                                <a href=”https://www.soilhealthpartnership.org/blog-story/3-types-of-soil-health-indicators/">
                                        50.https://www.soilhealthpartnership.org/blog-story/3-types-of-soil-health-indicators/ [Accesed 10/10/2022]
                </a>
                                </cite>

                <cite>
                                <a href=”https://doi.org/10.1093/pcp/pcz112
                https://academic.oup.com/pcp/article/60/10/2180/5518932">
                                        51.Shin-ichi Maeda, Risa Aoba, Yuma Nishino, Tatsuo Omata. A Novel Bacterial Nitrate Transporter Composed of Small Transmembrane Proteins. Plant and Cell Physiology, Volume 60, Issue 10, October 2019, Pages 2180–2192, https://doi.org/10.1093/pcp/pcz112
                https://academic.oup.com/pcp/article/60/10/2180/5518932
                </a>
                                </cite>

                <cite>
                                <a href=”https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4339647/">
                                        52.Eva Laura von der Heyde, Benjamin Klein, Lars Abram, and Armin Hallmann. The inducible nitA promoter provides a powerful molecular switch for transgene expression in Volvox carteri. Published online 2015 Feb 18. doi: 10.1186/s12896-015-0122-3
                https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4339647/
                </a>
                                </cite>

                <cite>
                                <a href=”https://www.frontiersin.org/articles/10.3389/fpls.2015.00657/full">
                                        53.Takayuki Fujiwara, Yu Kanesaki,Shunsuke Hirooka et.al. A nitrogen source-dependent inducible and repressible gene expression system in the red alga Cyanidioschyzon merolae
                https://www.frontiersin.org/articles/10.3389/fpls.2015.00657/full
                </a>
                                </cite>

                <cite>
                                <a href=”https://www.sciencedirect.com/science/article/abs/pii/S0304389421003411#!">
                                        54.Mingjing He,  Xinni Xiong. Lei Wang et.al. 
                A critical review on performance indicators for evaluating soil biota and soil health of biochar-amended soils. Volume 414, 15 July 2021, 125378
                https://www.sciencedirect.com/science/article/abs/pii/S0304389421003411#!
                </a>
                                </cite>
                <cite>
                                <a href=”https://www.cell.com/cell-reports/pdf/S2211-1247(20)30194-7.pdf">
                                        55.Finn Stirling, Alexander Naydich,
                Juliet Bramante, ..., Adam Cusolito,
                Jeffrey Way, Pamela Silver. Synthetic Cassettes for pH-Mediated Sensing,
                Counting, and Containment
                https://www.cell.com/cell-reports/pdf/S2211-1247(20)30194-7.pdf
                </a>
                                </cite>

                <cite>
                                <a href=”https://pubmed.ncbi.nlm.nih.gov/29597294/">
                                        56.Lisa M Stiller , Erwin A Galinski , Elisabeth M H J Witt. Engineering the Salt-Inducible Ectoine Promoter Region of Halomonas elongata for Protein Expression in a Unique Stabilizing Environment. PMID: 29597294 PMCID: PMC5924526 DOI: 10.3390/genes9040184
                https://pubmed.ncbi.nlm.nih.gov/29597294/
                </a>
                                </cite>

                <cite>
                                <a href=”https://www.ibm.com/case-studies/the-weather-company-ibm-cloud/">
                                        57.IBM https://www.ibm.com/case-studies/the-weather-company-ibm-cloud/ [Accessed10/10/2022]  </a>
                                </cite>


                <cite>
                                <a href=”https://www.researchgate.net/publication/335778882_Energy_and_Policy_Considerations_for_Deep_Learning_in_NLP">
                                        58.Emma Strubell,Ananya Ganesh,Ananya Ganesh,Andrew Mccallum,Andrew Mccallum. Energy and Policy Considerations for Deep Learning in NLP. January 2019 DOI: 10.18653/v1/P19-1355 https://www.researchgate.net/publication/335778882_Energy_and_Policy_Considerations_for_Deep_Learning_in_NLP
                </a>
                                </cite>

                <cite>
                                <a href=”https://docs.google.com/document/d/1UBrxT0jQdkihdWzH1DKZTVvNkP9KyuREhOODHCMZc0M/edit">
                                        59.Ashish Vaswani, Noam Shazeer,Niki Parmar,Jakob Uszkoreit et.al. Attention Is All You Need. 
                https://docs.google.com/document/d/1UBrxT0jQdkihdWzH1DKZTVvNkP9KyuREhOODHCMZc0M/edit

                </a>
                                </cite> 

                <cite>
                                <a href=”https://ieeexplore.ieee.org/abstract/document/8398872?casa_token=5fNwkRP0vB4AAAAA:iXZexoPYBkkVD_hF-80BcwJDJDXp7FEtgk2Pz3KF4AmCuUwE-fYdZN8aNLZ6DEwz43U5wpPb0A">
                                        60. .Mitulgiri H. Gauswami; Kiran R. Trivedi. Implementation of machine learning for gender detection using CNN on raspberry Pi platform. Publisher: IEEE https://ieeexplore.ieee.org/abstract/document/8398872?casa_token=5fNwkRP0vB4AAAAA:iXZexoPYBkkVD_hF-80BcwJDJDXp7FEtgk2Pz3KF4AmCuUwE-fYdZN8aNLZ6DEwz43U5wpPb0A 

                </a>
                                </cite> 

                <cite>
                                <a href=”https://www.mdpi.com/2072-666X/13/2/191 ">
                                        61. Triwiyanto Triwiyanto, Wahyu Caesarendra,Mauridhi Hery Purnomo et.al. Embedded Machine Learning Using a Multi-Thread Algorithm on a Raspberry Pi Platform to Improve Prosthetic Hand Performance. https://www.mdpi.com/2072-666X/13/2/191 

                </a>
                                </cite> 

                <cite>
                                <a href=”https://ieeexplore.ieee.org/abstract/document/9102342 ">
                                        62. Umer Saeed, Sana Ullah Jan, Young-Doo Lee, Insoo Koo et.al. Machine Learning-based Real-Time Sensor Drift Fault Detection using Raspberry Pi. https://ieeexplore.ieee.org/abstract/document/9102342

                </a>
                                </cite>

                <cite>
                <a href=”https://jbioleng.biomedcentral.com/articles/10.1186/s13036-015-0011-2#Bib1 ">
                63. Stefan Hennig, Gerhard Rödel & Kai Ostermann . Artificial cell-cell communication as an emerging tool in synthetic biology applications Journal of Biological Engineering volume 9, Article number: 13 (2015).
                https://jbioleng.biomedcentral.com/articles/10.1186/s13036-015-0011-2#Bib1
                </a>
                </cite>

          </div>
      </section>
  </div>
</div>


<script>
    function show(param_div_id) {
    document.getElementById('cc-senders').innerHTML = document.getElementById(param_div_id).innerHTML;
  }
</script>

</body>

<!-- Footer -->
{% include 'footer.html' %}

<!--Loader-->   
{% include 'loader.html'%}  

</html>
