<!doctype html>
<html>
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="shortcut icon" href="https://static.igem.wiki/teams/4294/wiki/favicon-white.png"/>
    <link rel="license" href="https://creativecommons.org/licenses/by/4.0/"/>
    
    <!-- Custom CSS -->
    <link href="{{ url_for('static', filename = 'css/results_style.css') }}" rel="stylesheet">

    <!-- Loader -->
    <script type= "text/javascript" src="../../athens/static/scripts/jquery.min.js"></script>

    <!--Custom menu CSS-->
    <link href="{{url_for('static',filename='css/menu_style_aspa.css')}}" rel="stylesheet">

    <!--Custom footer CSS-->
    <link href="{{url_for('static',filename='css/footer_style_aspa.css')}}" rel="stylesheet">

     <!--Custom to top button CSS-->
     <link href="{{url_for('static',filename='css/to_top_button.css')}}" rel="stylesheet">
    <title>PERspectives | Results</title>
</head>

<body>
<!-- Navigation -->
{% include 'menu_custom.html' %}

<!--To top button-->
{% include 'to_top_button.html'%}

<!--Loader-->   
{% include 'loader.html'%}
/index
<div class="page_header">
    <img class="img_header" src="https://static.igem.wiki/teams/4294/wiki/wet-lab/wet-bg-darker.gif" width="100%">
    <div class="heading">Results</div>    
</div>

<div class="buttons">
  <div class="button">
      <a href="#senders">Senders</a>
  </div>
  <div class="button">
      <a href="#receivers">Receivers</a>
  </div>
</div>

<div class="overview-section-senders" id="senders">
  <h1>Senders</h1>
</div>

<div class="cc-senders">
  <div class="left-side">
      <div class="index-bar">
          <ul class="lateral-index">
              <li class="lateral-content">
                  <a href="#senders"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">Senders</span></a>
              </li>
              <li class="lateral-content">
                  <a href="#weights">Weights </a>
              </li>
              <li class="lateral-content">
                  <a href="#rbs">RBS Characterisation</a>
              </li>
              <li class="lateral-content">
                  <a href="#receivers"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">Receivers</span></a>
              </li>
              <li class="lateral-content">
                <a href="#conclusions"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">Conclusions</span></a>
              </li>
              <li class="lateral-content">
                <a href="#references"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">References</span></a>
            </li>
          </ul>
      </div>
  </div>
  
  <div class="right-side">

<!-- Overview  -->
          <div class="overview-senders">
              <h1>Overview</h1>
              <p>
                The goal of our project was to create a system of bacterial artificial intelligence. But to reach this goal, we had to determine smaller 
                steps that would lead us to make the project feasible.
                <br><br>
                The first step was to characterize 11 RBS that we chose after the characterization made by our <a href="https://2022.igem.wiki/athens/model">Dry Lab</a>, using the De Novo DNA software. 
                This would help us  determine if all of them work and which of them are the most suitable for our system.                
              </p>
          </div>

<!-- Weights  -->
        <section class="weights">
            <a id="weights" class="anchorOffset"></a>
            <br><br>
            <h1>Determining the Weights</h1>
            <br>
            <p> 
            <center>
            <img class="weights-img" src="https://static.igem.wiki/teams/4294/wiki/results/results-gifs.gif">
            </center>
            <br>
            As mentioned in the <a href="https://2022.igem.wiki/athens/description">Description</a> the weights of our bacterial artificial system are determined by different Ribosome Binding Sites (RBS). Each RBS gives an alternative translation rate giving us the opportunity to assign different weights to an input. This means that different senders populations produce different amount of the Acylhomoserine synthase when encountering the input (aTc- C22H22N2O7 · HCl).
            <br><br>
            Our team performed Golden Gate Assembly (Level 1 and Level 2) in order to make the constructs with 11 RBS variants (Table 1) (see <a href="https://2022.igem.wiki/athens/design">Design</a> and <a href="https://2022.igem.wiki/athens/engineering">Engineering Success</a>). In order to make functional constructs and characterize our RBS variants we followed several engineering cycles as described in the Engineering Success tab.
            <br><br>
            Our first constructs, which included the overlapping version of the luxI gene and mNeonGreen (<a href="https://2022.igem.wiki/athens/design">bisistronic design</a>) fluorescent protein gene, did not produce any results as assessed both by induction experiments with multiple inducer concentrations (aTc) and in different bacterial strains (<a href="#img2">Figure 2</a>). We also assessed the production of Acylhomoserine synthase and mNeonGreen with SDS-PAGE analysis (<a href="#img1">Figure 1</a>) and in-gel fluorescense of cell lysates in two cell types BL21 (DE3) (level2 construct) and DH5a-z1 (level 1 construct) after 3h induction with aTc, centrifuge and sonication of the cells. The in-gel fluorescence did not produce any results except for the positive control (OL ColE1 induced). The SDS-PAGE confirmed that our constructs did not work in BL21 cells (or the small quantity of protein was lost in the transfer step) but worked in a leaky way in DH5a-z1. This was the first sign that made us realise there was a problem with the chosen promoter for the TetR repressor, as DH5a-z1 cells encode for this repressor in their genome.
            <br><br>     

            <center>
            <img class="img1" id="img1" src="https://static.igem.wiki/teams/4294/wiki/results/figure-1.png">
            <h5>Figure 1: SDS-PAGE: OL ColEI for positive control for mNeonGreen. S4- DH5a-z1 produces luxI but not mNeon Green both at induced and uninduced state. S4-BL21 is not producing  luxI or mNeonGreen with or without induction.
            </h5>
            </center>

            <br>
            Measurements for mNeonGreen Protein were performed with excitation wavelength 476nm and emission wavelength 547nm, as recommended by the plate reader manufacturer for proteins that the emission and excitation wavelength overlap, according to the fluorescent protein database mNeonGreen has an excitation wavelength at 506nm and an emission wavelength at 517nm. It is also important to mention that according Tutol et al. (2019) [1] chloride has an effect on mNeonGreen that is why we tried changing the buffer that we resuspended the pellets for measurement in case this was affecting the measurements we were conducting (<a href="#img3">Figure 3</a>).
            <br><br>

            <center>
            <img class="img2" id="img2" src="https://static.igem.wiki/teams/4294/wiki/results/figure-2.png">
            <h5>Figure 2: Testing senders pTU1- luxI mNeonGreen with multiple aTc concentrations (2.16μΜ,0.4μΜ,0.09μΜ). This induction resulted in very low fluorescence signal, maximum 340 RFU (ratio) 20h post induction.
            </h5>
            </center>
            
            <br>
            
            <center>
            <img class="img3" id="img3" src="https://static.igem.wiki/teams/4294/wiki/results/figure-3.png">
            <h5>Figure 3: Testing the effect of chloride in mNeonGreen Fluorescence, there was no significant effect in changing the solvent where we performed the resuspension of our pellets.
            </h5>
            </center>
            
            <br>
            The resuspension of the cells to a chloride free buffer did not change the negative results. 
            <br><br>
            Last but not least in an attempt to fully characterise the problem with our constructs we wanted once again to test if the <i>luxI</i> gene 
            is expressed in the luxI-mNeonGreen construct. That is why we tested the effect of the supernatant of BL21 luxI-mNeonGreen for S4 RBS after 
            3h induction with aTc on the receivers construct OL-ColE1 (BL21). This experiment proved that the luxI gene is merely  functional and can 
            produce OC6 to activate the receivers and made us realise the problem is  also in the bisistronic design with
             mNeonGreen (<a href="#img4">Figure 4</a>). Although luxI is expressed in BL21 cells, it is not detected in SDS-PAGE 
             (<a href="#img1">Figure 1</a>) and that can be attributed to very low production of this protein.Taking into account that the coomasie 
             staining happened after this polyacrylamide gel was used in transfer for western blotting for some other samples, the remaining quantity is 
             even smaller. That is maybe why the receivers are only slightly induced by the supernatant of BL21 senders. The same construct has been 
             proven to be highly induced when using externally provided OC6 (see Receivers results) and also compared to the final sender constructs 
             (luxI sfGFP, see <a href="https://2022.igem.wiki/athens/proof-of-concept">Proof of Concept</a>). For all these reasons we considered 
             these constructs not functional and proceeded in creating new constructs.
            <br><br>

            <center>
            <img class="img4" id="img4" src="https://static.igem.wiki/teams/4294/wiki/results/figure-4.png">
            <h5>Figure 4: We tested the effect of the supernatant of BL21 luxI-mNeonGreen for S4 RBS after 3h induction with aTc on the Receiver's construct OL-ColE1 (BL21). This experiment proved that the luxI gene is functional and can produce OC6 to activate the receivers and made us realise the problem is the bisistronic design with mNeonGreen.
            </h5>
            </center>

            <br>
            As described in the <a href="https://2022.igem.wiki/athens/engineering">Engineering Success</a> page, we then proceeded in creating a new construct 
            which included the first 36 nucleotides of the <i>luxI</i> gene followed by the <i>sfGFP</i> gene in order to estimate the translation rate 
            produced by the different RBS variants taking into account that RBS sequences are context dependent. We tested this construct (level 2) 
            for 11 different RBS in the BL21(DE3) cells but there was no sign of induction (<a href="#img5">Figure 5</a>). We present an example for this 
            induction experiment for BCD2. We also tried to change the promoter which constitutively controlled the expression of the TetR operator, but again we 
            did not get any positive results (See <a href="https://2022.igem.wiki/athens/engineering">Engineering Success</a>) (<a href="#img6">Figure 6</a>).
            <br><br>

            <center>
            <img class="img5" id="img5" src="https://static.igem.wiki/teams/4294/wiki/results/figure-5.png">
            <h5>Figure 5: Induction experiment of BL21 luxI 36nt -sfGFP. Example for BCD2. The cells produced a stable amount of fluorescence expressed in RFU that did not change after induction.
            </h5>
            </center>
            <br>

            <center>
            <img class="img6" id="img6" src="https://static.igem.wiki/teams/4294/wiki/results/figure-6.png">
            <h5>Figure 6: Induction experiment of DH5a cells with pLacI promoter for the expression of TetR. The change of promoter to a previously characterised one for the same system did not produce the expected results.
            </h5>
            </center>
            <br>

            So as described in the <a href="https://2022.igem.wiki/athens/proof-of-concept">Engineering Success</a> page we proceed in transforming the level 1 constructs in E.coli DH5a-z1  which encode in their genome the 
            TetR repressor under a promoter that was proved successful for our project. The DH5a-z1 cells gave us the opportunity to fully characterize 11 RBS and 
            the results are presented below.
            <br><br>
            
            </p>
          </section>

          <!-- RBS -->

          <section class="rbs">
            <a id="rbs" class="anchorOffset"></a>
            <br><br>        
            <h1>RBS Characterisation</h1>
            <br>
            <center>
            <table class="content-table" id="tb1">
              <thead>
                <tr>
                  <th>Abbreviation</th>
                  <th>Full Name</th>
                  <th>Part</th>
                  <th>Strenght</th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <td>S4</td>
                  <td>Synthetic RBS Ath4</td>
                  <td><a href="http://parts.igem.org/Part:BBa_K4294004"> BBa_K4294004  </a></td>
                  <td>Strong</td>
                </tr>
                <tr class="active-row">
                  <td>S5</td>
                  <td>Synthetic RBS Ath5</td>
                  <td><a href="http://parts.igem.org/Part:BBa_K4294005"> BBa_K4294005 </a></td>
                  <td>Strong</td>
                </tr>
                <tr>
                   <td>S10</td>
                  <td>Synthetic RBS Ath10</td>
                  <td><a href="http://parts.igem.org/Part:BBa_K4294010"> BBa_K4294010 </a></td>
                  <td>Medium</td>
                </tr>
                <tr class="active-row">
                  <td>S11</td>
                  <td>Synthetic RBS Ath11</td>
                  <td><a href="http://parts.igem.org/Part:BBa_K4294011"> BBa_K4294011  </a></td>
                  <td>Medium</td>
                </tr>
                <tr>
                   <td>S12</td>
                  <td>Synthetic RBS Ath12</td>
                  <td><a href="http://parts.igem.org/Part:BBa_K4294012"> BBa_K4294012  </a></td>
                  <td>Weak</td>
                </tr>
                <tr class="active-row">
                  <td>S13</td>
                  <td>Synthetic RBS Ath13</td>
                  <td><a href="http://parts.igem.org/Part:BBa_K4294013"> BBa_K4294013 </a></td>
                  <td>Weak</td>
                </tr>
                <tr>
                   <td>BCD1</td>
                  <td>BCD1</td>
                  <td><a href='http://parts.igem.org/Part:BBa_J428034'> BBa_J428034 </a></td>
                  <td>Weak</td>
                </tr>
                <tr class="active-row">
                  <td>BCD2</td>
                  <td>BCD2</td>
                  <td><a href="http://parts.igem.org/Part:BBa_K4294102"> BBa_K4294102 </a></td>
                  <td>Very Strong</td>
                </tr>
                <tr>
                   <td>BCD8</td>
                  <td>BCD8</td>
                  <td><a href='http://parts.igem.org/Part:BBa_K2680561'> BBa_K2680561 </a></td>
                  <td>Weak</td>
                </tr>
                <tr class="active-row">
                  <td>BCD12</td>
                  <td>BCD12</td>
                  <td><a href="http://parts.igem.org/Part:BBa_K2680529"> BBa_K2680529  </a></td>
                  <td>Weak</td>
                </tr>
                <tr>
                   <td>BCD14</td>
                  <td>BCD14</td>
                  <td><a href='http://parts.igem.org/Part:BBa_K4294114'> BBa_K4294114 </a></td>
                  <td>Weak</td>
                </tr>
                
              </tbody>
          </table>
          <figcaption>Table 1: All the Ribosome Binding Sites (RBS) characterised in our project classified by their strength as indicated by De Novo DNA 
            (RBS calculator). The BCDs were also more accurately predicted by Mutalik, V. et al. [2] for our project but are not represented in this 
            table</figcaption>
          </center>
          <br>
          <p> 
          As described in the protocols, we used the FlexStation3 (Molecular Devices) Plate Reader to perform the Fluorescence and the OD600nm measurements. 
          The exact Protocol for the induction is described in the protocol section. Below we present in diagrams the results for the luxI-sfGFP constructs for 
          the 11 RBS in different time frames (0-240min) from the initiation of the induction and in different inducer concentrations (2-5,12*10^(-6)μΜ) 
          (Table 2). Last but not least, we compare the RBS for the difference in sfGFP production with the luxI context dependence (Figures 4,5,6). All the 
          measurements of fluorescence for sfGFP were performed with gain 300, excitation wavelength 485, emission wavelength 510. Regarding the OD600nm, all 
          measurements were recorded without using the Pathcheck option to fulfil the needs of our project.
          </p>
          <br>

          <center>
          <img class="img6" id="tb2" src="https://static.igem.wiki/teams/4294/wiki/results/1-1.png">
          <img class="img6" id="tb2" src="https://static.igem.wiki/teams/4294/wiki/results/1-2.png">
          <img class="img6" id="tb2" src="https://static.igem.wiki/teams/4294/wiki/results/1-3.png">
          <img class="img6" id="tb2" src="https://static.igem.wiki/teams/4294/wiki/results/1-4.png">
          <img class="img6" id="tb2" src="https://static.igem.wiki/teams/4294/wiki/results/1-5.png">
          <img class="img6" id="tb2" src="https://static.igem.wiki/teams/4294/wiki/results/1-6.png">
          <img class="img6" id="tb2" src="https://static.igem.wiki/teams/4294/wiki/results/1-7.png">
          <img class="img6" id="tb2" src="https://static.igem.wiki/teams/4294/wiki/results/1-8.png">

          <img class="img6" id="tb2" src="https://static.igem.wiki/teams/4294/wiki/results/1-13.png">
          <img class="img6" id="tb2" src="https://static.igem.wiki/teams/4294/wiki/results/1-14.png">
          <figcaption>Table 2: Our results for S4,S5,S10,S11,S12,S13,BCD2,BCD8,BCD12,BCD14 in different concentrations at different time points, the concentration range used for aTc was 2μM to 5,12 *10^(-6) performing 5-fold dilutions, the maximum induction time was 6 hours, here we present data until 4 hours, further induction did not result in increase of sfGFP production. BCD1 is not represented here because it did not produce significant induction after repeating the same experiment two times.</figcaption>
          </center>
          <br><br>



          <center>
          <img class="img7" id="img7" src="https://static.igem.wiki/teams/4294/wiki/results/figure-7.png">
          <h5>Figure 7: Comparison of all Synthetic RBS as characterised by our measurements.
          </h5>
          </center>
          <br>
          <center>
          <img class="img8" id="img8" src="https://static.igem.wiki/teams/4294/wiki/results/figure-8.png">
          <h5>Figure 8: Comparison of BCDs as characterised by our measurements.
          </h5>
          </center>
          <br>
          <center>
          <img class="img9a" id="img9a" src="https://static.igem.wiki/teams/4294/wiki/results/figure-9-a.png">
          <img class="img9b" id="img9b" src="https://static.igem.wiki/teams/4294/wiki/results/figure-9.png">
          <h5>Figure 9: (a) Comparison of all 11 RBS  as characterised by our measurements (b) Comparison of the weaker RBS that can not be best represented in figure 6(a).
          </h5>
          </center>
          <br>

          From <a href="tb2">Table 2</a> and Figures <a href="img4">4</a>,<a href="img5">5</a>,<a href="img6">6</a> we can conclude that the characterised 
          RBS can be shorted by strength as follows: 
          <br><br>

          <h2>BCD2 > S11 > S13 > S10 > BCD12 > S4 > BCD14 > S12 >BCD8 > S5 > BCD1</h2>

          <br><br>
          These results vary a lot from our predictions. This can be attributed to change of strain from our initial BL21 (DE3) for which the Dry Lab made the calculations, to DH5a-z1 in order for our system to work. It is also important to note that, as we learned during meetings that are recorded in our Integrated Human Practices Page, both Dr. Skretas and Dr. Di Ventura mentioned that RBSs are a good way to calculate weights but RBS calculators are not always reliable and should always be confirmed with experimental procedures.Τhat is why our team tested 11 RBS sequences instead of 3 that were needed for our project. 
          <br><br>
          In this set of experiments we achieved the characterization of 11 RBS in the context dependence of <i>luxI</i>. This is a library that future teams 
          may use in order to incorporate this gene to their contstracts and control the luxI translation in a way that fits the needs of their project.
          <br><br>
          We then proceeded in a second series of experiments for our constructs. This time we used the <i>luxI</i> only constructs. Because the <i>luxI</i> 
          gene expression does not result in any visual output and because we did not have the resources to perform either affinity chromatography or western 
          blotting even though we had included a 6*His-tag in our constructs, we chose an indirect way to quantify its production. 
          <br><br>
          These experiments aimed to characterise the activation of the Receiver caused by the different RBS. As mentioned in the protocols, we used the 
          supernatant of each one of the senders after 3h induction with multiple concentrations of aTc for the induction of the Receivers 
          (OL ColE1 construct) and quantified the production of mNeonGreen produced when induced with different supernatants. The results are presented 
          in <a href="tb3">Table 3</a>.
          <br><br>

          <center>
            <img class="img6" id="tb3" src="https://static.igem.wiki/teams/4294/wiki/results/table-3-3-1.png">
            <img class="img6-3-2" id="tb3" src="https://static.igem.wiki/teams/4294/wiki/results/table-3-2.png">
            <img class="img6" id="tb3" src="https://static.igem.wiki/teams/4294/wiki/results/table-3-3.png">
            <img class="img6" id="tb3" src="https://static.igem.wiki/teams/4294/wiki/results/1-4.png">
            <img class="img6" id="tb3" src="https://static.igem.wiki/teams/4294/wiki/results/table-3-4.png">
            <img class="img6" id="tb3" src="https://static.igem.wiki/teams/4294/wiki/results/table-3-5.png">
            <img class="img6" id="tb3" src="https://static.igem.wiki/teams/4294/wiki/results/table-3-6.png">
            <img class="img6-3-9" id="tb3" src="https://static.igem.wiki/teams/4294/wiki/results/table-3-9.png">
            <img class="img6" id="tb3" src="https://static.igem.wiki/teams/4294/wiki/results/table-3-8.png">
            <img class="img6" id="tb3" src="https://static.igem.wiki/teams/4294/wiki/results/table-3-10.png">
            <img class="img6" id="tb3" src="https://static.igem.wiki/teams/4294/wiki/results/table-3-11.png">
            <img class="img6" id="tb3" src="https://static.igem.wiki/teams/4294/wiki/results/table-3-12.png">
            <figcaption>Table 3: Our results for use of supernatants of  S4,S5,S10,S11,S12,S13,BCD1,BCD2,BCD8,BCD12,BCD14 senders which were induced for 3h, the 
              concentration range used for aTc was 2μM to 5,12 *10^(-6) performing 5-fold dilutions, the maximum induction time for receivers was 6 hours, here we 
              present data until 4 hours, further induction did not result in increase of mNeonGreen production.
            </figcaption>
            </center>
            <br><br>

            <center>
              <img class="img10" id="img10" src="https://static.igem.wiki/teams/4294/wiki/results/figure-10.png">
              <h5>Figure 10: Comparison of all Synthetic RBS for the differences in activating the Receivers population.
              </h5>
              </center>
              <br>
              <center>
              <img class="img11" id="img11" src="https://static.igem.wiki/teams/4294/wiki/results/figure-11.png">
              <h5>Figure 11: Comparison of all BCDs for the differences in activating the Receivers population.
              </h5>
              </center>
              <br>

              <p>
              The results are not  similar to the characterisation of the RBSs with sfGFP for many reasons. We tried to name some reasons why this 
              difference is observed. First of all, these results concern the activation of the Receivers. The activation rate depends on the amount of
               OC6 produced by the Senders, which is not known, and it also depends on the concentration of Sender cells at the 3 hours, when the
                induction happened. Our team did not take into account the sender cells population density after the 3 hour induction, therefore the 
                differences encountered between our characterization and the measurement of the luxI-sfGFP constructs can not be considered to be the 
                same because the results caused by the supernatants could be attributed to differences in the senders population density and not the RBS 
                strength. Also, the induction time of the senders may not have been the appropriate and more experiments are needed to optimise that. 
                We also came to the conclusion that a different ratio of Senders:Receivers is needed. After sharing our results with the Dry lab, they 
                informed us that the optimal ratio is 1:10. We incorporated this knowledge into the proof of concept experiments. Taking into account the 
                above, we present these results to prove the capability of the senders to activate in different amounts the receivers and to point out 
                the importance for the appropriate balance to exist between the two populations. We elaborate more on that in 
                the <a href="https://2022.igem.wiki/athens/proof-of-concept">Proof of Concept</a> page.
              </p>

        </section>


  </div>
</div>

<div class="overview-section-receivers" id="receivers">
  <h1>Receivers</h1>
</div>

<div class="cc-receivers">
  <div class="left-side">
      <div class="index-bar">
          <ul class="lateral-index">
              <li class="lateral-content">
                  <a href="#senders"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">Senders</span></a>
              </li>
              <li class="lateral-content">
                  <a href="#weights">Weights </a>
              </li>
              <li class="lateral-content">
                  <a href="#rbs">RBS Characterisation</a>
              </li>
              <li class="lateral-content">
                  <a href="#receivers"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">Receivers</span></a>
              </li>
              <li class="lateral-content">
                <a href="#conclusions"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">Conclusions</span></a>
              </li>
              <li class="lateral-content">
                  <a href="#references"><span style="font-size: 28px; font-family: 'Glacial Indifference Bold'; color: #61b872;">References</span></a>
              </li>
          </ul>
      </div>
  </div>
  
  <div class="right-side">


<!-- Receivers  -->
        <section class="res-sec">
            <a id="res-sec" class="anchorOffset"></a>
            As we described in our <a href="https://2022.igem.wiki/athens/design">Design</a> page, we wanted to test different constructs of the Receivers’ circuit in order to better represent the binary form of the output of the Perceptron algorithm. When the intermediate molecule exceeds the threshold, then the output is produced. We designed our system that way to achieve a steep activation that simulates better the function of the algorithm (Output:1 or Output:0), and also achieve the reduction of leakiness. 
            <br><br> 
            The constructs we tried to test are presented in the table below:
            <br>
            <center>
            <table class="content-table">
              <thead>
                <tr>
                  <th>Construct</th>
                  <th>Plasmid</th>
                  <th>BioBrick</th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <td>Open Loop (OL)</td>
                  <td>pTU2A-RFP p15 or.</td>
                  <td>BBa_K4294801</td>
                </tr>
                <tr class="active-row">
                  <td>Open Loop (OL)</td>
                  <td>pTU2A-RFP colE1 or.</td>
                  <td>BBa_K4294801</td>
                </tr>
                <tr>
                  <td>Positive Feedback (PF)</td>
                  <td>pTU2A-RFP colE1 or.</td>
                  <td>BBa_K4294802</td>
                </tr>
                <tr class="active-row">
                  <td>Positive Feedback constitutive (PFc)</td>
                  <td>pTU2A-RFP colE1 or.</td>
                  <td>BBa_K4294803</td>
                </tr>
                <tr>
                  <td>Positive Feedback represor (PFr)</td>
                  <td>pTU2A-RFP colE1 or.</td>
                  <td>BBa_K4294804</td>
                </tr>
                <tr class="active-row">
                  <td>Positive Feedback constitutiveand represor (PFc+r)</td>
                  <td>pTU2A-RFP colE1 or.</td>
                  <td>BBa_K4294805</td>
                </tr>
              </tbody>
          </table>
        </center>

            We used different concentrations of externally provided OC6 to induce the Receivers separately from the Senders in order to characterise their activation.
            <br><br> 
            After some test experiments we decided to measure the Fluorescence without using Pathcheck at the plate reader, as we realised that after some hours, the OD could not be measured with Pathcheck, due to cells' exponential growth. 
            Also, our measurements were taken until the timepoint of 4h, because we observed that at approximately 3h fluorescence stabilises and dying cells start to precipitate  at the bottom of the plate, affecting the OD measurement.
            <br><br>
            Measurements for mNeonGreen Protein were performed with excitation wavelength 476nm and emission wavelength 547nm, as recommended by the plate reader manufacturer for proteins that the emission and excitation wavelength overlap, according to the fluorescent protein database mNeonGreen has an excitation wavelength at 506nm and an emission wavelength at 517nm.The protocol we followed for the induction with OC6 can be found in the protocols page. 
            <br><br>
            We used different concentrations of OC6 for each construct according to the optimization experiments. We measured Absorbance at 600 nm and Fluorescence as described in our Protocols book. We then calculated the ratio Fluorescnce/OD600nm. Below are presented the results of ratio increase in different concentrations and in time.
            <br><br>


            <center>
            <img class="img12" id="img12" src="https://static.igem.wiki/teams/4294/wiki/results/figure-12.png">
            <h5>Figure 12: Induction of BL21 OL pTU2-RFP p15A or. with the following OC6 concentrations (μΜ): 100, 10, 1, 0.1, 0.01, 0.001, 0.0001. The uninduced cells are represented by the concentration value 0.0000001 for the diagram purposes. 
            </h5>
            </center>
            <br>

            It is observed that the fluorescence ratio increases as the concentration increases. But after 0.01 μΜ the ratio reaches its highest value. 
            <br><br>
            We present at <a href="#img13">Figure 13</a> the results of the induction of the same construct, but using the other pTU2-RFP origin plasmid (coLE1).
            <br><br>

            <center>
            <img class="img13" id="img13" src="https://static.igem.wiki/teams/4294/wiki/results/figure-13.png">
            <h5>Figure 13: Induction of BL21 OpLo pTU2-RFP colE1 or. with the following OC6 concentrations (μΜ): 100, 10, 1, 0.1, 0.01, 0.001, 0.0001. The uninduced cells are represented by the concentration value 0.0000001 for the diagram purposes. 
            </h5>
            </center>
            <br> 

            Then, in order to achieve steeper activation, we tested the PF construct, but as it can be seen in <a href="#img14">Figure 14</a>, the results were not expected. 
            <br><br>

            <center>
            <img class="img14" id="img14" src="https://static.igem.wiki/teams/4294/wiki/results/figure-14.png">
            <h5>Figure 14: Induction of BL21 PF pTU2-RFP colE1 or. with the following OC6 concentrations (μΜ): 0.062, 0.031, 0.0155, 0.003875, 0.0019375, 0.00096875, 0.000484375, 0.000242188, 0.000121094. The uninduced cells are represented by the concentration value 0.00001 for the diagram purposes.
            </h5>
            </center>
            <br> 

            <center>
            <img class="img15" id="img15" src="https://static.igem.wiki/teams/4294/wiki/results/figure-15.png">
            <h5>Figure 15: Induction of BL21 PFc pTU2-RFP colE1 or. with the following OC6 concentrations (μΜ): 10, 10, 1, 0.1, 0.01, 0.001, 0.0001. The uninduced cells are represented by the concentration value 0.000001 for the diagram purposes. 
            </h5>
            </center>
            <br>           
            
            Once again PFc did not present higher and steeper activation than the OL. 

            <center>
            <img class="img16" id="img16" src="https://static.igem.wiki/teams/4294/wiki/results/figure-16.png">
            <h5>Figure 16: Induction of BL21 PFr pTU2-RFP colE1 or. with the following OC6 concentrations (μΜ):100, 20, 4, 0.16,    0.0064, 0.00128, 0.000256, 0.00001024. The uninduced cells are represented by the concentration value 0.000001 for the diagram purposes. 
            </h5>
            </center>
            <br>

            PFr presented a steeper activation in comparison with OL, as it can be seen at Figure 12. By designing and testing all the different constructs we managed to accomplish the goal of a steeper activation and less leakiness.
            <br><br>
            Finally, the last construct PFc+r did not work either, as presented in <a href="#img17">Figure 17</a>.
            <br><br>

            <center>
            <img class="img17" id="img17" src="https://static.igem.wiki/teams/4294/wiki/results/figure-17.png">
            <h5>Figure 17: Induction of BL21 PFc+r pTU2-RFP colE1 or. with the following OC6 concentrations (μΜ): 10, 2, 0.4, 0.08, 0.016, 0.0032, 0.00064, 0.000128, 0.0000256, 0.00000512. The uninduced cells are represented by the concentration value 0.0000001 for the diagram purposes.
            </h5>
            </center>
            <br><br>
            
          </p>
          

          <div class="overview-receivers">
            <a id="conclusions" class="anchorOffset"></a>
            <h1>Conclusions</h1>
            <p>
              “Perspectives” aims to create a functional bacterial artificial intelligence tool. A tool that leads the receivers populations making a 
              decision according to their surroundings. In order to make a functional communication system, there were several obstacles that had to 
              be surpassed. First we had to choose a way that the weights - analogues, would be represented, our team chose for this cause RBS sequences
               in order for “Perspectives” to be a versatile, “plug in” system, for future use. Choosing the right RBS sequences was challenging, because 
               although there are helpful online tools that determine an RBS' strength, they do not always reflect the experimental results. That is why 
               our team chose to experimentally characterize 11 RBS variants, instead of just the three we needed. The RBS were chosen after using RBS 
               calculators. We were called to do a lot of troubleshooting in order to make our system work, but we finally achieved a very thorough 
               characterization of 11 RBS variants, which consist of different Senders populations that can be used to fulfil the needs of 
               future <a href="https://2022.igem.wiki/athens/implementation">implementations</a>. Our characterization also describes the behaviour 
               of different RBS variants in the context of the <i>luxI</i> gene.
              <br><br>
              Our future goals regarding the senders include introducing different inputs of different natures such as chemicals, light or temperature. 
              We also aim to include negative weights to our project by including inputs that lead to the expression of a quorum quenching enzyme in 
              order to increase safety and also be able to introduce factors that “inform” our system that there is a factor that contradicts the 
              activation of the receivers.
              <br><br>
              The second component of “Perspectives”, the receivers, had also to be carefully designed and also had some challenging aspects. 
              As our project is inspired by computer science, in bioinformatics language, we wanted the receivers population to be able to take a binary 
              decision, a decision in the form of “0” or “1”. This means that the receivers population as mentioned in our results but also in the 
              <a href="https://2022.igem.wiki/athens/design">Design</a> page had to be steeply activated, that is why we tested multiple constructs, some simple and some more complex. From the 
              constructs we tested as indicated by the figures above, the most functional but without steep activation function were the open loops 
              (ColE1+P15A) that we also used for the interaction between the senders and the receivers. But the one that seems extremely promising is 
              the PFR construct that is presented with a very steep slope.
              <br><br>
              Our future goals include testing the system as a whole with the PFR construct to optimise its function.              
            </p>
        </div>
      </section>


  <section class="collapsible" id="references">
  <input type="checkbox" id="collapsible-head">
      <label for ="collapsible-head">References</label>    
      <div class="collapsible-content">
        <cite>
          <a href="https://chemistry-europe.onlinelibrary.wiley.com/doi/abs/10.1002/cbic.201900147">
              1. Tutol, J. N., Kam, H. C., & Dodani, S. C. (2019). Identification of mNeonGreen as a pH-Dependent, Turn-On Fluorescent Protein Sensor for Chloride. Chembiochem : a European journal of chemical biology, 20(14), 1759–1765. https://doi.org/10.1002/cbic.201900147

          </a>
      </cite>
      <cite>
          <a href="https://www.nature.com/articles/nmeth.2404">
              2.  Mutalik, V. K., Guimaraes, J. C., Cambray, G., Lam, C., Christoffersen, M. J., Mai, Q. A., Tran, A. B., Paull, M., Keasling, J. D., Arkin, A. P., & Endy, D. (2013). Precise and reliable gene expression via standard transcription and translation initiation elements. Nature methods, 10(4), 354–360. https://doi.org/10.1038/nmeth.2404
          </a>
      </cite>

      </div>
  </section>
  </div>
</div>

<!-- Footer -->
{% include 'footer.html' %}

<!--Loader-->   
{% include 'loader.html'%}  

</html>